<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>2 A role for cuticle stretch in developmental growth control | Mechanisms of Larval Growth Control in Caenorhabditis elegans</title>
  <meta name="description" content="Nyaanga PhD Dissertation" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="2 A role for cuticle stretch in developmental growth control | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/img/NU_seal.png" />
  <meta property="og:description" content="Nyaanga PhD Dissertation" />
  <meta name="github-repo" content="jnyaanga" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="2 A role for cuticle stretch in developmental growth control | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  
  <meta name="twitter:description" content="Nyaanga PhD Dissertation" />
  <meta name="twitter:image" content="/img/NU_seal.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="introduction.html"/>
<link rel="next" href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="">Nyaanga Dissertation</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="dedication.html"><a href="dedication.html"><i class="fa fa-check"></i>Dedication</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#Section1.1"><i class="fa fa-check"></i><b>1.1</b> Phenotypic variation</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#from-genotype-to-phenotype"><i class="fa fa-check"></i><b>1.1.1</b> From genotype to phenotype</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#Section1.2"><i class="fa fa-check"></i><b>1.2</b> Quantitative trait loci mapping</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#Section1.3"><i class="fa fa-check"></i><b>1.3</b> <em>Caenorhabditis elegans</em></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#as-a-model-organism"><i class="fa fa-check"></i><b>1.3.1</b> As a model organism</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#post-embryonic-development"><i class="fa fa-check"></i><b>1.3.2</b> Post-embryonic development</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#growth-regulation"><i class="fa fa-check"></i><b>1.3.3</b> Growth regulation</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#natural-genetic-variation"><i class="fa fa-check"></i><b>1.3.4</b> Natural genetic variation</a></li>
<li class="chapter" data-level="1.3.5" data-path="introduction.html"><a href="introduction.html#linkage-mapping"><i class="fa fa-check"></i><b>1.3.5</b> Linkage mapping</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><i class="fa fa-check"></i><b>2</b> A role for cuticle stretch in developmental growth control</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#preface"><i class="fa fa-check"></i><b>2.1</b> Preface</a></li>
<li class="chapter" data-level="2.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#abstract-1"><i class="fa fa-check"></i><b>2.2</b> Abstract</a></li>
<li class="chapter" data-level="2.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#introduction-1"><i class="fa fa-check"></i><b>2.3</b> Introduction</a></li>
<li class="chapter" data-level="2.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#materials-and-methods"><i class="fa fa-check"></i><b>2.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#worm-culture"><i class="fa fa-check"></i><b>2.4.1</b> Worm culture</a></li>
<li class="chapter" data-level="2.4.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#bacterial-food"><i class="fa fa-check"></i><b>2.4.2</b> Bacterial food</a></li>
<li class="chapter" data-level="2.4.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#growth-of-the-animals"><i class="fa fa-check"></i><b>2.4.3</b> Growth of the animals</a></li>
<li class="chapter" data-level="2.4.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#high-throughput-measurements-of-body-size-and-fluorescence"><i class="fa fa-check"></i><b>2.4.4</b> High-throughput measurements of body size and fluorescence</a></li>
<li class="chapter" data-level="2.4.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#image-processing"><i class="fa fa-check"></i><b>2.4.5</b> Image processing</a></li>
<li class="chapter" data-level="2.4.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#data-processing"><i class="fa fa-check"></i><b>2.4.6</b> Data processing</a></li>
<li class="chapter" data-level="2.4.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#molt-analysis"><i class="fa fa-check"></i><b>2.4.7</b> Molt analysis</a></li>
<li class="chapter" data-level="2.4.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#comparison-of-model-fits"><i class="fa fa-check"></i><b>2.4.8</b> Comparison of model fits</a></li>
<li class="chapter" data-level="2.4.9" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#stretcher-model-analysis"><i class="fa fa-check"></i><b>2.4.9</b> Stretcher model analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#results"><i class="fa fa-check"></i><b>2.5</b> Results</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#quantitative-measurements-of-c.-elegans-growth"><i class="fa fa-check"></i><b>2.5.1</b> Quantitative measurements of <em>C. elegans</em> growth</a></li>
<li class="chapter" data-level="2.5.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#fluorescence-provides-a-quantitative-measurement-of-animal-feeding-behavior-and-developmental-progression"><i class="fa fa-check"></i><b>2.5.2</b> Fluorescence <strong>provides a quantitative measurement of animal feeding behavior and developmental progression</strong></a></li>
<li class="chapter" data-level="2.5.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#changes-in-c.-elegans-body-shape-occur-at-larval-stage-transitions"><i class="fa fa-check"></i><b>2.5.3</b> Changes in <em>C. elegans</em> body shape occur at larval-stage transitions</a></li>
<li class="chapter" data-level="2.5.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#modeling-c.-elegans-cuticle-stretch-dynamics"><i class="fa fa-check"></i><b>2.5.4</b> Modeling <em>C. elegans</em> cuticle stretch dynamics</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#discussion"><i class="fa fa-check"></i><b>2.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#a-potential-role-for-cuticle-stretch-in-the-timing-of-larval-stage-transitions"><i class="fa fa-check"></i><b>2.6.1</b> A potential role for cuticle stretch in the timing of larval-stage transitions</a></li>
<li class="chapter" data-level="2.6.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#development-comprises-complex-interactions-of-growth-regulation-across-diverse-scales"><i class="fa fa-check"></i><b>2.6.2</b> Development comprises complex interactions of growth regulation across diverse scales</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#contributions"><i class="fa fa-check"></i><b>2.7</b> Contributions</a></li>
<li class="chapter" data-level="2.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#supplement"><i class="fa fa-check"></i><b>2.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><i class="fa fa-check"></i><b>3</b> Multiple genetic loci underlie differences in <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="3.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#preface-1"><i class="fa fa-check"></i><b>3.1</b> Preface</a></li>
<li class="chapter" data-level="3.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#abstract-2"><i class="fa fa-check"></i><b>3.2</b> Abstract</a></li>
<li class="chapter" data-level="3.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#introduction-2"><i class="fa fa-check"></i><b>3.3</b> Introduction</a></li>
<li class="chapter" data-level="3.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#materials-and-methods-1"><i class="fa fa-check"></i><b>3.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#strains"><i class="fa fa-check"></i><b>3.4.1</b> Strains</a></li>
<li class="chapter" data-level="3.4.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-growth-assay"><i class="fa fa-check"></i><b>3.4.2</b> High-throughput growth assay</a></li>
<li class="chapter" data-level="3.4.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-fitness-assay-for-linkage-mapping"><i class="fa fa-check"></i><b>3.4.3</b> High-throughput fitness assay for linkage mapping</a></li>
<li class="chapter" data-level="3.4.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#linkage-mapping-1"><i class="fa fa-check"></i><b>3.4.4</b> Linkage mapping</a></li>
<li class="chapter" data-level="3.4.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#generation-of-chromosome-substitution-strains-csss-and-near-isogenic-lines-nils"><i class="fa fa-check"></i><b>3.4.5</b> Generation of chromosome substitution strains (CSSs) and near-isogenic lines (NILs)</a></li>
<li class="chapter" data-level="3.4.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#statistical-analysis-of-css-and-nil-results"><i class="fa fa-check"></i><b>3.4.6</b> Statistical analysis of CSS and NIL results</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#results-1"><i class="fa fa-check"></i><b>3.5</b> Results</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#growth-dynamics-of-the-n2-and-cb4856-strains"><i class="fa fa-check"></i><b>3.5.1</b> Growth dynamics of the N2 and CB4856 strains</a></li>
<li class="chapter" data-level="3.5.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-qtl-underlying-variation-in-growth"><i class="fa fa-check"></i><b>3.5.2</b> Identification of QTL underlying variation in growth</a></li>
<li class="chapter" data-level="3.5.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#validation-of-loci-associated-with-differences-in-animal-size"><i class="fa fa-check"></i><b>3.5.3</b> Validation of loci associated with differences in animal size</a></li>
<li class="chapter" data-level="3.5.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-candidate-genes-in-the-chromosome-x-qtl"><i class="fa fa-check"></i><b>3.5.4</b> Identification of candidate genes in the chromosome X QTL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#discussion-1"><i class="fa fa-check"></i><b>3.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#a-complex-genetic-architecture-underlies-differences-in-body-size"><i class="fa fa-check"></i><b>3.6.1</b> A complex genetic architecture underlies differences in body size</a></li>
<li class="chapter" data-level="3.6.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#candidate-genes-for-variation-in-body-size"><i class="fa fa-check"></i><b>3.6.2</b> Candidate genes for variation in body size</a></li>
<li class="chapter" data-level="3.6.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#comparison-with-previous-qtl-studies-of-c.-elegans-growth"><i class="fa fa-check"></i><b>3.6.3</b> Comparison with previous QTL studies of <em>C. elegans</em> growth</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#contributions-1"><i class="fa fa-check"></i><b>3.7</b> Contributions</a></li>
<li class="chapter" data-level="3.8" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#supplement-1"><i class="fa fa-check"></i><b>3.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><i class="fa fa-check"></i><b>4</b> easyXpress: An R package to analyze and visualize <em>C. elegans</em> microscopy data generated using CellProfiler</a>
<ul>
<li class="chapter" data-level="4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preface-2"><i class="fa fa-check"></i><b>4.1</b> Preface</a></li>
<li class="chapter" data-level="4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#abstract-3"><i class="fa fa-check"></i><b>4.2</b> Abstract</a></li>
<li class="chapter" data-level="4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#introduction-3"><i class="fa fa-check"></i><b>4.3</b> Introduction</a></li>
<li class="chapter" data-level="4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#materials-and-methods-2"><i class="fa fa-check"></i><b>4.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preparation-of-animals-for-imaging"><i class="fa fa-check"></i><b>4.4.1</b> Preparation of animals for imaging</a></li>
<li class="chapter" data-level="4.4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#imaging"><i class="fa fa-check"></i><b>4.4.2</b> Imaging</a></li>
<li class="chapter" data-level="4.4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#paraquat-dose-response"><i class="fa fa-check"></i><b>4.4.3</b> Paraquat dose response</a></li>
<li class="chapter" data-level="4.4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#naming-conventions"><i class="fa fa-check"></i><b>4.4.4</b> Naming conventions</a></li>
<li class="chapter" data-level="4.4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-availability"><i class="fa fa-check"></i><b>4.4.5</b> Data availability</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#results-2"><i class="fa fa-check"></i><b>4.5</b> Results</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#design-and-implementation"><i class="fa fa-check"></i><b>4.5.1</b> Design and implementation</a></li>
<li class="chapter" data-level="4.5.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-import-and-model-assignment"><i class="fa fa-check"></i><b>4.5.2</b> Data import and model assignment</a></li>
<li class="chapter" data-level="4.5.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-pruning-and-summarization"><i class="fa fa-check"></i><b>4.5.3</b> Data pruning and summarization</a></li>
<li class="chapter" data-level="4.5.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#visualization"><i class="fa fa-check"></i><b>4.5.4</b> Visualization</a></li>
<li class="chapter" data-level="4.5.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#application-to-c.-elegans-growth-data"><i class="fa fa-check"></i><b>4.5.5</b> Application to <em>C. elegans</em> growth data</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#conclusions"><i class="fa fa-check"></i><b>4.6</b> Conclusions</a></li>
<li class="chapter" data-level="4.7" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#contributions-2"><i class="fa fa-check"></i><b>4.7</b> Contributions</a></li>
<li class="chapter" data-level="4.8" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#supplement-2"><i class="fa fa-check"></i><b>4.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion-2.html"><a href="discussion-2.html"><i class="fa fa-check"></i><b>5</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion-2.html"><a href="discussion-2.html#summary"><i class="fa fa-check"></i><b>5.1</b> Summary</a></li>
<li class="chapter" data-level="5.2" data-path="discussion-2.html"><a href="discussion-2.html#the-future-of-developmental-growth-studies"><i class="fa fa-check"></i><b>5.2</b> The future of developmental growth studies</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="discussion-2.html"><a href="discussion-2.html#the-c.-elegans-cuticle-and-its-role-in-developmental-timing"><i class="fa fa-check"></i><b>5.2.1</b> The <em>C. elegans</em> cuticle and its role in developmental timing</a></li>
<li class="chapter" data-level="5.2.2" data-path="discussion-2.html"><a href="discussion-2.html#identifying-genetic-factors-that-contribute-to-variation-in-growth-and-body-size"><i class="fa fa-check"></i><b>5.2.2</b> Identifying genetic factors that contribute to variation in growth and body size</a></li>
<li class="chapter" data-level="5.2.3" data-path="discussion-2.html"><a href="discussion-2.html#phenotypic-plasticity-and-genotype-by-environment-interaction"><i class="fa fa-check"></i><b>5.2.3</b> Phenotypic plasticity and genotype-by-environment interaction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
<li class="chapter" data-level="7" data-path="appendix.html"><a href="appendix.html"><i class="fa fa-check"></i><b>7</b> Appendix</a></li>
<li><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#a.-growth-dynamics-of-c.-elegans-cuticle-mutants">A. Growth dynamics of <em>C. elegans</em> cuticle mutants</a>
<ul>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#description"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#methods"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#contributions-3"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li><a href="b.-food-allocation-during-c.-elegans-growth.html#b.-food-allocation-during-c.-elegans-growth">B. Food allocation during <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#methods-1"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#contributions-4"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Mechanisms of Larval Growth Control in <em>Caenorhabditis elegans</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="a-role-for-cuticle-stretch-in-developmental-growth-control" class="section level1" number="2">
<h1><span class="header-section-number">2</span> A role for cuticle stretch in developmental growth control</h1>
<div id="preface" class="section level2" number="2.1">
<h2><span class="header-section-number">2.1</span> Preface</h2>
<p>In the spring quarter of 2019, I decided to complete my final first-year rotation in the Andersen Lab. At the time, I was coming off a rotation with Dr. Niall Mangan who was involved in a relatively new collaboration with Dr. Erik Andersen as part of the NSF-Simons Center for Quantitative Biology. The general goal of the project was to study the interplay between organismal growth and environmental variables such as diet and temperature. Needing a little extra experimental manpower, Erik recruited me to the team. Little did I know this project would become the heart of my thesis. What ensued was a series of daunting, exhausting, and incredibly rewarding time course experiments. The following chapter highlights the results of just one of these experiments, submitted as a first-author manuscript to Cells and Development in January 2022 <span class="citation">[<a href="references.html#ref-Nyaanga2022-zd" role="doc-biblioref">86</a>]</span>.</p>
</div>
<div id="abstract-1" class="section level2" number="2.2">
<h2><span class="header-section-number">2.2</span> Abstract</h2>
<p>Growth control establishes organism size, requiring mechanisms to sense and adjust growth during development. Studies of single cells revealed that size homeostasis uses distinct control methods. In multicellular organisms, mechanisms that regulate single cell growth must integrate control across organs and tissues during development to generate adult size and shape. We leveraged the roundworm <em>Caenorhabditis elegans</em> as a scalable and tractable model to collect precise growth measurements of thousands of individuals, measure feeding behavior, and quantify changes in animal size and shape during a densely sampled developmental time course. As animals transitioned from one developmental stage to the next, we observed changes in body aspect ratio while body volume remained constant. Then, we modeled a physical mechanism by which constraints on cuticle stretch could cause changes in <em>C. elegans</em> body shape. The model-predicted shape changes are consistent with those observed in the data. Theoretically, cuticle stretch could be sensed by the animal to initiate larval-stage transitions, providing a means for physical constraints to influence developmental timing and growth rate in <em>C. elegans</em>.</p>
</div>
<div id="introduction-1" class="section level2" number="2.3">
<h2><span class="header-section-number">2.3</span> Introduction</h2>
<p>Growth is a complex process fundamental to development. Individual cells and whole animals must reach an appropriate size to remain competitive in their environment. A larger body size conveys many selective advantages to an organism, including increased success in predation, defense against predation, success in mating, or successful intraspecific as well as interspecific competition. Offsetting these advantages, larger organisms require more food resources to grow, take longer to develop, and produce fewer offspring <span class="citation">[<a href="references.html#ref-Hone2005-ex" role="doc-biblioref">87</a>]</span>. Therefore, it is critical for multicellular organisms to effectively coordinate the growth of both individual cells and the whole body. Additionally, growth at both of these scales must be coupled with developmental progression to ensure the proper timing of irreversible developmental events.</p>
<p>In recent years, efforts have focused on understanding how organisms control growth to achieve size homeostasis <span class="citation">[<a href="references.html#ref-Bjorklund2019-ox" role="doc-biblioref">88</a>–<a href="references.html#ref-Turner2012-ky" role="doc-biblioref">90</a>]</span>. Many of these studies are motivated by the decades-long debate about whether growth is linear or exponential; two separate models each having unique implications for size regulation. In a linear model with constant growth rate, smaller organisms must grow proportionally more than larger organisms to maintain size homeostasis. In this paradigm, organism size can be controlled simply by specifying growth duration. Subsequently, this method of growth control was named the “Timer” model <span class="citation">[<a href="references.html#ref-Donnan1983-uq" role="doc-biblioref">91</a>,<a href="references.html#ref-Wang2000-zi" role="doc-biblioref">92</a>]</span>. <em>Caulobacter vibrioides</em>, a bacterium whose cells divide after a certain amount of time, has been reported to follow a “Timer” growth model <span class="citation">[<a href="references.html#ref-Iyer-Biswas2014-ik" role="doc-biblioref">93</a>]</span>. Alternatively, organisms can instead monitor size and adjust duration of growth to reach an optimal size. This method of control is referred to as the “Sizer” model <span class="citation">[<a href="references.html#ref-Tzur2009-hl" role="doc-biblioref">94</a>–<a href="references.html#ref-Sveiczer1996-yu" role="doc-biblioref">96</a>]</span>. In an exponential model, growth rate is proportional to size. Here, a time-based control mechanism alone would fail to maintain size homeostasis because larger organisms would grow proportionally more during a specified period of time. This difference in growth requires a size-based control mechanism to ensure that growth is halted once a maximum size is reached. Although “Timer” and “Sizer” are the most often proposed size-control models, other models have been suggested. The “Adder” model proposes that a fixed volume is added to a cell or organism during growth <span class="citation">[<a href="references.html#ref-Campos2014-sl" role="doc-biblioref">97</a>,<a href="references.html#ref-Taheri-Araghi2015-sw" role="doc-biblioref">98</a>]</span>, whereas the “Folder” model specifies that an organism increases in volume by a fixed proportion in order to control growth <span class="citation">[<a href="references.html#ref-Towbin2021-vx" role="doc-biblioref">99</a>]</span>. It is not trivial to determine which model most accurately describes growth of individual cells or whole organisms because quantitative measurements of growth must be collected at high precision and throughput under tightly controlled experimental conditions. In unicellular organisms, the development of high-throughput experimental techniques in combination with theoretical models have advanced the understanding of size control <span class="citation">[<a href="references.html#ref-Soifer2016-tn" role="doc-biblioref">100</a>–<a href="references.html#ref-Cadart2018-dv" role="doc-biblioref">104</a>]</span>. Progress has been slower for multicellular organisms because cell growth within tissues and tissue growth within organisms often progress at different rates, suggesting that they are likely not regulated in the same ways <span class="citation">[<a href="references.html#ref-Uppaluri2015-wd" role="doc-biblioref">42</a>,<a href="references.html#ref-Moss-Taylor2019-bf" role="doc-biblioref">105</a>,<a href="references.html#ref-Spence2009-dk" role="doc-biblioref">106</a>]</span>.</p>
<p>The nematode <em>Caenorhabditis elegans</em> presents both a scalable and tractable multicellular animal model to study growth control. With an adult body length of approximately 1 mm, hundreds of thousands of individuals are easily cultured in controlled laboratory conditions <span class="citation">[<a href="references.html#ref-Wood1988-zk" role="doc-biblioref">30</a>]</span>. Moreover, <em>C. elegans</em> post-embryonic development is marked by several molts that provide clear developmental milestones <span class="citation">[<a href="references.html#ref-Page2007-im" role="doc-biblioref">64</a>]</span>. Each molt is initiated by a period of quiescence (lethargus) and terminated once the animal successfully sheds its collagen-rich outer cuticle (ecdysis) <span class="citation">[<a href="references.html#ref-Singh1978-xi" role="doc-biblioref">40</a>]</span>. Four molts separate the <em>C. elegans</em> life cycle into five distinct stages: four larval stages (L1-L4) and adult. The timing of these molts determines the completion of stage-specific development <span class="citation">[<a href="references.html#ref-Monsalve2011-vo" role="doc-biblioref">47</a>,<a href="references.html#ref-Zaidel-Bar2010-up" role="doc-biblioref">48</a>]</span> and underscores the importance of growth regulation during <em>C. elegans</em> larval development.</p>
<p>A full description of an organism’s development includes the assessment of how growth and body size are regulated. Initial studies of <em>C. elegans</em> development described whole-organism growth as a sigmoidal curve characterized by continuous larval growth in length that reaches saturation in adulthood <span class="citation">[<a href="references.html#ref-Byerly1976-xt" role="doc-biblioref">107</a>]</span>. These early studies hypothesized that molt events had little effect on continuous growth as the <em>C. elegans</em> cuticle allowed for stretch during larval stages. Later work determined that larval progression was not continuous but rather piecewise in nature <span class="citation">[<a href="references.html#ref-Knight2002-fb" role="doc-biblioref">108</a>]</span>. This study showed that <em>C. elegans</em> volumetric growth rate increased from stage to stage such that L1 animals had the slowest rate of growth and L4 animals had the fastest. This finding suggests that <em>C. elegans</em> have a mechanism for regulating growth rate, potentially at each molt. Next, researchers using single-animal imaging strategies observed that animals did not advance to the next developmental stage until a critical volume was reached <span class="citation">[<a href="references.html#ref-Uppaluri2015-wd" role="doc-biblioref">42</a>]</span>. This finding suggests that <em>C. elegans</em> growth follows a “Sizer” model with each molt decision controlled by a volume threshold and further implies that individual cells are able to communicate information about body size to precisely regulate growth. Most recently, live imaging and characterization of body volume heterogeneity revealed that with respect to the start of a larval stage, <em>C. elegans</em> volume fold change within a stage is nearly invariant thereby preventing rapid divergence in volume between fast- and slow-growing animals <span class="citation">[<a href="references.html#ref-Towbin2021-vx" role="doc-biblioref">99</a>]</span>. A mechanism that maintains a constant volume fold change within each larval stage relies on the coupling between growth rate and developmental timing. Notably, such coupling is consistent with recent observations of temporal scaling in <em>C. elegans</em> development where despite inter-individual variability in the absolute duration of a larval stage, relative timing of a stage is similar <span class="citation">[<a href="references.html#ref-Faerberg2021-ui" role="doc-biblioref">109</a>,<a href="references.html#ref-Filina2020-ia" role="doc-biblioref">110</a>]</span>.</p>
<p>To understand <em>C. elegans</em> growth control at the whole-organism level, we used a combination of quantitative measurements and mathematical modeling. We performed a high-resolution longitudinal study of <em>C. elegans</em> larval progression and captured high-precision details about animal length, width, volume, and feeding dynamics. By investigating <em>C. elegans</em> feeding and growth in tandem for thousands of individual animals, we found decreases in feeding behavior associated with each larval transition that were also correlated in time with changes in growth rate. We used our large-scale measurements of body shape to further analyze the periods of time surrounding each larval transition. At each molt, we observed simultaneous increases in length, decreases in width, and maintenance of volume. Based on the physical characteristics of the cuticle, we propose a “Stretcher” mechanism whereby physical constraints on cuticle stretch influence body shape. We find the Stretcher model-predicted shape changes are consistent with observed data. Animals may be able to detect when the cuticle reaches its maximum capacity for stretch providing a signal to initiate larval-stage transitions.</p>
</div>
<div id="materials-and-methods" class="section level2" number="2.4">
<h2><span class="header-section-number">2.4</span> Materials and methods</h2>
<div id="worm-culture" class="section level3" number="2.4.1">
<h3><span class="header-section-number">2.4.1</span> Worm culture</h3>
<p>The canonical laboratory strain N2 was obtained from the <em>C. elegans</em> Natural Diversity Resource <span class="citation">[<a href="references.html#ref-Cook2016-lh" role="doc-biblioref">111</a>]</span>. Animals were cultured at <span class="math inline">\(20^{\circ}C\)</span> on 6 cm plates of modified nematode growth media (NGMA), which contained 1% agar and 0.7% agarose seeded with E. coli OP50 bacteria <span class="citation">[<a href="references.html#ref-Andersen2014-zv" role="doc-biblioref">77</a>]</span>.</p>
</div>
<div id="bacterial-food" class="section level3" number="2.4.2">
<h3><span class="header-section-number">2.4.2</span> Bacterial food</h3>
<p><em>E. coli</em> HB101 bacteria were prepared from cultures grown for 15 hours in Superbroth and then pelleted by centrifugation. HB101 bacteria were diluted to OD100 in K medium (51 mM NaCl, 32 mM KCl, 3 mM <span class="math inline">\(CaCl_2\)</span>, and 3 mM MgSO4 in distilled water) and stored at <span class="math inline">\(-80^{\circ}C\)</span>. Bacteria were thawed and fed to animals at a concentration sufficient to sustain population growth from hatching to adulthood (OD20).</p>
</div>
<div id="growth-of-the-animals" class="section level3" number="2.4.3">
<h3><span class="header-section-number">2.4.3</span> Growth of the animals</h3>
<p>Populations of animals were propagated on NGMA plates for two generations without starvation. In the third generation, gravid adults were bleach-synchronized <span class="citation">[<a href="references.html#ref-Stiernagle2006-ch" role="doc-biblioref">112</a>]</span>. Embryos were resuspended in K medium, aliquoted into a 500 mL flask at a concentration of one embryo per <span class="math inline">\(\mu L\)</span>, and allowed to hatch overnight. The following day, arrested L1s were fed HB101 bacteria at a final concentration of OD20 in a final flask volume of 100 mL K medium and HB101 food. Animals were grown for three days at <span class="math inline">\(20^{\circ}C\)</span> with constant shaking. Following these three days, adult animals were bleach-synchronized once more and embryos were aliquoted to seven replicate 500 mL flasks at a concentration of one embryo per <span class="math inline">\(\mu L\)</span> in 100 mL. The following morning, six flasks were fed HB101 bacterial food at a final concentration of OD20 in a final flask volume of 100 mL K medium and HB101 food. Two additional flasks were included to control for L1 animal size and possible clumping of bacterial food: one flask contained L1 larvae but did not have food added and one flask contained no larvae but the same concentration of HB101 bacteria as the six flasks containing L1 larvae. All replicate flasks were kept in an incubator at <span class="math inline">\(20^{\circ}C\)</span> with shaking for the duration of the experiment. A small temperature gradient of <span class="math inline">\(1.25^{\circ}C\)</span> was recorded in the shaking incubator with the highest temperature readings on the right side and lowest temperature readings on the left side. This slight variation in temperature contributed to variation in developmental rate among replicates based on position within the incubator (replicates were placed in numerical order with replicate 1 positioned on the far right side of the incubator).</p>
</div>
<div id="high-throughput-measurements-of-body-size-and-fluorescence" class="section level3" number="2.4.4">
<h3><span class="header-section-number">2.4.4</span> High-throughput measurements of body size and fluorescence</h3>
<p>Flasks were sampled each hour beginning one hour after feeding and continuing for 72 consecutive hours. At each hour, <span class="math inline">\(500\ \mu L\)</span> was removed from each flask and transferred to a well of a deep 96-well plate. Each flask was sampled at each time point. Fluorescent polychromatic beads (Polysciences, 19507-5) with a <span class="math inline">\(0.5\ \mu m\)</span> particle size were added to each well at a final concentration of <span class="math inline">\(3.64x10^8\)</span> beads/mL and incubated at <span class="math inline">\(20^{\circ}C\)</span> for 10 minutes with shaking. Following the bead incubation, <span class="math inline">\(30 \mu L\)</span> from each well of the deep 96-well plate was aliquoted to a 96-well microtiter plate. The process was repeated 11 times to 11 separate wells of the same microtiter plate with pipetting to mix the well contents from the deep 96-well plate. Animals were then treated with sodium azide at a final concentration of 50 mM to paralyze and prevent defecation of the ingested beads. The 96-well plate was imaged with an ImageXpress Nano (Molecular Devices, SanJose, CA) using both 2x (Nikon MRD00025) and 10x (Nikon MRH00101) objectives. The ImageXpress Nano acquires brightfield images using a 4.7 megaPixel CMOS camera. Images are stored in 16-bit TIFF format. Finally, animals were scored using a large-particle flow cytometer (COPAS BIOSORT, Union Biometrica, Holliston MA). The COPAS BIOSORT sheath flow rate was kept at a constant <span class="math inline">\(10.3 ± 0.1\)</span> mL per minute to reduce variability in length measurements.</p>
</div>
<div id="image-processing" class="section level3" number="2.4.5">
<h3><span class="header-section-number">2.4.5</span> Image processing</h3>
<p>Manual measurements of animal size were performed using the free Java image-processing program ImageJ <span class="citation">[<a href="references.html#ref-Abramoff2004-my" role="doc-biblioref">113</a>]</span>. Well images for the six replicate flasks, excluding controls, were loaded into ImageJ software. Length was measured from head to tail, and width was measured at the widest point of the animal. Five animals were measured per well across thirty total wells for each hour. Measurements were repeated for all 72 time points in the assay. Body length and width were used to estimate cross-sectional area <span class="math inline">\((length*width)\)</span>. This metric was used to describe animal area for the extent of the text. Volume was calculated from body length and width by approximating the animal as a cylinder. Pixels were converted to <span class="math inline">\(\mu m\)</span> using a conversion factor of <span class="math inline">\(3.2937\ \mu m/pixels\)</span>.</p>
</div>
<div id="data-processing" class="section level3" number="2.4.6">
<h3><span class="header-section-number">2.4.6</span> Data processing</h3>
<p>The COPAS BIOSORT was used to collect measurements of animal length (TOF), optical extinction (EXT), and fluorescence for every animal in each well. These traits measure properties of nematode development and, as such, increase as animals progress to adulthood <span class="citation">[<a href="references.html#ref-Andersen2015-uj" role="doc-biblioref">114</a>]</span>. Optical extinction measurements correspond to the amount of light absorbed over the full length of an animal as it passes through the instrument. An additional measurement (norm.EXT) can be calculated by normalizing optical extinction by length. The raw data collected were imported and processed using the <em>easysorter</em> R package <span class="citation">[<a href="references.html#ref-Shimko2014-eb" role="doc-biblioref">78</a>]</span>.</p>
<p>The COPAS BIOSORT data were analyzed further using Gaussian finite mixture modeling as implemented in the mclust R package <span class="citation">[<a href="references.html#ref-Scrucca2016-by" role="doc-biblioref">115</a>]</span>. These probabilistic models assume that data are generated from a mixture of multivariate normal distributions and, therefore, can be used to classify unstructured data into meaningful groups. Specifically, the <em>mclust</em> package fits a mixture model to data and selects the optimal number of clusters using an expectation-maximization algorithm and Bayes Information Criteria. For model-based clustering, log transformed animal length (logTOF) and log transformed optical extinction (logEXT) were used as inputs for the <em>Mclust</em> function. Data from each hour of the experiment was analyzed by replicate and clusters that did not appear to include majority animal objects were identified and removed as described previously <span class="citation">[<a href="references.html#ref-Smith2009-lx" role="doc-biblioref">116</a>]</span>. This processing removed non-animal objects such as bacterial clumps, shed cuticles, and next generation larval animals from the time-course data.</p>
<p>We used a numpy polyfit regression of well-median data from the COPAS BIOSORT and image measurements to convert TOF and norm.EXT data to microns. Only the unit-corrected BIOSORT data were used for further analysis.</p>
</div>
<div id="molt-analysis" class="section level3" number="2.4.7">
<h3><span class="header-section-number">2.4.7</span> Molt analysis</h3>
<p>Fluorescence data obtained from the COPAS BIOSORT was used as a proxy for feeding behavior to distinguish animals in a molt from growing animals. First, fluorescence was normalized by EXT to account for the ability of larger animals to consume more food and beads. Next, an analysis of variance statistical model was fit to the fluorescence data normalized by EXT to determine the amount of variance contributed by replicate and well (Table S1). A local kernel regression smoothing method was then applied to the residuals of the variance analysis using the <em>lokern</em> R package <span class="citation">[<a href="references.html#ref-Hermann2016-qs" role="doc-biblioref">117</a>]</span>. Residuals were used to address only the differences over time and ignore minor variation among replicates and wells. The local minima of the regression function were found by solving for where the first derivative of this function equaled zero. The time associated with each local minimum was used to represent the timing of each molt. Molts occurred at 14, 25, 36, and 48 hours.</p>
<p>To identify periods of time that contained a majority of growing animals, the inflection points of the regression function were calculated by solving for where the second derivative of the function equaled zero. Time points between inflection points that did not contain a local fluorescence minimum were considered as growth periods. These hours were 1-13, 17-22, 27-32, and 39-45 corresponding to L1, L2, L3, and L4 growth periods.</p>
<p>Each molt is initiated when animals enter lethargus: a behavioral state where animals cease active feeding. To classify individual animals as in a molt or growing, we set a quiescence threshold using fluorescence measurements at each local minimum. The fluorescence measurement at each local minimum was as follows: 0.07, 0.06, 0.06, 0.06. The average of these measurements (0.06) was used as the fluorescence threshold signifying quiescent behavior. Any individual animals that fell below this threshold fluorescence value were designated as in a molt and animals above this threshold value were classified as growing.</p>
</div>
<div id="comparison-of-model-fits" class="section level3" number="2.4.8">
<h3><span class="header-section-number">2.4.8</span> Comparison of model fits</h3>
<p>To determine the volume growth model, we fit linear, exponential, and cubic functions to data designated as growth periods for each larval stage. Both linear and nonlinear functions were fitted using least-squares regression. Akaike’s information criterion (AIC) <span class="citation">[<a href="references.html#ref-Sakamoto1986-er" role="doc-biblioref">118</a>]</span> and Bayesian information criterion (BIC) <span class="citation">[<a href="references.html#ref-Schwarz1978-mc" role="doc-biblioref">119</a>]</span> were goodness of fit criteria used to evaluate candidate models. To assess the strength of evidence for each candidate model, we identified the model with the smallest AIC/BIC value and assessed the difference between this value and the AIC/BIC of the other two models. The magnitude of the difference was used to determine the level of support for each candidate model as previously described <span class="citation">[<a href="references.html#ref-Kass1995-si" role="doc-biblioref">120</a>,<a href="references.html#ref-Burnham2007-ye" role="doc-biblioref">121</a>]</span>. All model fits and analysis were performed using the <em>stats</em> R package.</p>
</div>
<div id="stretcher-model-analysis" class="section level3" number="2.4.9">
<h3><span class="header-section-number">2.4.9</span> Stretcher model analysis</h3>
<p>To analyze shape dynamics, length and width data from growth time periods were extracted from the full COPAS BIOSORT population data and analyzed from each replicate separately to avoid issues with replicate variability. For replicate 2, the hours defining growth periods were 1-13, 16.37-22.39, and 26.93-32.96; corresponding to L1, L2, and L3. Hours defining larval stages were rounded as data was collected at exact hour increments. The L4 stage was excluded from the analysis because of the high variability within the population. We applied a local kernel regression, <em>lokern</em> R package <span class="citation">[<a href="references.html#ref-Hermann2016-qs" role="doc-biblioref">117</a>]</span>, to smooth the population dynamics of length and width. To calculate mean and standard deviation, the smoothed population measurements were bootstrapped using 2,000 samples with replacement. To determine cuticle properties throughout larval stages, we calculated the mean ratio of derivatives of regression width and length.</p>
</div>
</div>
<div id="results" class="section level2" number="2.5">
<h2><span class="header-section-number">2.5</span> Results</h2>
<div id="quantitative-measurements-of-c.-elegans-growth" class="section level3" number="2.5.1">
<h3><span class="header-section-number">2.5.1</span> Quantitative measurements of <em>C. elegans</em> growth</h3>
<p>We have optimized a quantitative growth assay that reliably measures small changes in <em>C. elegans</em> body size throughout development (<strong>Figure 2-1</strong>). Our method provides both high-throughput and high-precision assessment of developmental growth. In brief, populations of 100,000 animals were cultured in flasks. We cultured six replicate populations of <em>C. elegans</em> for a total of 600,000 synchronized and growing animals. Every hour after feeding, a sample of the population from each flask (~300 animals/flask) was collected to measure animal length, width, and feeding rate. Feeding rate, examined using fluorescent microspheres, and body size were measured using the COPAS BIOSORT (Union Biometrica). Then, the ImageXpress system (Molecular Devices) was used to collect images of sampled animals. This platform enabled further analysis of life stage and body size, contributing added precision to our measurements.</p>
<p><img src="img/ch2/fig2-1.png" width="90%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 2-1. An overview of the quantitative growth assay.</strong> Synchronized animals were cultured in flasks where six flasks contained replicate populations of nematodes, one flask had a population of unfed animals, and one flask only contained bacterial food. At each hour of the experiment, all eight flasks were sampled. In step 1, animals were transferred from each flask to a single well of a 96-well microtiter plate. In step 2, fluorescent beads were added to each well. Following a 10-minute incubation period, animals from each well of the deep-well plate were transferred to several wells of a 96-well microtiter plate for step 3. In step 4, animals in each well of the microtiter plate were imaged. In step 5, the same animals were measured using the COPAS BIOSORT. This process was repeated every hour after feeding for 72 consecutive hours (see Methods). Schematic of the experimental workflow was created with BioRender.com.</p>
<p>The raw data from the quantitative growth assay provides measurements of body size and feeding behavior, which are traits related to animal growth. Two measurements of body size were collected from raw data taken from the COPAS BIOSORT: time of flight (TOF) and optical extinction (EXT) (Figure S1). Time of flight is a measurement of body length, and optical extinction corresponds to optical density, a measurement influenced by body length, thickness, and composition <span class="citation">[<a href="references.html#ref-Andersen2015-uj" role="doc-biblioref">114</a>,<a href="references.html#ref-Pulak2006-lc" role="doc-biblioref">122</a>]</span>. We investigated whether optical extinction could be correlated to a different measure of body size using the collection of manual size measurements from images (see Methods). We calculated the median length, width, area, and volume of animals in a subset of imaged wells from each hour of the experiment. We then compared these values to well median statistics from the processed COPAS BIOSORT data. We found a strong correlation between manual measurements of animal length from the image analysis and TOF measurements from the COPAS BIOSORT (Figure S2-2). We also observed an equally strong correlation between manual measurements of animal area and EXT as well as animal width and EXT normalized by body length (norm.EXT). We then approximated animal volume using measurements from the COPAS BIOSORT by using a cylindrical approximation for <em>C. elegans</em> shape (see Methods). This result expanded the number of body size parameters that we were able to assess using the COPAS BIOSORT data, allowing us to investigate growth dynamics in length, width, and volume (<strong>Figure 2-1 A-C</strong>). To disentangle nematode objects from non-animal objects (bacteria clumps, detritus, shed cuticles), we employed model-based clustering to remove unwanted objects and better examine growth of animals (Figure S2-3). Lastly, we converted COPAS BIOSORT measurements into microns (see Methods).</p>
<p><img src="img/ch2/fig2-2.png" width="80%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 2-2. Quantitative measurements of animal size.</strong> COPAS BIOSORT data of animal length (A), width (B), and volume (C) after the removal of non-animal objects using model-based clustering methods (see Methods).</p>
<p>We report body length, width, and volume of animals at each hour of development from L1 to adult (Figure S2-1 and <strong>Figure 2-2</strong>). Historically, growth of <em>C. elegans</em> has been shown as a sigmoidal curve where exponential growth during larval stages reaches a maximum rate in adulthood <span class="citation">[<a href="references.html#ref-Byerly1976-xt" role="doc-biblioref">107</a>]</span>. More recently, researchers have identified that growth curves are discontinuous during development punctuated by larval transitions <span class="citation">[<a href="references.html#ref-Uppaluri2015-wd" role="doc-biblioref">42</a>,<a href="references.html#ref-Knight2002-fb" role="doc-biblioref">108</a>]</span>. Using our quantitative growth assay, we captured these small-scale discontinuities in larval growth rate as well as an apparent growth maximum during early adulthood. We noticed that all size variables (length, width, and volume) displayed these dynamics. Objects identified as animals appear to grow in size. However, in particular time windows during development, growth dynamics visibly shift, producing discontinuities in animal growth rate. With these data, we were able to further investigate <em>C. elegans</em> growth and size control.</p>
</div>
<div id="fluorescence-provides-a-quantitative-measurement-of-animal-feeding-behavior-and-developmental-progression" class="section level3" number="2.5.2">
<h3><span class="header-section-number">2.5.2</span> Fluorescence <strong>provides a quantitative measurement of animal feeding behavior and developmental progression</strong></h3>
<p>In addition to body size and shape, the raw data from the quantitative growth assay described above measured fluorescence of each animal object. To readily assess the thousands of measurements acquired at each hour, we generated summary statistics of median well measurements (Table S2-1). With these summarized data, we investigated the relationship between feeding behavior and developmental stage. It is well established that temporary suspensions of <em>C. elegans</em> feeding occur during each molt <span class="citation">[<a href="references.html#ref-Cassada1975-zx" role="doc-biblioref">43</a>,<a href="references.html#ref-Byerly1976-xt" role="doc-biblioref">107</a>]</span>. As such, active feeding is frequently used to distinguish growing animals from individuals in a molt. We quantified feeding behavior by exposing animals to fluorescent beads the approximate size of bacteria and measuring fluorescence of animals <span class="citation">[<a href="references.html#ref-Nika2016-kc" role="doc-biblioref">123</a>]</span>. Because larger animals are able to consume more food and therefore contain more ingested food, we normalized fluorescence by animal area to account for increases in body size (Figure S2-4). The resulting fluorescence data showed a dynamic pattern (<strong>Figure 2-3A</strong>). At approximately 15 hours, fluorescence steadily increased to a peak before decreasing back to initial levels at approximately hour 27. This pattern, repeated three additional times, established clear time windows of local minimal fluorescence. These local minima represent periods of time where a large proportion of the population had reduced or ceased feeding and therefore suggests time windows where a majority of animals were likely not feeding because they were in a molt. We used a local kernel regression method to estimate a smooth curve and calculate the derivative to identify the time associated with each local minimum (see Methods). We then assessed images collected from the growth assay and demonstrated that periods of decreased feeding are concurrent with the presence of shed cuticles, supporting that animals are undergoing a molt during these periods of time (Figure S2-5). When we overlaid the timing of each local minimum on the population size data, we were able to outline the start and end of each larval stage (<strong>Figure 2-3 B-D</strong>). Notably, local minima occurred approximately every ten hours, consistent with well established observations of molt timing <span class="citation">[<a href="references.html#ref-Byerly1976-xt" role="doc-biblioref">107</a>]</span>. Furthermore, we observed a clear relationship between changes in feeding behavior and growth dynamics where decreases in feeding occurred simultaneously with discontinuous growth in length, width, and volume.</p>
<p><img src="img/ch2/fig2-3.png" width="90%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 2-3. Fluorescence dynamics outline larval stages.</strong> (A) Median normalized red fluorescence (y-axis) over time (x-axis) is shown. The blue line represents the kernel regression fit to the data. The red vertical lines correspond to the local minima of the regression and represent the transition between larval stages. Median length (B), median width (C), and median log volume (D) are shown with larval-stage transitions as well. Upper and lower bounds of the box plots correspond to the first and third quartiles. The upper and lower whiskers extend to 1.5 times the value of the interquartile range.</p>
</div>
<div id="changes-in-c.-elegans-body-shape-occur-at-larval-stage-transitions" class="section level3" number="2.5.3">
<h3><span class="header-section-number">2.5.3</span> Changes in <em>C. elegans</em> body shape occur at larval-stage transitions</h3>
<p>Adult body size is ultimately determined by the coordination of developmental progression and rate of growth. To understand how <em>C. elegans</em> achieve final size, we must first examine how <em>C. elegans</em> grow. Quantitative studies of <em>C. elegans</em> growth frequently assess changes in length or volume over time; however, to fully characterize changes associated with growth, it is also important to consider the dynamics of width. Two general models were proposed for <em>C. elegans</em> growth in volume: linear and exponential <span class="citation">[<a href="references.html#ref-Uppaluri2015-wd" role="doc-biblioref">42</a>,<a href="references.html#ref-Byerly1976-xt" role="doc-biblioref">107</a>,<a href="references.html#ref-Knight2002-fb" role="doc-biblioref">108</a>]</span>. Notably, these volume growth models require different dynamics in length and width. To achieve linear volume growth, length and width must increase at precise sublinear rates that together result in a linear increase in volume. If animal length and width increased at a constant linear rate, then volume would increase at a cubic rate. Alternatively, if both length and width grew exponentially, then volume would fit an exponential model. We sought to identify which model best described <em>C. elegans</em> growth behavior but were unable to consistently distinguish between linear, exponential, and cubic models using statistical information criterion because of the similarity in the shapes of the growth curves (Figure S2-6 and Table S2-2). This result is not surprising because computational simulations have shown that increases in experimental noise, above 2% added noise, limit the correct identification of growth models <span class="citation">[<a href="references.html#ref-Vuaridel-Thurre2020-ug" role="doc-biblioref">124</a>]</span>.</p>
<p>Growth has important implications for how animals regulate size. Size homeostasis requires that growth rate and developmental rate are coordinated. <em>C. elegans</em> reach a similar volume at each larval transition despite significant variation in individual growth rates <span class="citation">[<a href="references.html#ref-Uppaluri2015-wd" role="doc-biblioref">42</a>]</span>. Because individuals in a population maintain similar sizes despite differences in growth rate, a control mechanism to regulate developmental progression must exist. Early work proposed a size-based growth control model in <em>C. elegans</em> <span class="citation">[<a href="references.html#ref-Uppaluri2015-wd" role="doc-biblioref">42</a>]</span>, although recent work suggests that size homeostasis is achieved through a folder mechanism where growth rate and development are coupled <span class="citation">[<a href="references.html#ref-Towbin2021-vx" role="doc-biblioref">99</a>]</span>. To assess changes in body size and shape during a larval transition, we examined the dynamics of animal length, width, and volume in the hours before, during, and after each molt. We find that for each shape variable, larger animals enter molt first (<strong>Figure 2-4</strong>). We also observe differences in the distributions of lengths during a larval transition compared to widths and volumes. Measurements of animal width and volume remain unimodal throughout a molt, but length does not. As larger animals begin to exit the molt, an increase in body length occurs that leads to the appearance of bimodality of lengths across the population. Importantly, volume remains constant while length increases and width decreases, indicating a change in body geometry not size. Notably, the length increase occurs simultaneously with a decrease in width across the population (<strong>Figure 2-4D</strong>). These changes in the physical dimensions at each larval transition suggests that body shape, in addition to size, is involved in the control of <em>C. elegans</em> growth.</p>
<p><img src="img/ch2/fig2-4.png" width="827"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 2-4. Changes in body shape occur during larval-stage transitions.</strong> Population density curves of length (A), width (B) and volume (C) for the hours surrounding the L3 - L4 larval transition (red horizontal line at 36 hours corresponds to the molt). Each distribution was divided into five quantiles. The percentage of quiescent animals present within each quantile was calculated (see Methods), and each quantile was colored to reflect this percentage. In all shape variables, quantiles that contain the largest animals displayed an increase in quiescence earlier than quantiles that contain the smallest animals. These dynamics were consistent across all larval-stage transitions (Figure S2-7). (D) Median width vs. median length for experimental hours 1 - 55. Red indicates measurements that fall above the quiescence threshold (see Methods). Simultaneous changes in length and width occur during periods of increased quiescence.</p>
</div>
<div id="modeling-c.-elegans-cuticle-stretch-dynamics" class="section level3" number="2.5.4">
<h3><span class="header-section-number">2.5.4</span> Modeling <em>C. elegans</em> cuticle stretch dynamics</h3>
<div id="sensing-of-cuticle-stretch-as-a-trigger-for-larval-stage-transitions" class="section level4" number="2.5.4.1">
<h4><span class="header-section-number">2.5.4.1</span> Sensing of cuticle stretch as a trigger for larval-stage transitions</h4>
<p>Previous studies theorized that the internal mechanism for sensing body size and triggering molts in <em>C. elegans</em> is driven, in part, by the properties of the collagen-rich cuticle <span class="citation">[<a href="references.html#ref-Uppaluri2015-wd" role="doc-biblioref">42</a>,<a href="references.html#ref-Towbin2021-vx" role="doc-biblioref">99</a>]</span>. Many cuticle collagen mutations cause morphological defects in nematode shape some of which cause animals to be shorter but do not impact animal width, implying that the cuticle affects length and width independently <span class="citation">[<a href="references.html#ref-Brenner1974-kr" role="doc-biblioref">32</a>]</span>. The <em>C. elegans</em> cuticle does not grow through the addition of new material, but rather stretches to accommodate increases in animal body size. Cuticle stretch is likely limited by the material properties of the cuticle. The <em>C. elegans</em> cuticle is primarily made of cross-linked collagens organized into lateral ridges and circumferential bands <span class="citation">[<a href="references.html#ref-Page2007-im" role="doc-biblioref">64</a>]</span>. Commonly found in many biological systems, collagen-based materials are fairly flexible under low stress conditions. However, as stress increases, collagen fibrils may become elongated and orient in the load bearing direction leading to a decrease in elasticity <span class="citation">[<a href="references.html#ref-Holzapfel2017-ds" role="doc-biblioref">125</a>]</span>. Previous work using atomic force microscopy revealed a high level of biomechanical stiffness at the circumferential bands <span class="citation">[<a href="references.html#ref-Essmann2017-zv" role="doc-biblioref">126</a>]</span>, leading others to speculate that mechanical strain on these structures is likely adjusted as internal body pressure changes <span class="citation">[<a href="references.html#ref-Dodd2018-mz" role="doc-biblioref">127</a>]</span>. Additionally, in <em>nekl-3(sv3)</em> molting mutants, the cuticle is not properly removed from the middle part of their body, leaving the free head and tail to grow normally while the encased middle is constricted by the old cuticle to pre-molt dimensions <span class="citation">[<a href="references.html#ref-Yochem2015-bs" role="doc-biblioref">128</a>]</span>. Given this body restriction, we speculate that the old cuticle stretches beyond its tolerance, becomes stiff, and constricts the center of the nematode relative to the growing head and tail size. We hypothesize that <em>C. elegans</em> may be able to sense when the cuticle becomes restrictive. If so, the reduction of elasticity or “stretchiness” of the cuticle can act as one signal, among many, used to determine when to initiate a molt.</p>
<p>To understand the impact of cuticle elasticity on body shape, we developed a “Stretcher” model. We propose that the nematode passes through three distinct regimes related to cuticle stretch: linear stretch dynamics, non-linear stretch dynamics, and larval stage transition (<strong>Figure 2-5</strong>). These regimes arise naturally from the following biologically supported assumptions. The cuticle structure is anisotropic, possibly leading to distinct properties in the length and width directions <span class="citation">[<a href="references.html#ref-Petzold2011-vi" role="doc-biblioref">129</a>,<a href="references.html#ref-Cox1981-ua" role="doc-biblioref">130</a>]</span>. We approximated the cuticle as a hollow cylinder of negligible thickness filled by the body of the nematode. Growth was modelled as internal pressure evenly applied to the cuticle in all directions. An anisotropic, elastic cuticle responds differently to pressure from growth during linear stretch, nonlinear stretch, and post-molt relaxation, leading to differences in shape during each stage of development.</p>
<p><img src="img/ch2/fig2-5.png" width="85%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 2-5. Cuticle stretch dynamics guide larval-stage transitions.</strong> The Stretcher model describes each larval stage as a cycle. Nematodes are modeled as a cylindrical object with a thin cuticle epidermis. (Box 1) Linear Stretch Dynamics: uniform growth pressure stretches the cuticle linearly in both length and width. (Box 2) Nonlinear Stretch Dynamics: the cuticle has reached a stretch threshold in length, and under uniform growth pressure the length stretches less (sub-linear) and width stretches linearly. (Box 3) Larval Stage Transition: a new cuticle is formed and the old cuticle is shed (ecdysis), removing constraints in length. The nematode body “relaxes” in length, causing an increase in length, a decrease in width, and constant volume.</p>
<p>In the linear stretch regime (<strong>Figure 2-5</strong>), the cuticle would be linearly elastic in both the length and width directions, stretching proportionally to the pressure exerted on the cuticle. Previous work found evidence for a linearly elastic cuticle <span class="citation">[<a href="references.html#ref-Park2007-rm" role="doc-biblioref">131</a>,<a href="references.html#ref-Gilpin2015-hq" role="doc-biblioref">132</a>]</span> in animals expanded in a negative external pressure environment or after positive force was applied to the cuticle. Gilpin <em>et al.</em> have found evidence of linear elasticity in the nematode body. We conjecture that this linear elasticity is caused by the constraints applied by the cuticle <span class="citation">[<a href="references.html#ref-Park2007-rm" role="doc-biblioref">131</a>,<a href="references.html#ref-Gilpin2015-hq" role="doc-biblioref">132</a>]</span>. A linearly elastic cuticle will have <span class="math inline">\(\Delta L\)</span> stretch in the length direction and W stretch in the width direction, each related to growth-applied pressure <span class="math inline">\(\Delta p\)</span> by</p>
<p><span class="math display">\[
\Delta L = a_L\Delta p
\]</span></p>
<p><span class="math display">\[
\Delta W = a_W\Delta p.
\]</span></p>
<p>The “stretch coefficients” in length, <span class="math inline">\(a_L\)</span>, and width, <span class="math inline">\(a_W\)</span>, measure the stiffness of the cuticle. Smaller values correspond to a stiffer material, which is less able to stretch in response to pressure. The stretch coefficients are constant in the linearly elastic regime and are determined by geometric constants and material properties. The ratio of the change in length (Eq. 1) and width (Eq. 2) produces a pressure-independent relationship that depends only on the ratio of the geometric and material properties, which can be verified using measurements of length and width (Figure 2-3). During the linearly elastic regime, the ratio of growth in width to growth in length is constant throughout a larval stage where the cuticle properties are fixed as in</p>
<p><span class="math display">\[
\Delta W/ \Delta L = a_W/a_L = constant.
\]</span></p>
<p>In the non-linear stretch regime (<strong>Figure 2-5</strong>), growth continues to apply pressure to the cuticle uniformly in all directions. As observed in <em>nekl-3(sv3)</em> mutants, the cuticle can restrict body growth <span class="citation">[<a href="references.html#ref-Yochem2015-bs" role="doc-biblioref">128</a>]</span>. Once outside of the linearly elastic regime, the cuticle would hardly stretch, even under large forces. We hypothesize that this shift from linear to nonlinear regimes can provide a mechanism for size-sensing and cues the larval-stage transition (<strong>Figure 2-5</strong>). In principle, this transition could occur in either the width or length directions. For simplicity we illustrate a transition from linear to non-linear stretch in the length direction while linear stretch in the width direction is maintained. In the nonlinear regime, the stretch in the length direction in response to pressure becomes</p>
<p><span class="math display">\[
\Delta L \approx \ \tilde{a_L}(p)\Delta p.
\]</span></p>
<p>The nonlinear “stretch coefficient,” <span class="math inline">\(\tilde{a_L}(p)\)</span>, is no longer constant and decreases with increasing pressure. It is smaller than <span class="math inline">\(a_L\)</span> because the cuticle has become less elastic than in the linear regime. If the length-direction enters the nonlinear regime and has reduced stretch response, while width has the same constant stretch response then, we expect the <span class="math inline">\(\Delta W/\Delta L\)</span> ratio to increase</p>
<p><span class="math display">\[
\Delta W/ \Delta L|_{non-linear} = a_W/\tilde{a_L}(p) &gt; a_W/a_L = \Delta W/ \Delta L|_{linear}.
\]</span></p>
<p>During the larval-stage transition (Figure 2-5), a new, larger cuticle is formed beneath the old cuticle that is shed during ecdysis. Because the old cuticle constrained growth in length, we predict a rapid increase in the length direction when the constraint is removed. Animal volume is conserved as growth does not occur during this process. Therefore, the relaxation in length is accompanied by a corresponding decrease in width.</p>
</div>
<div id="discontinuities-in-animal-growth-rate-driven-by-limits-of-cuticle-stretch" class="section level4" number="2.5.4.2">
<h4><span class="header-section-number">2.5.4.2</span> Discontinuities in animal growth rate driven by limits of cuticle stretch</h4>
<p>To test whether the shape dynamics predicted by the Stretcher model are supported by data, we analyzed the relationship between measured nematode length and width over developmental time. Although the shape relaxation for an individual animal is expected to happen at a much shorter time scale than data collection (seconds to minutes), the measured shape change within the population occurs on a larger time scale (several hours). As such, we can observe the sudden shape change at transitions by assessing changes in length and width of the population (<strong>Figure 2-6</strong>). Doing so, we detected all three regimes predicted by the Stretcher model in the COPAS BIOSORT data: linear stretch, non-linear stretch, and relaxation (<strong>Figure 2-6A</strong>). In all larval stages, we observed an approximately constant instantaneous <span class="math inline">\(\Delta W/\Delta L\)</span> ratio, consistent with a linear stretch regime (<strong>Figure 2-6B</strong> and <strong>Figure 2-6C</strong>). We also detected a large slope decrease during the L1 stage, which could correspond in time to the metabolic decision for entry into the dauer stage <span class="citation">[<a href="references.html#ref-Golden1984-hp" role="doc-biblioref">133</a>]</span> or divisions of the seam cells <span class="citation">[<a href="references.html#ref-Sulston1977-ie" role="doc-biblioref">33</a>]</span>. Following the linear stretch regime, we noted a simultaneous increase in length and decrease in width at the transition between larval stages, consistent with a length threshold in the Stretcher model (<strong>Figure 2-5D</strong>, <strong>Figure 2-6A</strong>, and Figure S2-8). Although transitions in the slope (<span class="math inline">\(\Delta W/\Delta L\)</span> ratio) are difficult to detect at all larval stages because of noise amplification and population effects (Figure S2-9), near the end of L2 and L3 stages we observed a sharp slope increase, consistent with a non-linear stretch regime in length prior to lethargus. As a whole, these results suggest that the material properties of the cuticle influence body shape at larval-stage transitions, leading us to speculate about its role in molt initiation. The ability to sense when a critical cuticle stiffness is reached would allow animals to monitor fold changes in body growth within each larval stage, serving as a connection between growth rate and developmental timing.</p>
<p><img src="img/ch2/fig2-6.png" width="100%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 2-6. Experimental data are consistent with a length threshold in cuticle stretch.</strong> (A) A grayscale histogram of the width (y-axis) vs length (x-axis) of all sampled animals in replicate 2. The range of all bootstrap regressions is in gold. (B) Demonstration of calculating the ratio of width-to-length stretch as the local slope using L3. Left panel is a repetition of L3 data from <strong>Figure 2-6A</strong>. Right panel is a repetition of results from <strong>Figure 2-6C</strong>. (C) Within a larval stage, the ratio of width to length stretch varies over time. The standard deviation captures population variation (grey).</p>
</div>
</div>
</div>
<div id="discussion" class="section level2" number="2.6">
<h2><span class="header-section-number">2.6</span> Discussion</h2>
<p>Using an integrated image-based and flow-based phenotyping strategy to precisely evaluate feeding, growth, and molt dynamics at high replication, we detected oscillations in feeding behavior consistent with larval progressions and used these dynamics to define larval stages. We observed changes in body shape at each larval-stage transition that can be driven by differences in physical cuticle properties along length or width (anisotropy). These results suggest a mechanism by which animals sense their size and control molt timing by detecting the physical stretch of the cuticle, demonstrating how physical constraints can influence developmental timing and growth rate.</p>
<div id="a-potential-role-for-cuticle-stretch-in-the-timing-of-larval-stage-transitions" class="section level3" number="2.6.1">
<h3><span class="header-section-number">2.6.1</span> A potential role for cuticle stretch in the timing of larval-stage transitions</h3>
<p>Measurement of both animal length and width allowed us to observe changes in body shape as well as body size. We propose a mechanism in which a stretch threshold along the body length axis acts as a trigger to larval-stage transitions. Previously, a folder mechanism for <em>C. elegans</em> growth has been suggested <span class="citation">[<a href="references.html#ref-Towbin2021-vx" role="doc-biblioref">99</a>]</span>. Mechanical stretch sensing could provide organisms a way to couple the rate of growth and development to maintain a constant volume fold change within a larval stage and achieve size homeostasis within a population. In this way, smaller animals would reach a stretch limit at a smaller size as the cuticle would only stretch a percentage of its original size before reaching a threshold. For cuticle stretch to trigger larval-stage transitions, animals must either have the ability to measure the amount the cuticle has stretched or the stiffness of the cuticle. Across biological systems, cells can respond to the stiffness of their environment using mechanosensitive components <span class="citation">[<a href="references.html#ref-Schiller2013-ub" role="doc-biblioref">134</a>,<a href="references.html#ref-Wolfenson2011-ad" role="doc-biblioref">135</a>]</span>, but few examples in tissues or whole-organisms are known. In <em>C. elegans</em>, it has been demonstrated that hemidesmosomes, which connect the cuticle and the epidermis, are mechanosensitive during embryogenesis <span class="citation">[<a href="references.html#ref-Suman2019-wz" role="doc-biblioref">136</a>,<a href="references.html#ref-Zhang2011-ea" role="doc-biblioref">137</a>]</span>. Additionally, dense bodies, which connect the epidermis and muscles, are hypothesized to be mechanosensitive as well <span class="citation">[<a href="references.html#ref-Zaidel-Bar2010-up" role="doc-biblioref">48</a>,<a href="references.html#ref-Moerman2006-yw" role="doc-biblioref">138</a>–<a href="references.html#ref-Broday2007-fx" role="doc-biblioref">140</a>]</span>. Changes in cuticle composition, and presumably stiffness, have been shown to also affect well known growth controlling pathways such as the BMP signaling pathway <span class="citation">[<a href="references.html#ref-Madaan2020-ni" role="doc-biblioref">141</a>]</span>. These possible mechanosensitive components could monitor the stiffness of the cuticle and be part of the signaling pathways that regulate the initiation of molts. Further experiments are required to explicitly test whether these components control larval-stage transitions. It is important to note, however, that evidence for a stretch-based mechanism for growth control does not preclude the possibility of a developmental timer. It is likely that physical constraint-based events are part of a larger regulatory system coordinating developmental decisions.</p>
<p>Our analysis of width-to-length ratio variation over larval stages provides a first approximation of the timing of cuticle stretch properties. The sudden increase we observed in the width-to-length ratio suggests that the elasticity in the length direction decreases substantially before a molt and may be detectable by the animal. These observations of shape change also indicate that <em>C. elegans</em> do not preempt the shape change by molting before they hit the “stretch limit,” supporting the role of a physical constraint mechanism. Interestingly, when observing the L4 to adult transition, others have detected anisotropic constriction on the transverse (width) axis followed by gradual relaxation driven by rearrangements in cortical actin networks <span class="citation">[<a href="references.html#ref-Katz2018-we" role="doc-biblioref">142</a>]</span>. Single-worm, high frequency measurements targeting hours surrounding the sudden width-to-length ratio increase, are needed to better resolve cuticle shape dynamics. Although analysis of the larval growth dynamics for <em>C. elegans</em> body shape mutants (<em>dpy, lon, sma</em>) may provide insight into width-to-length ratio variation, measurements of whole animal length and width only provide a total stiffness estimate, leaving us unable to distinguish the contributions of cuticle stiffness from other tissues. For instance, previous work has shown that bodies of <em>dpy-10(e128)</em> are twice as stiff as wild type <span class="citation">[<a href="references.html#ref-Fechner2018-zw" role="doc-biblioref">143</a>]</span> while <em>dpy-5(e61)</em> animals are softer than wild type animals <span class="citation">[<a href="references.html#ref-Park2007-rm" role="doc-biblioref">131</a>]</span>. Authors have speculated that this difference in stiffness is due to an increased internal glycerol concentration in <em>dpy-10(e128)</em> animals that is absent in <em>dpy-5(e61)</em> <span class="citation">[<a href="references.html#ref-Fechner2018-zw" role="doc-biblioref">143</a>]</span>, indicating that body shape alone does not predict body stiffness. To assess cuticle properties independent of other nematode tissues and organs, future experiments probing the stiffness of free cuticles are necessary.</p>
<p>Additionally, within the L1 stage, the relative stretch measured in width and length did not follow the pattern observed in other larval stages. We observed a mid-stage dip in the width-to-length ratio that is otherwise approximately constant throughout the L1 stage. Experiments exploring the structural properties of cuticles at all larval stages can help to determine where the L1 shape changes originate.</p>
</div>
<div id="development-comprises-complex-interactions-of-growth-regulation-across-diverse-scales" class="section level3" number="2.6.2">
<h3><span class="header-section-number">2.6.2</span> Development comprises complex interactions of growth regulation across diverse scales</h3>
<p>Our results demonstrate that <em>C. elegans</em> may use physical constraints on animal size, in tandem with other regulatory mechanisms, to control growth rate and determine developmental transitions. This type of regulation could be applicable to organisms with stiff cuticles or other physical barriers to growth, like many species of Ecdysozoa. The control of whole-organism growth requires cells, tissues, and organs to orchestrate final size and cell number. In <em>C. elegans</em>, cell number is precisely defined and invariant from animal to animal <span class="citation">[<a href="references.html#ref-Horvitz1980-ha" role="doc-biblioref">144</a>]</span>, so the final adult size of an individual must come from cell size as opposed to number. Future studies should focus on how whole-organism size is determined by the integration of cell, tissue, and organ size. By incorporating these different developmental scales, the Stretcher model can be refined to completely describe how physical constraints on parts of the organism impact the whole. <em>C. elegans</em> gives investigators a method to investigate animal-to-animal variation in developmental trajectories across each of these scales.</p>
</div>
</div>
<div id="contributions" class="section level2" number="2.7">
<h2><span class="header-section-number">2.7</span> Contributions</h2>
<p>This work was supported by the NSF-Simons Center for Quantitative Biology at Northwestern University. Dr. Erik Andersen, Dr. Gaotian Zhang, and Nicole Roberto assisted with the high-throughput sampling. Dr. Sasha Shirman and Christina Goss built and tested the mathematical models. Hannah N. Ahmed, Elliot J. Andersen, Isabella R. Miller, Justine K. Rozenich, Iris L. Swarthout, and Jordan A. Vaughn helped collect manual size measurements of animals from images. Drs. Erik Andersen and Niall Mangan supervised the project.</p>
</div>
<div id="supplement" class="section level2" number="2.8">
<h2><span class="header-section-number">2.8</span> Supplement</h2>
<p><img src="img/ch2/supp/S1Fig.png" width="95%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure S2-1. Raw measurements of animal size.</strong> Raw COPAS BIOSORT data of animal length (A), width (B), and volume (C) are shown here. After 60 hours, animals have developed to the adult stage. Smaller objects observed after 65 hours were the next generation of newly hatched L1 larvae laid by the animals that developed during the time course.</p>
<p><img src="img/ch2/supp/S2Fig.png" width="95%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure S2-2. Correlation analysis of body size measurements.</strong> Manual measurements of animal length, width, and estimated area were compared to COPAS BIOSORT measurements of TOF, norm.EXT, and EXT.Kendall correlation value is shown in each plot.</p>
<p><img src="img/ch2/supp/S3Fig.png" width="100%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure S2-3. Mixture modeling of COPAS BIOSORT data was used to prune data.</strong> Mixture models of Gaussian distributions were fit to log transformed animal length (x-axis) and log transformed optical extinction (y-axis). Data from each hour of the experiment was analyzed and processed to remove clusters that did not include animal objects. All replicates were pruned independently; a subset of data from replicate 2 is shown here. Panels indicate experimental hours from which data were taken.</p>
<p><img src="img/ch2/supp/S4Fig.png" width="100%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure S2-4. Fluorescence measurements normalized by body size.</strong> Red fluorescence beads were fed to animals during experimentation and fluorescence data was collected by the COPAS BIOSORT. Fluctuations in fluorescence indicate fluctuations in feeding behavior. Fluorescence data was normalized by body size measurements to account for increases in body size. Dividing fluorescence by area was most successful in normalizing fluorescence dynamics to account for changes in animal size over time.</p>
<p><img src="img/ch2/supp/S5Fig.png" width="100%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure S2-5.</strong> <strong>Cuticles identified during periods of decreased feeding.</strong> Images of wells collected during the experiment were examined for evidence of shed cuticles. (A) Experimental hours where cuticles were identified from images overlap with hours where population feeding behavior is low. Cuticles shed from the L4-Adult molt persisted longer than previous larval stage cuticle debri. (B) Example image of animal without visible cuticle during a period of elevated feeding. (C) Example image of an animal with visible cuticle indicating completion of molt during a period of decreased feeding.</p>
<p><img src="img/ch2/supp/S6Fig.png" width="100%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure S2-6. Volume growth data fit with linear, exponential, and cubic models.</strong> Volume data of individuals in time points defined as growth periods are analyzed for each stage. L1 stage was further separated into two periods to account for the volume dip that occurs mid-stage.</p>
<p><img src="img/ch2/supp/S7Fig.png" width="100%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure S2-7. Density plots of population size dynamics across all larval transitions.</strong> Density curves of length (A), width (B) and volume (C). Curves are divided into five quantiles and colored by the percentage of quiescent animals present within that quantile. Molts are estimated to occur at experimental hours 14, 25, 36, and 48 (see Methods).</p>
<p><img src="img/ch2/supp/S8Fig.png" width="80%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure S2-8. Animals in all replicates, measured from images.</strong> Animal length and width over <em>C. elegans</em> development captured from image data. Higher noise levels in these measurements preclude accurate regressions to individual larval stages. Length jumps and width dips are still apparent. Compare with Fig 6<strong>.</strong></p>
<p><img src="img/ch2/supp/S9Fig.png" width="100%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure S2-9. Stretcher model analysis of replicate 2 COPAS BIOSORT data for different stage thresholding.</strong> Compare to Figure 2-5<strong>.</strong> Larval hours were defined by taking the ceiling of the lower boundary and the floor of the upper boundary. This rounding method for larval stage definition demonstrates the sensitivity of the analysis to edge effects. The unexpected step in the L2 larval stage (Fig 5) was significantly reduced with this rounding method.</p>
<p><img src="img/ch2/supp/S1Table.png" width="85%"  style="display: block; margin: auto;" /></p>
<p><strong>Table S2-1. Results of analysis of variance models fit to COPAS BIOSORT data.</strong> Analysis of variance tests were used to quantify the amount of variance in our data contributed by the sampling technique. The sampling technique involved unbiased sampling of animals from six replicate populations and subsequent distribution into multiple wells of a microtiter plate for analysis. We quantified the amount of variance contributed by replicate and well. We find that the variance explained by well is nearly negligible whereas replicate contributes minor variance in some measurements. Given this information, we deem the generated summary statistics an appropriate representation of the population.</p>
<p><img src="img/ch2/supp/S2Table.png" width="95%"  style="display: block; margin: auto;" /></p>
<p><strong>Table S2-2. Model fit criteria used to assess candidate growth models.</strong> To determine the level of support for each model, the candidate model with the smallest raw AIC/BIC was identified and compared to other AIC/BIC values. If the delta value was greater than 6, the model with the smallest AIC/BIC value was denoted as the best model. If the delta value was less than 6 but greater than 2, the model with the smallest AIC/BIC value was determined to likely be the best model. If the delta value was less than 2, we are unable to distinguish the model of best fit.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="introduction.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": false
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-thesis.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
