<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>A. Growth dynamics of C. elegans cuticle mutants | Mechanisms of Larval Growth Control in Caenorhabditis elegans</title>
  <meta name="description" content="Nyaanga PhD Dissertation" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="A. Growth dynamics of C. elegans cuticle mutants | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/img/NU_seal.png" />
  <meta property="og:description" content="Nyaanga PhD Dissertation" />
  <meta name="github-repo" content="jnyaanga" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="A. Growth dynamics of C. elegans cuticle mutants | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  
  <meta name="twitter:description" content="Nyaanga PhD Dissertation" />
  <meta name="twitter:image" content="/img/NU_seal.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="appendix.html"/>
<link rel="next" href="b.-food-allocation-during-c.-elegans-growth.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="">Nyaanga Dissertation</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="dedication.html"><a href="dedication.html"><i class="fa fa-check"></i>Dedication</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#Section1.1"><i class="fa fa-check"></i><b>1.1</b> Phenotypic variation</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#from-genotype-to-phenotype"><i class="fa fa-check"></i><b>1.1.1</b> From genotype to phenotype</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#Section1.2"><i class="fa fa-check"></i><b>1.2</b> Quantitative trait loci mapping</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#Section1.3"><i class="fa fa-check"></i><b>1.3</b> <em>Caenorhabditis elegans</em></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#as-a-model-organism"><i class="fa fa-check"></i><b>1.3.1</b> As a model organism</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#post-embryonic-development"><i class="fa fa-check"></i><b>1.3.2</b> Post-embryonic development</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#growth-regulation"><i class="fa fa-check"></i><b>1.3.3</b> Growth regulation</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#natural-genetic-variation"><i class="fa fa-check"></i><b>1.3.4</b> Natural genetic variation</a></li>
<li class="chapter" data-level="1.3.5" data-path="introduction.html"><a href="introduction.html#linkage-mapping"><i class="fa fa-check"></i><b>1.3.5</b> Linkage mapping</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><i class="fa fa-check"></i><b>2</b> A role for cuticle stretch in developmental growth control</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#preface"><i class="fa fa-check"></i><b>2.1</b> Preface</a></li>
<li class="chapter" data-level="2.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#abstract-1"><i class="fa fa-check"></i><b>2.2</b> Abstract</a></li>
<li class="chapter" data-level="2.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#introduction-1"><i class="fa fa-check"></i><b>2.3</b> Introduction</a></li>
<li class="chapter" data-level="2.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#materials-and-methods"><i class="fa fa-check"></i><b>2.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#worm-culture"><i class="fa fa-check"></i><b>2.4.1</b> Worm culture</a></li>
<li class="chapter" data-level="2.4.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#bacterial-food"><i class="fa fa-check"></i><b>2.4.2</b> Bacterial food</a></li>
<li class="chapter" data-level="2.4.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#growth-of-the-animals"><i class="fa fa-check"></i><b>2.4.3</b> Growth of the animals</a></li>
<li class="chapter" data-level="2.4.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#high-throughput-measurements-of-body-size-and-fluorescence"><i class="fa fa-check"></i><b>2.4.4</b> High-throughput measurements of body size and fluorescence</a></li>
<li class="chapter" data-level="2.4.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#image-processing"><i class="fa fa-check"></i><b>2.4.5</b> Image processing</a></li>
<li class="chapter" data-level="2.4.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#data-processing"><i class="fa fa-check"></i><b>2.4.6</b> Data processing</a></li>
<li class="chapter" data-level="2.4.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#molt-analysis"><i class="fa fa-check"></i><b>2.4.7</b> Molt analysis</a></li>
<li class="chapter" data-level="2.4.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#comparison-of-model-fits"><i class="fa fa-check"></i><b>2.4.8</b> Comparison of model fits</a></li>
<li class="chapter" data-level="2.4.9" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#stretcher-model-analysis"><i class="fa fa-check"></i><b>2.4.9</b> Stretcher model analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#results"><i class="fa fa-check"></i><b>2.5</b> Results</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#quantitative-measurements-of-c.-elegans-growth"><i class="fa fa-check"></i><b>2.5.1</b> Quantitative measurements of <em>C. elegans</em> growth</a></li>
<li class="chapter" data-level="2.5.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#fluorescence-provides-a-quantitative-measurement-of-animal-feeding-behavior-and-developmental-progression"><i class="fa fa-check"></i><b>2.5.2</b> Fluorescence <strong>provides a quantitative measurement of animal feeding behavior and developmental progression</strong></a></li>
<li class="chapter" data-level="2.5.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#changes-in-c.-elegans-body-shape-occur-at-larval-stage-transitions"><i class="fa fa-check"></i><b>2.5.3</b> Changes in <em>C. elegans</em> body shape occur at larval-stage transitions</a></li>
<li class="chapter" data-level="2.5.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#modeling-c.-elegans-cuticle-stretch-dynamics"><i class="fa fa-check"></i><b>2.5.4</b> Modeling <em>C. elegans</em> cuticle stretch dynamics</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#discussion"><i class="fa fa-check"></i><b>2.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#a-potential-role-for-cuticle-stretch-in-the-timing-of-larval-stage-transitions"><i class="fa fa-check"></i><b>2.6.1</b> A potential role for cuticle stretch in the timing of larval-stage transitions</a></li>
<li class="chapter" data-level="2.6.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#development-comprises-complex-interactions-of-growth-regulation-across-diverse-scales"><i class="fa fa-check"></i><b>2.6.2</b> Development comprises complex interactions of growth regulation across diverse scales</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#contributions"><i class="fa fa-check"></i><b>2.7</b> Contributions</a></li>
<li class="chapter" data-level="2.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#supplement"><i class="fa fa-check"></i><b>2.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><i class="fa fa-check"></i><b>3</b> Multiple genetic loci underlie differences in <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="3.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#preface-1"><i class="fa fa-check"></i><b>3.1</b> Preface</a></li>
<li class="chapter" data-level="3.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#abstract-2"><i class="fa fa-check"></i><b>3.2</b> Abstract</a></li>
<li class="chapter" data-level="3.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#introduction-2"><i class="fa fa-check"></i><b>3.3</b> Introduction</a></li>
<li class="chapter" data-level="3.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#materials-and-methods-1"><i class="fa fa-check"></i><b>3.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#strains"><i class="fa fa-check"></i><b>3.4.1</b> Strains</a></li>
<li class="chapter" data-level="3.4.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-growth-assay"><i class="fa fa-check"></i><b>3.4.2</b> High-throughput growth assay</a></li>
<li class="chapter" data-level="3.4.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-fitness-assay-for-linkage-mapping"><i class="fa fa-check"></i><b>3.4.3</b> High-throughput fitness assay for linkage mapping</a></li>
<li class="chapter" data-level="3.4.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#linkage-mapping-1"><i class="fa fa-check"></i><b>3.4.4</b> Linkage mapping</a></li>
<li class="chapter" data-level="3.4.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#generation-of-chromosome-substitution-strains-csss-and-near-isogenic-lines-nils"><i class="fa fa-check"></i><b>3.4.5</b> Generation of chromosome substitution strains (CSSs) and near-isogenic lines (NILs)</a></li>
<li class="chapter" data-level="3.4.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#statistical-analysis-of-css-and-nil-results"><i class="fa fa-check"></i><b>3.4.6</b> Statistical analysis of CSS and NIL results</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#results-1"><i class="fa fa-check"></i><b>3.5</b> Results</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#growth-dynamics-of-the-n2-and-cb4856-strains"><i class="fa fa-check"></i><b>3.5.1</b> Growth dynamics of the N2 and CB4856 strains</a></li>
<li class="chapter" data-level="3.5.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-qtl-underlying-variation-in-growth"><i class="fa fa-check"></i><b>3.5.2</b> Identification of QTL underlying variation in growth</a></li>
<li class="chapter" data-level="3.5.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#validation-of-loci-associated-with-differences-in-animal-size"><i class="fa fa-check"></i><b>3.5.3</b> Validation of loci associated with differences in animal size</a></li>
<li class="chapter" data-level="3.5.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-candidate-genes-in-the-chromosome-x-qtl"><i class="fa fa-check"></i><b>3.5.4</b> Identification of candidate genes in the chromosome X QTL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#discussion-1"><i class="fa fa-check"></i><b>3.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#a-complex-genetic-architecture-underlies-differences-in-body-size"><i class="fa fa-check"></i><b>3.6.1</b> A complex genetic architecture underlies differences in body size</a></li>
<li class="chapter" data-level="3.6.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#candidate-genes-for-variation-in-body-size"><i class="fa fa-check"></i><b>3.6.2</b> Candidate genes for variation in body size</a></li>
<li class="chapter" data-level="3.6.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#comparison-with-previous-qtl-studies-of-c.-elegans-growth"><i class="fa fa-check"></i><b>3.6.3</b> Comparison with previous QTL studies of <em>C. elegans</em> growth</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#contributions-1"><i class="fa fa-check"></i><b>3.7</b> Contributions</a></li>
<li class="chapter" data-level="3.8" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#supplement-1"><i class="fa fa-check"></i><b>3.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><i class="fa fa-check"></i><b>4</b> easyXpress: An R package to analyze and visualize <em>C. elegans</em> microscopy data generated using CellProfiler</a>
<ul>
<li class="chapter" data-level="4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preface-2"><i class="fa fa-check"></i><b>4.1</b> Preface</a></li>
<li class="chapter" data-level="4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#abstract-3"><i class="fa fa-check"></i><b>4.2</b> Abstract</a></li>
<li class="chapter" data-level="4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#introduction-3"><i class="fa fa-check"></i><b>4.3</b> Introduction</a></li>
<li class="chapter" data-level="4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#materials-and-methods-2"><i class="fa fa-check"></i><b>4.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preparation-of-animals-for-imaging"><i class="fa fa-check"></i><b>4.4.1</b> Preparation of animals for imaging</a></li>
<li class="chapter" data-level="4.4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#imaging"><i class="fa fa-check"></i><b>4.4.2</b> Imaging</a></li>
<li class="chapter" data-level="4.4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#paraquat-dose-response"><i class="fa fa-check"></i><b>4.4.3</b> Paraquat dose response</a></li>
<li class="chapter" data-level="4.4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#naming-conventions"><i class="fa fa-check"></i><b>4.4.4</b> Naming conventions</a></li>
<li class="chapter" data-level="4.4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-availability"><i class="fa fa-check"></i><b>4.4.5</b> Data availability</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#results-2"><i class="fa fa-check"></i><b>4.5</b> Results</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#design-and-implementation"><i class="fa fa-check"></i><b>4.5.1</b> Design and implementation</a></li>
<li class="chapter" data-level="4.5.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-import-and-model-assignment"><i class="fa fa-check"></i><b>4.5.2</b> Data import and model assignment</a></li>
<li class="chapter" data-level="4.5.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-pruning-and-summarization"><i class="fa fa-check"></i><b>4.5.3</b> Data pruning and summarization</a></li>
<li class="chapter" data-level="4.5.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#visualization"><i class="fa fa-check"></i><b>4.5.4</b> Visualization</a></li>
<li class="chapter" data-level="4.5.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#application-to-c.-elegans-growth-data"><i class="fa fa-check"></i><b>4.5.5</b> Application to <em>C. elegans</em> growth data</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#conclusions"><i class="fa fa-check"></i><b>4.6</b> Conclusions</a></li>
<li class="chapter" data-level="4.7" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#contributions-2"><i class="fa fa-check"></i><b>4.7</b> Contributions</a></li>
<li class="chapter" data-level="4.8" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#supplement-2"><i class="fa fa-check"></i><b>4.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion-2.html"><a href="discussion-2.html"><i class="fa fa-check"></i><b>5</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion-2.html"><a href="discussion-2.html#summary"><i class="fa fa-check"></i><b>5.1</b> Summary</a></li>
<li class="chapter" data-level="5.2" data-path="discussion-2.html"><a href="discussion-2.html#the-future-of-developmental-growth-studies"><i class="fa fa-check"></i><b>5.2</b> The future of developmental growth studies</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="discussion-2.html"><a href="discussion-2.html#the-c.-elegans-cuticle-and-its-role-in-developmental-timing"><i class="fa fa-check"></i><b>5.2.1</b> The <em>C. elegans</em> cuticle and its role in developmental timing</a></li>
<li class="chapter" data-level="5.2.2" data-path="discussion-2.html"><a href="discussion-2.html#identifying-genetic-factors-that-contribute-to-variation-in-growth-and-body-size"><i class="fa fa-check"></i><b>5.2.2</b> Identifying genetic factors that contribute to variation in growth and body size</a></li>
<li class="chapter" data-level="5.2.3" data-path="discussion-2.html"><a href="discussion-2.html#phenotypic-plasticity-and-genotype-by-environment-interaction"><i class="fa fa-check"></i><b>5.2.3</b> Phenotypic plasticity and genotype-by-environment interaction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
<li class="chapter" data-level="7" data-path="appendix.html"><a href="appendix.html"><i class="fa fa-check"></i><b>7</b> Appendix</a></li>
<li><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#a.-growth-dynamics-of-c.-elegans-cuticle-mutants">A. Growth dynamics of <em>C. elegans</em> cuticle mutants</a>
<ul>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#description"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#methods"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#contributions-3"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li><a href="b.-food-allocation-during-c.-elegans-growth.html#b.-food-allocation-during-c.-elegans-growth">B. Food allocation during <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#methods-1"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#contributions-4"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Mechanisms of Larval Growth Control in <em>Caenorhabditis elegans</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="a.-growth-dynamics-of-c.-elegans-cuticle-mutants" class="section level1 unnumbered">
<h1>A. Growth dynamics of <em>C. elegans</em> cuticle mutants</h1>
<p><strong>Characterization of larval growth in <em>C. elegans</em> cuticle mutants</strong></p>
<p>Joy Nyaanga, Sasha Shirman, Niall M. Mangan, and Erik C. Andersen</p>
<div id="abstract-4" class="section level2 unnumbered">
<h2>Abstract</h2>
<p>In <em>Caenorhabditis elegans</em>, many genes involved in the formation of the cuticle are also known to influence body size and shape. We assessed post-embryonic growth of both long and short <em>C. elegans</em> body size mutants from the L1 to L4 stage. We found similar developmental trajectories of N2 and <em>lon-3</em> animals. By contrast, we observed overall decreases in body length and increases in body width of tested <em>dpy</em> mutants compared to N2, consistent with the Dpy phenotype. We further show that the dynamics of animal shape in the mutant strains are consistent with a previously proposed “Stretcher” growth model.</p>
<p><img src="img/appendixA/fig1.png" width="100%" style="display: block; margin: auto;" /></p>
<p><strong>Figure A-1: Quantitative assessment of C. elegans larval growth.</strong> (A) Representative images of strains used in this study taken at the L4 stage. Tukey boxplots for median animal fluorescence normalized by area (B), median animal length (<span class="math inline">\(\mu M\)</span>) (C), and median animal width (<span class="math inline">\(\mu M\)</span>) (D). The horizontal line in the middle of the box is the median, and the box denotes the 25th to 75th quantiles of the data. The vertical line represents the 1.5 interquartile range. Each point corresponds to the median value of a population of animals in each well. (E) Median length (x-axis) plotted against median length (y-axis). (F) The ratio of the change in width to length over time. Calculated from the local slope of data in panel (E). The standard deviation captures population variation (grey).</p>
</div>
<div id="description" class="section level2 unnumbered">
<h2>Description</h2>
<p>Body size largely influences an organism’s functional characteristics: growth, reproduction, metabolism, lifespan. As a result, the determinant factors of organism size, particularly during development, have been explored in many systems. <em>Caenorhabditis elegans</em>, a free-living nematode, presents a versatile genetic model system to study how the processes of growth and development are regulated. <em>C. elegans</em> matures to an adult after multiple molting events during which time animals synthesize a new exoskeleton (cuticle) and expel their old one. The <em>C. elegans</em> cuticle is a complex, multi-layered structure primarily composed of collagens. As animals progress through their life-cycle, the structure and thickness of the cuticle changes but its role in the maintenance of body morphology and integrity remains. To date, 21 cuticle collagen mutants have been identified that cause a range of body morphology defects (Page and Johnstone 2007). Some of these mutants exhibit a disproportionate reduction in body size, while others are noticeably larger than wild type (Cho et al. 2021), clearly demonstrating the importance of the physical structure of the cuticle on growth. Analyzing the characteristics of size during development in various <em>C. elegans</em> body shape mutants is central to understanding the role these genetic pathways have on body growth.</p>
<p>We performed a high-resolution longitudinal study of growth in a selection of <em>C. elegans</em> cuticle mutants. We selected mutants that were both reportedly shorter (<em>dpy-1(e1)</em>, <em>dpy-5(e61)</em>) and longer (<em>lon-3(e2175)</em>) than wild type (Cho et al. 2021) (<strong>Figure A-1A</strong>). We then collected high-precision measurements of animal fluorescence (<strong>Figure A-1B</strong>), length (<strong>Figure A-1C</strong>), and width (<strong>Figure A-1D</strong>) from the L1 stage through the L4 stage. As we previously demonstrated (Nyaanga et al. 2022) we can use oscillations in fluorescence as a proxy for feeding behavior to characterize larval progression by associating periods of decreased feeding with molt events. Doing so, we notice that <em>lon-3(e2175)</em> animals follow similar developmental trajectories to the N2 wild type. By contrast, we detect a delay in the molt timing of both <em>dpy</em> mutant strains, with <em>dpy-1(e1)</em> undergoing each larval transition later than all other tested strains. We also observe a marked decrease in animal length and increase in animal width noticeable after the L1 stage in both <em>dpy</em> strains, consistent with their characteristic dumpy phenotype. Interestingly, we note little size divergence between <em>lon-3(e2175)</em> and N2 animals during our time course.</p>
<p>Measurements of both animal length and width allow us to assess changes in body shape as well as size. Previously, motivated by changes in the body aspect ratio of animals we observed at larval stage transitions, we modeled a physical mechanism by which constraints on cuticle stretch could cause changes in <em>C. elegans</em> body shape (Nyaanga et al. 2022). We found that model-predicted shape changes were consistent with those seen in our data of N2 animals. Given this result, we proposed a “Stretcher” model for growth wherein <em>C. elegans</em> sense changes in cuticle elasticity, in tandem with other regulatory mechanisms, to control growth rate and determine developmental transitions. Given the structural impacts of <em>dpy-1(e1)</em>, <em>dpy-5(e61)</em> and <em>lon-3(e2175)</em> mutants, we sought to determine whether the shape dynamics predicted by the Stretcher model would be consistent with the mutant data. By analyzing the relationship between measured animal length and width (W/L) over time, we are able to detect the linear and nonlinear stretch regimes predicted by the Stretcher model (<strong>Figure A-1F</strong>). In all strains, we observe an approximately constant W/L ratio in all larval stages, consistent with a linear stretch regime. Next, we observe a shape slope increase, consistent with a nonlinear stretch regime in length preceding larval stage transitions.</p>
</div>
<div id="methods" class="section level2 unnumbered">
<h2>Methods</h2>
<p><strong>Worm culture</strong></p>
<p>The laboratory strain N2 was obtained from the C. elegans Natural Diversity Resource (Cook et al. 2017). All other strains were provided by the CGC, which is funded by NIH Office of Research Infrastructure Programs (P40 OD010440). Animals were cultured at 20C on 6 cm plates of modified nematode growth media (NGMA), which contained 1% agar and 0.7% agarose seeded with <em>E. coli</em> OP50 bacteria.</p>
<p><strong>High-throughput growth assay</strong></p>
<p>Measurements of body size and fluorescence were measured as previously described (Nyaanga et al. 2022). Briefly, strains were propagated for three generations, bleach-synchronized, and titered at a concentration of 1 embryo per <span class="math inline">\(\mu L\)</span> into 250 mL flasks. The following day, arrested L1s were fed HB101 food at a final concentration of OD20 in a final flask volume of 100 mL K medium and HB101 food. Animals were grown with constant shaking at 20C. Flasks were sampled each hour beginning one hour after feeding and continuing for 42 consecutive hours. At each hour, 800 <span class="math inline">\(\mu L\)</span> was removed from each flask and incubated with fluorescent polychromatic beads (Polysciences, 19507-5) for 10 minutes with shaking. Following the bead incubation, animals were transferred to a 96-well microtiter plate, treated with sodium azide, imaged with an ImageXpress Nano (Molecular Devices, SanJose, CA), and scored using a large-particle flow cytometer (COPAS BIOSORT, Union Biometrica, Holliston MA). COPAS BIOSORT was used to collect measurements of animal length (TOF), optical extinction (EXT), and red fluorescence for every animal in each well.</p>
<p><strong>Data processing</strong></p>
<p>COPAS BIOSORT data were processed as previously described (Nyaanga et al. 2022). To remove non-animal objects such as bacterial clumps, shed cuticles, and next generation larval animals from the time-course data. Data for each well was summarized to obtain median well measurements. TOF and norm.EXT data were then converted to microns. Only the unit-corrected data were used for further analysis. “Stretcher” model analysis of shape dynamics was performed as previously described (Nyaanga et al. 2022).</p>
<p><strong>Reagents</strong></p>
<table>
<thead>
<tr class="header">
<th><strong>STRAIN</strong></th>
<th><strong>GENOTYPE</strong></th>
<th><strong>AVAILABLE FROM</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>N2</td>
<td><em>Caenorhabditis elegans</em></td>
<td>CeNDR</td>
</tr>
<tr class="even">
<td>CB1</td>
<td><em>dpy-1(e1)</em></td>
<td>CGC</td>
</tr>
<tr class="odd">
<td>CB61</td>
<td><em>dpy-5(e61)</em></td>
<td>CGC</td>
</tr>
<tr class="even">
<td>CB4123</td>
<td><em>lon-3(e2175)</em></td>
<td>CGC</td>
</tr>
</tbody>
</table>
</div>
<div id="contributions-3" class="section level2 unnumbered">
<h2>Contributions</h2>
<p>Joy Nyaanga: Conceptualization, Methodology, Investigation, Formal Analysis, Visualization, Writing - original draft</p>
<p>Sasha Shirman: Methodology, Formal Analysis, Visualization</p>
<p>Niall M. Mangan: Methodology, Funding acquisition, Supervision</p>
<p>Erik C. Andersen: Conceptualization, Methodology, Investigation, Funding acquisition, Supervision</p>
</div>
<div id="references-1" class="section level2 unnumbered">
<h2>References</h2>
<ol style="list-style-type: decimal">
<li><p>Page AP, Johnstone IL. The cuticle. WormBook. 2007 Mar 19;1–15. PMCID: PMC4781593</p></li>
<li><p>Cho JY, Choi T-W, Kim SH, Ahnn J, Lee S-K. Morphological Characterization of small, dumpy, and long Phenotypes in <em>Caenorhabditis elegans</em>. Mol Cells. Korean Society for Molecular and Cellular Biology; 2021 Mar 31;44(3):160–167. PMCID: PMC8019597</p></li>
<li><p>Nyaanga J, Goss C, Zhang G, Ahmed HN, Andersen EJ, Miller IR, Rozenich JK, Swarthout IL, Vaughn JA, Mangan NM, Shirman S, Andersen EC. Changes in body shape implicate cuticle stretch in <em>C. elegans</em> growth control. bioRxiv. 2022. p. 2021.04.01.438121. Available from: <a href="https://www.biorxiv.org/content/10.1101/2021.04.01.438121v3" class="uri">https://www.biorxiv.org/content/10.1101/2021.04.01.438121v3</a></p></li>
<li><p>Cook DE, Zdraljevic S, Roberts JP, Andersen EC. CeNDR, the <em>Caenorhabditis elegans</em> natural diversity resource. Nucleic Acids Res [Internet]. 2017 Jan 4;45(D1):D650–D657. Available from: <a href="http://dx.doi.org/10.1093/nar/gkw893" class="uri">http://dx.doi.org/10.1093/nar/gkw893</a> PMCID: PMC5210618</p></li>
</ol>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="appendix.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="b.-food-allocation-during-c.-elegans-growth.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": false
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-thesis.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
