<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>B. Food allocation during C. elegans growth | Mechanisms of Larval Growth Control in Caenorhabditis elegans</title>
  <meta name="description" content="Nyaanga PhD Dissertation" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="B. Food allocation during C. elegans growth | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/img/NU_seal.png" />
  <meta property="og:description" content="Nyaanga PhD Dissertation" />
  <meta name="github-repo" content="jnyaanga" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="B. Food allocation during C. elegans growth | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  
  <meta name="twitter:description" content="Nyaanga PhD Dissertation" />
  <meta name="twitter:image" content="/img/NU_seal.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"/>

<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="">Nyaanga Dissertation</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="dedication.html"><a href="dedication.html"><i class="fa fa-check"></i>Dedication</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#Section1.1"><i class="fa fa-check"></i><b>1.1</b> Phenotypic variation</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#from-genotype-to-phenotype"><i class="fa fa-check"></i><b>1.1.1</b> From genotype to phenotype</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#Section1.2"><i class="fa fa-check"></i><b>1.2</b> Quantitative trait loci mapping</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#Section1.3"><i class="fa fa-check"></i><b>1.3</b> <em>Caenorhabditis elegans</em></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#as-a-model-organism"><i class="fa fa-check"></i><b>1.3.1</b> As a model organism</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#post-embryonic-development"><i class="fa fa-check"></i><b>1.3.2</b> Post-embryonic development</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#growth-regulation"><i class="fa fa-check"></i><b>1.3.3</b> Growth regulation</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#natural-genetic-variation"><i class="fa fa-check"></i><b>1.3.4</b> Natural genetic variation</a></li>
<li class="chapter" data-level="1.3.5" data-path="introduction.html"><a href="introduction.html#linkage-mapping"><i class="fa fa-check"></i><b>1.3.5</b> Linkage mapping</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><i class="fa fa-check"></i><b>2</b> A role for cuticle stretch in developmental growth control</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#preface"><i class="fa fa-check"></i><b>2.1</b> Preface</a></li>
<li class="chapter" data-level="2.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#abstract-1"><i class="fa fa-check"></i><b>2.2</b> Abstract</a></li>
<li class="chapter" data-level="2.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#introduction-1"><i class="fa fa-check"></i><b>2.3</b> Introduction</a></li>
<li class="chapter" data-level="2.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#materials-and-methods"><i class="fa fa-check"></i><b>2.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#worm-culture"><i class="fa fa-check"></i><b>2.4.1</b> Worm culture</a></li>
<li class="chapter" data-level="2.4.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#bacterial-food"><i class="fa fa-check"></i><b>2.4.2</b> Bacterial food</a></li>
<li class="chapter" data-level="2.4.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#growth-of-the-animals"><i class="fa fa-check"></i><b>2.4.3</b> Growth of the animals</a></li>
<li class="chapter" data-level="2.4.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#high-throughput-measurements-of-body-size-and-fluorescence"><i class="fa fa-check"></i><b>2.4.4</b> High-throughput measurements of body size and fluorescence</a></li>
<li class="chapter" data-level="2.4.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#image-processing"><i class="fa fa-check"></i><b>2.4.5</b> Image processing</a></li>
<li class="chapter" data-level="2.4.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#data-processing"><i class="fa fa-check"></i><b>2.4.6</b> Data processing</a></li>
<li class="chapter" data-level="2.4.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#molt-analysis"><i class="fa fa-check"></i><b>2.4.7</b> Molt analysis</a></li>
<li class="chapter" data-level="2.4.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#comparison-of-model-fits"><i class="fa fa-check"></i><b>2.4.8</b> Comparison of model fits</a></li>
<li class="chapter" data-level="2.4.9" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#stretcher-model-analysis"><i class="fa fa-check"></i><b>2.4.9</b> Stretcher model analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#results"><i class="fa fa-check"></i><b>2.5</b> Results</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#quantitative-measurements-of-c.-elegans-growth"><i class="fa fa-check"></i><b>2.5.1</b> Quantitative measurements of <em>C. elegans</em> growth</a></li>
<li class="chapter" data-level="2.5.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#fluorescence-provides-a-quantitative-measurement-of-animal-feeding-behavior-and-developmental-progression"><i class="fa fa-check"></i><b>2.5.2</b> Fluorescence <strong>provides a quantitative measurement of animal feeding behavior and developmental progression</strong></a></li>
<li class="chapter" data-level="2.5.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#changes-in-c.-elegans-body-shape-occur-at-larval-stage-transitions"><i class="fa fa-check"></i><b>2.5.3</b> Changes in <em>C. elegans</em> body shape occur at larval-stage transitions</a></li>
<li class="chapter" data-level="2.5.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#modeling-c.-elegans-cuticle-stretch-dynamics"><i class="fa fa-check"></i><b>2.5.4</b> Modeling <em>C. elegans</em> cuticle stretch dynamics</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#discussion"><i class="fa fa-check"></i><b>2.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#a-potential-role-for-cuticle-stretch-in-the-timing-of-larval-stage-transitions"><i class="fa fa-check"></i><b>2.6.1</b> A potential role for cuticle stretch in the timing of larval-stage transitions</a></li>
<li class="chapter" data-level="2.6.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#development-comprises-complex-interactions-of-growth-regulation-across-diverse-scales"><i class="fa fa-check"></i><b>2.6.2</b> Development comprises complex interactions of growth regulation across diverse scales</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#contributions"><i class="fa fa-check"></i><b>2.7</b> Contributions</a></li>
<li class="chapter" data-level="2.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#supplement"><i class="fa fa-check"></i><b>2.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><i class="fa fa-check"></i><b>3</b> Multiple genetic loci underlie differences in <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="3.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#preface-1"><i class="fa fa-check"></i><b>3.1</b> Preface</a></li>
<li class="chapter" data-level="3.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#abstract-2"><i class="fa fa-check"></i><b>3.2</b> Abstract</a></li>
<li class="chapter" data-level="3.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#introduction-2"><i class="fa fa-check"></i><b>3.3</b> Introduction</a></li>
<li class="chapter" data-level="3.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#materials-and-methods-1"><i class="fa fa-check"></i><b>3.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#strains"><i class="fa fa-check"></i><b>3.4.1</b> Strains</a></li>
<li class="chapter" data-level="3.4.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-growth-assay"><i class="fa fa-check"></i><b>3.4.2</b> High-throughput growth assay</a></li>
<li class="chapter" data-level="3.4.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-fitness-assay-for-linkage-mapping"><i class="fa fa-check"></i><b>3.4.3</b> High-throughput fitness assay for linkage mapping</a></li>
<li class="chapter" data-level="3.4.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#linkage-mapping-1"><i class="fa fa-check"></i><b>3.4.4</b> Linkage mapping</a></li>
<li class="chapter" data-level="3.4.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#generation-of-chromosome-substitution-strains-csss-and-near-isogenic-lines-nils"><i class="fa fa-check"></i><b>3.4.5</b> Generation of chromosome substitution strains (CSSs) and near-isogenic lines (NILs)</a></li>
<li class="chapter" data-level="3.4.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#statistical-analysis-of-css-and-nil-results"><i class="fa fa-check"></i><b>3.4.6</b> Statistical analysis of CSS and NIL results</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#results-1"><i class="fa fa-check"></i><b>3.5</b> Results</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#growth-dynamics-of-the-n2-and-cb4856-strains"><i class="fa fa-check"></i><b>3.5.1</b> Growth dynamics of the N2 and CB4856 strains</a></li>
<li class="chapter" data-level="3.5.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-qtl-underlying-variation-in-growth"><i class="fa fa-check"></i><b>3.5.2</b> Identification of QTL underlying variation in growth</a></li>
<li class="chapter" data-level="3.5.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#validation-of-loci-associated-with-differences-in-animal-size"><i class="fa fa-check"></i><b>3.5.3</b> Validation of loci associated with differences in animal size</a></li>
<li class="chapter" data-level="3.5.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-candidate-genes-in-the-chromosome-x-qtl"><i class="fa fa-check"></i><b>3.5.4</b> Identification of candidate genes in the chromosome X QTL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#discussion-1"><i class="fa fa-check"></i><b>3.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#a-complex-genetic-architecture-underlies-differences-in-body-size"><i class="fa fa-check"></i><b>3.6.1</b> A complex genetic architecture underlies differences in body size</a></li>
<li class="chapter" data-level="3.6.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#candidate-genes-for-variation-in-body-size"><i class="fa fa-check"></i><b>3.6.2</b> Candidate genes for variation in body size</a></li>
<li class="chapter" data-level="3.6.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#comparison-with-previous-qtl-studies-of-c.-elegans-growth"><i class="fa fa-check"></i><b>3.6.3</b> Comparison with previous QTL studies of <em>C. elegans</em> growth</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#contributions-1"><i class="fa fa-check"></i><b>3.7</b> Contributions</a></li>
<li class="chapter" data-level="3.8" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#supplement-1"><i class="fa fa-check"></i><b>3.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><i class="fa fa-check"></i><b>4</b> easyXpress: An R package to analyze and visualize <em>C. elegans</em> microscopy data generated using CellProfiler</a>
<ul>
<li class="chapter" data-level="4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preface-2"><i class="fa fa-check"></i><b>4.1</b> Preface</a></li>
<li class="chapter" data-level="4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#abstract-3"><i class="fa fa-check"></i><b>4.2</b> Abstract</a></li>
<li class="chapter" data-level="4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#introduction-3"><i class="fa fa-check"></i><b>4.3</b> Introduction</a></li>
<li class="chapter" data-level="4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#materials-and-methods-2"><i class="fa fa-check"></i><b>4.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preparation-of-animals-for-imaging"><i class="fa fa-check"></i><b>4.4.1</b> Preparation of animals for imaging</a></li>
<li class="chapter" data-level="4.4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#imaging"><i class="fa fa-check"></i><b>4.4.2</b> Imaging</a></li>
<li class="chapter" data-level="4.4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#paraquat-dose-response"><i class="fa fa-check"></i><b>4.4.3</b> Paraquat dose response</a></li>
<li class="chapter" data-level="4.4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#naming-conventions"><i class="fa fa-check"></i><b>4.4.4</b> Naming conventions</a></li>
<li class="chapter" data-level="4.4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-availability"><i class="fa fa-check"></i><b>4.4.5</b> Data availability</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#results-2"><i class="fa fa-check"></i><b>4.5</b> Results</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#design-and-implementation"><i class="fa fa-check"></i><b>4.5.1</b> Design and implementation</a></li>
<li class="chapter" data-level="4.5.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-import-and-model-assignment"><i class="fa fa-check"></i><b>4.5.2</b> Data import and model assignment</a></li>
<li class="chapter" data-level="4.5.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-pruning-and-summarization"><i class="fa fa-check"></i><b>4.5.3</b> Data pruning and summarization</a></li>
<li class="chapter" data-level="4.5.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#visualization"><i class="fa fa-check"></i><b>4.5.4</b> Visualization</a></li>
<li class="chapter" data-level="4.5.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#application-to-c.-elegans-growth-data"><i class="fa fa-check"></i><b>4.5.5</b> Application to <em>C. elegans</em> growth data</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#conclusions"><i class="fa fa-check"></i><b>4.6</b> Conclusions</a></li>
<li class="chapter" data-level="4.7" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#contributions-2"><i class="fa fa-check"></i><b>4.7</b> Contributions</a></li>
<li class="chapter" data-level="4.8" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#supplement-2"><i class="fa fa-check"></i><b>4.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion-2.html"><a href="discussion-2.html"><i class="fa fa-check"></i><b>5</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion-2.html"><a href="discussion-2.html#summary"><i class="fa fa-check"></i><b>5.1</b> Summary</a></li>
<li class="chapter" data-level="5.2" data-path="discussion-2.html"><a href="discussion-2.html#the-future-of-developmental-growth-studies"><i class="fa fa-check"></i><b>5.2</b> The future of developmental growth studies</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="discussion-2.html"><a href="discussion-2.html#the-c.-elegans-cuticle-and-its-role-in-developmental-timing"><i class="fa fa-check"></i><b>5.2.1</b> The <em>C. elegans</em> cuticle and its role in developmental timing</a></li>
<li class="chapter" data-level="5.2.2" data-path="discussion-2.html"><a href="discussion-2.html#identifying-genetic-factors-that-contribute-to-variation-in-growth-and-body-size"><i class="fa fa-check"></i><b>5.2.2</b> Identifying genetic factors that contribute to variation in growth and body size</a></li>
<li class="chapter" data-level="5.2.3" data-path="discussion-2.html"><a href="discussion-2.html#phenotypic-plasticity-and-genotype-by-environment-interaction"><i class="fa fa-check"></i><b>5.2.3</b> Phenotypic plasticity and genotype-by-environment interaction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
<li class="chapter" data-level="7" data-path="appendix.html"><a href="appendix.html"><i class="fa fa-check"></i><b>7</b> Appendix</a></li>
<li><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#a.-growth-dynamics-of-c.-elegans-cuticle-mutants">A. Growth dynamics of <em>C. elegans</em> cuticle mutants</a>
<ul>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#description"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#methods"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#contributions-3"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li><a href="b.-food-allocation-during-c.-elegans-growth.html#b.-food-allocation-during-c.-elegans-growth">B. Food allocation during <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#methods-1"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#contributions-4"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Mechanisms of Larval Growth Control in <em>Caenorhabditis elegans</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="b.-food-allocation-during-c.-elegans-growth" class="section level1 unnumbered">
<h1>B. Food allocation during <em>C. elegans</em> growth</h1>
<p><strong>Mechanical control of feeding and allocation of food energy likely constrain growth dynamics in <em>C. elegans</em></strong></p>
<p>Sasha Shirman, Joy Nyaanga, Erik C. Andersen, and Niall M. Mangan</p>
<p><img src="img/appendixB/fig1.png" width="90%" style="display: block; margin: auto;" /></p>
<p><strong>Figure B-1: Visualization and analysis of food intake and utilization models.</strong> (A) A single cycle in which animals take up and transport food to the gut consists of four general steps. The cycle begins with relaxed pharyngeal muscles and a closed pharyngeal lumen (step 1). The animal opens the pharyngeal lumen so that fluid and food flow through the buccal cavity and into the pharyngeal lumen (step 2). The animal relaxes the muscles that control the opening, stopping the flow of fluid through the buccal cavity and trapping a volume of media approximately equal to the maximum volume of the pharyngeal lumen (step 3). Finally, the pharynx closes, extruding excess media and trapping bacteria and beads that are then ‘swallowed’ (step 4). (B) Product of pumping frequency and pharyngeal lumen fraction. (C) Dynamics of volume growth rate. (D) Dynamics of the estimated food utilization towards growth. (E) Dynamics of red fluorescence. (F) Dynamics of food allocation breakdown. Food allocated toward maintenance is proportional to animal size (pink). Food allocated toward growth is calculated from food intake (C) and utilization (D) to produce low (dark green) and high (combined green) estimates. Food allocated to other metabolic processes consists of remaining food resources (grey). In panels C-E, the solid blue line represents the mean bootstrap regression. Standard deviation on the regression is marked by the shaded blue region. In panels C-F, Vertical red lines mark molt times. Errors on shaded regions are +/- 0.7 hours. These data are from replicate 2 and are representative of all replicates.</p>
<div id="description-1" class="section level2 unnumbered">
<h2>Description</h2>
<p>We have previously shown that changes in physical properties of the <em>C. elegans</em> cuticle might serve as a cue for developmental timing (Nyaanga et al. 2021). <em>C. elegans</em> must have mechanisms to control growth throughout development, particularly in response to these cues. Like most species, <em>C. elegans</em> do not increase their growth rate indefinitely in response to increased food availability (Uppaluri and Brangwynne 2015). The animals could control growth entirely using feeding rate, as they actively control the feeding rate (Fang-Yen et al. 2009) and stop feeding at the initiation of a molt (Singh and Sulston 1978). In addition to this mechanical control, they could use metabolic control to preemptively divert ingested resources toward or away from growth. Metabolic processing of stored resources could be especially useful if animals complete their molt and enter a food-limited environment. Using a quantitative feeding model, we examine the <em>C. elegans</em> growth data (Nyaanga et al. 2021) to investigate the possible mechanical control of feeding and the metabolic control of the allocation of ingested food toward organismal growth and development.</p>
<p><em>C. elegans</em> is a filter feeder that pumps its food through a cycle of pharyngeal muscle contractions and relaxations that alternatively open and close the pharyngeal cavity. <em>C. elegans</em> are capable of actively modulating the length of time used to contract or open a subset of the pharyngeal muscles (Fang-Yen et al. 2009). Our feeding model describes how control of this time length and thus the control of the overall pumping period, defined as the length of time of one full pumping cycle, translates to changes in the rate of food uptake (<strong>Figure B-1A</strong>). The results of our analysis show both an oscillation in pumping rates and a slow change in pharyngeal lumen size throughout development (<strong>Figure B-1B</strong>). The minima of the product of pumping frequency and pharynx fraction occurred during molt times, consistent with prior knowledge (Singh and Sulston 1978; Byerly et al. 1976). The relative length of the pharynx to animal length has been shown to decrease over development (Avery 2003) and a similar trend in the volume of the pharyngeal lumen could explain the slow decrease in the product of pumping frequency and pharynx fraction.</p>
<p>To understand whether animals control growth rate primarily using the mechanical feeding process or metabolic regulation, we describe how food is utilized once it is ingested by <em>C. elegans</em>. We assumed three general categories of food utilization: maintenance, observed volumetric growth, and all other processes. We assume that resources required for maintenance are proportional to the volume of the animal. Other processes may include the development of tissues and structures, such as reproductive components, within the worm which are resource intensive, but are not detected during the measurements of the width, length, or volume. Food intake rate (using red fluorescence as a proxy) and volume growth rate (numerically differentiated from the volume regression) follow similar dynamics throughout much of <em>C. elegans</em> development (<strong>Figure B-1C</strong> and <strong>Figure B-1E</strong>). Each larval stage consists of three types of dynamics: steady growth during which food intake and growth rate are both increasing (green), preparation for molt during which food intake and growth rate are both decreasing (red), and transition regions where food intake and growth rate do not vary together (white). During steady growth and molt initiation, mechanical food intake rate and allocation of food resources towards observable growth have similar dynamics. At transition points the allocation of food resources towards growth, reaches a local minimum or maximum. These local extrema in food utilization at transition points likely indicate large changes in metabolic regulation (<strong>Figure B-1D</strong>).</p>
<p>Our analysis of the relationship between growth rate and food intake rate quantified the interplay of metabolic regulation and mechanical food intake (<strong>Figure B-1F</strong>). Mechanical food intake provides the upper bound of available food resources for growth, but metabolic regulation substantially changes how much of this available food is utilized for volumetric growth across development. Local extrema in metabolic regulation dynamics coincide with the start of transition times. Within the L2 and L3 stages we observed a decoupling between growth rate and food intake dynamics (<strong>Figure B-1F</strong> white region) twice. The first of these time periods corresponds to the times at which the width-to-length ratio drastically changes and the second corresponds to ecdysis. We estimated that throughout larval stages metabolic resources allocated to non-growth processes varies much less than food resources required for growth and maintenance. The oscillatory behavior of food utilization motivates the need for further metabolomic experiments to probe metabolic regulatory dynamics particularly in around larval-stage transitions.</p>
</div>
<div id="methods-1" class="section level2 unnumbered">
<h2>Methods</h2>
<p><strong>Worm culture</strong></p>
<p>The canonical laboratory strain N2 was obtained from the <em>C. elegans</em> Natural Diversity Resource (Cook et al. 2016) and animals were cultured at 20C on 6 cm plates of modified nematode growth media (NGMA), which contained 1% agar and 0.7% agarose seeded with <em>E. coli</em> OP50 bacteria.</p>
<p><strong>High-throughput growth assay</strong></p>
<p>Measurements of body size and fluorescence were measured as previously described (Nyaanga et al. 2022). Briefly, strains were propagated for three generations, bleach-synchronized, and titered at a concentration of 1 embryo per <span class="math inline">\(\mu L\)</span> into six replicate 500 mL flasks. The following day, arrested L1s were fed HB101 food at a final concentration of OD20 in a final flask volume of 100 mL K medium and HB101 food. Animals were allowed to grow at 20C with constant shaking. Flasks were sampled each hour beginning one hour after feeding and continuing for 72 consecutive hours. At each hour, 500 <span class="math inline">\(\mu L\)</span> was removed from each flask and incubated with fluorescent polychromatic beads (Polysciences, 19507-5) for 10 minutes with shaking. Following the bead incubation, animals were aliquoted to a 96-well microtiter plate, treated with sodium azide, imaged with an ImageXpress Nano (Molecular Devices, SanJose, CA), and scored using a large-particle flow cytometer (COPAS BIOSORT, Union Biometrica, Holliston MA). COPAS BIOSORT was used to collect measurements of animal length (TOF), optical extinction (EXT), and red fluorescence for every animal in each well.</p>
<p><strong>Feeding model analysis</strong></p>
<p>To analyze volumetric growth dynamics and feeding dynamics, volume regression was calculated using a cylindrical approximation for animal shape and the same local kernel regression previously described (Nyaanga et al. 2021) was applied to red fluorescence data. Volume growth rate was calculated using the python numpy gradient function applied to the volume regressions. An additional moving time window average (1.4 hours) was applied to smooth numerical errors in the derivative when determining feeding and growth regime transition points. Model derivations can be found in the Extended Data file.</p>
</div>
<div id="contributions-4" class="section level2 unnumbered">
<h2>Contributions</h2>
<p>Sasha Shirman: Methodology, Formal Analysis, Visualization</p>
<p>Joy Nyaanga: Investigation, Data curation, Writing - original draft</p>
<p>Erik C. Andersen and Niall M. Mangan: Conceptualization, Funding acquisition, Supervision</p>
</div>
<div id="references-2" class="section level2 unnumbered">
<h2>References</h2>
<ol style="list-style-type: decimal">
<li><p>Nyaanga J, Goss C, Zhang G, Ahmed HN, Andersen EJ, Miller IR, Rozenich JK, Swarthout IL, Vaughn JA, Mangan NM, Shirman S, Andersen EC. Changes in body shape implicate cuticle stretch in <em>C. elegans</em> growth control. bioRxiv. 2022. p. 2021.04.01.438121. Available from: <a href="https://www.biorxiv.org/content/10.1101/2021.04.01.438121v3" class="uri">https://www.biorxiv.org/content/10.1101/2021.04.01.438121v3</a></p></li>
<li><p>Uppaluri S, Brangwynne CP. A size threshold governs <em>Caenorhabditis elegans</em> developmental progression. Proc Biol Sci. 2015 Aug 22;282(1813):20151283. Available from: <a href="http://dx.doi.org/10.1098/rspb.2015.1283" class="uri">http://dx.doi.org/10.1098/rspb.2015.1283</a> PMCID: PMC4632629</p></li>
<li><p>Fang-Yen C, Avery L, Samuel ADT. Two size-selective mechanisms specifically trap bacteria-sized food particles in <em>Caenorhabditis elegans</em>. Proc Natl Acad Sci U S A. 2009 Nov 24;106(47):20093–20096. Available from: <a href="http://dx.doi.org/10.1073/pnas.0904036106" class="uri">http://dx.doi.org/10.1073/pnas.0904036106</a> PMCID: PMC2785297</p></li>
<li><p>Singh RN, Sulston JE. Some Observations On Moulting in <em>Caenorhabditis Elegans</em>. Nematologica. Brill; 1978 Jan 1;24(1):63–71. Available from: <a href="https://brill.com/view/journals/nema/24/1/article-p63_7.xml" class="uri">https://brill.com/view/journals/nema/24/1/article-p63_7.xml</a></p></li>
<li><p>Byerly L, Cassada RC, Russell RL. The life cycle of the nematode <em>Caenorhabditis elegans</em>. I. Wild-type growth and reproduction. Dev Biol. 1976 Jul 1;51(1):23–33. Available from: <a href="http://dx.doi.org/10.1016/0012-1606(76)90119-6" class="uri">http://dx.doi.org/10.1016/0012-1606(76)90119-6</a> PMID: 988845</p></li>
<li><p>Avery L. Food transport in the <em>C. elegans</em> pharynx. Journal of Experimental Biology. 2003. p. 2441–2457. Available from: <a href="http://dx.doi.org/10.1242/jeb.00433" class="uri">http://dx.doi.org/10.1242/jeb.00433</a></p></li>
<li><p>Cook DE, Zdraljevic S, Roberts JP, Andersen EC. CeNDR, the <em>Caenorhabditis elegans</em> natural diversity resource. Nucleic Acids Res [Internet]. 2017 Jan 4;45(D1):D650–D657. Available from: <a href="http://dx.doi.org/10.1093/nar/gkw893" class="uri">http://dx.doi.org/10.1093/nar/gkw893</a> PMCID: PMC5210618</p></li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": false
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-thesis.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
