<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Discussion | Mechanisms of Larval Growth Control in Caenorhabditis elegans</title>
  <meta name="description" content="Nyaanga PhD Dissertation" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Discussion | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/img/NU_seal.png" />
  <meta property="og:description" content="Nyaanga PhD Dissertation" />
  <meta name="github-repo" content="jnyaanga" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Discussion | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  
  <meta name="twitter:description" content="Nyaanga PhD Dissertation" />
  <meta name="twitter:image" content="/img/NU_seal.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"/>
<link rel="next" href="references.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="">Nyaanga Dissertation</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="dedication.html"><a href="dedication.html"><i class="fa fa-check"></i>Dedication</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#Section1.1"><i class="fa fa-check"></i><b>1.1</b> Phenotypic variation</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#from-genotype-to-phenotype"><i class="fa fa-check"></i><b>1.1.1</b> From genotype to phenotype</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#Section1.2"><i class="fa fa-check"></i><b>1.2</b> Quantitative trait loci mapping</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#Section1.3"><i class="fa fa-check"></i><b>1.3</b> <em>Caenorhabditis elegans</em></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#as-a-model-organism"><i class="fa fa-check"></i><b>1.3.1</b> As a model organism</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#post-embryonic-development"><i class="fa fa-check"></i><b>1.3.2</b> Post-embryonic development</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#growth-regulation"><i class="fa fa-check"></i><b>1.3.3</b> Growth regulation</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#natural-genetic-variation"><i class="fa fa-check"></i><b>1.3.4</b> Natural genetic variation</a></li>
<li class="chapter" data-level="1.3.5" data-path="introduction.html"><a href="introduction.html#linkage-mapping"><i class="fa fa-check"></i><b>1.3.5</b> Linkage mapping</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><i class="fa fa-check"></i><b>2</b> A role for cuticle stretch in developmental growth control</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#preface"><i class="fa fa-check"></i><b>2.1</b> Preface</a></li>
<li class="chapter" data-level="2.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#abstract-1"><i class="fa fa-check"></i><b>2.2</b> Abstract</a></li>
<li class="chapter" data-level="2.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#introduction-1"><i class="fa fa-check"></i><b>2.3</b> Introduction</a></li>
<li class="chapter" data-level="2.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#materials-and-methods"><i class="fa fa-check"></i><b>2.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#worm-culture"><i class="fa fa-check"></i><b>2.4.1</b> Worm culture</a></li>
<li class="chapter" data-level="2.4.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#bacterial-food"><i class="fa fa-check"></i><b>2.4.2</b> Bacterial food</a></li>
<li class="chapter" data-level="2.4.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#growth-of-the-animals"><i class="fa fa-check"></i><b>2.4.3</b> Growth of the animals</a></li>
<li class="chapter" data-level="2.4.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#high-throughput-measurements-of-body-size-and-fluorescence"><i class="fa fa-check"></i><b>2.4.4</b> High-throughput measurements of body size and fluorescence</a></li>
<li class="chapter" data-level="2.4.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#image-processing"><i class="fa fa-check"></i><b>2.4.5</b> Image processing</a></li>
<li class="chapter" data-level="2.4.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#data-processing"><i class="fa fa-check"></i><b>2.4.6</b> Data processing</a></li>
<li class="chapter" data-level="2.4.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#molt-analysis"><i class="fa fa-check"></i><b>2.4.7</b> Molt analysis</a></li>
<li class="chapter" data-level="2.4.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#comparison-of-model-fits"><i class="fa fa-check"></i><b>2.4.8</b> Comparison of model fits</a></li>
<li class="chapter" data-level="2.4.9" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#stretcher-model-analysis"><i class="fa fa-check"></i><b>2.4.9</b> Stretcher model analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#results"><i class="fa fa-check"></i><b>2.5</b> Results</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#quantitative-measurements-of-c.-elegans-growth"><i class="fa fa-check"></i><b>2.5.1</b> Quantitative measurements of <em>C. elegans</em> growth</a></li>
<li class="chapter" data-level="2.5.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#fluorescence-provides-a-quantitative-measurement-of-animal-feeding-behavior-and-developmental-progression"><i class="fa fa-check"></i><b>2.5.2</b> Fluorescence <strong>provides a quantitative measurement of animal feeding behavior and developmental progression</strong></a></li>
<li class="chapter" data-level="2.5.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#changes-in-c.-elegans-body-shape-occur-at-larval-stage-transitions"><i class="fa fa-check"></i><b>2.5.3</b> Changes in <em>C. elegans</em> body shape occur at larval-stage transitions</a></li>
<li class="chapter" data-level="2.5.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#modeling-c.-elegans-cuticle-stretch-dynamics"><i class="fa fa-check"></i><b>2.5.4</b> Modeling <em>C. elegans</em> cuticle stretch dynamics</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#discussion"><i class="fa fa-check"></i><b>2.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#a-potential-role-for-cuticle-stretch-in-the-timing-of-larval-stage-transitions"><i class="fa fa-check"></i><b>2.6.1</b> A potential role for cuticle stretch in the timing of larval-stage transitions</a></li>
<li class="chapter" data-level="2.6.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#development-comprises-complex-interactions-of-growth-regulation-across-diverse-scales"><i class="fa fa-check"></i><b>2.6.2</b> Development comprises complex interactions of growth regulation across diverse scales</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#contributions"><i class="fa fa-check"></i><b>2.7</b> Contributions</a></li>
<li class="chapter" data-level="2.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#supplement"><i class="fa fa-check"></i><b>2.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><i class="fa fa-check"></i><b>3</b> Multiple genetic loci underlie differences in <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="3.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#preface-1"><i class="fa fa-check"></i><b>3.1</b> Preface</a></li>
<li class="chapter" data-level="3.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#abstract-2"><i class="fa fa-check"></i><b>3.2</b> Abstract</a></li>
<li class="chapter" data-level="3.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#introduction-2"><i class="fa fa-check"></i><b>3.3</b> Introduction</a></li>
<li class="chapter" data-level="3.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#materials-and-methods-1"><i class="fa fa-check"></i><b>3.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#strains"><i class="fa fa-check"></i><b>3.4.1</b> Strains</a></li>
<li class="chapter" data-level="3.4.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-growth-assay"><i class="fa fa-check"></i><b>3.4.2</b> High-throughput growth assay</a></li>
<li class="chapter" data-level="3.4.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-fitness-assay-for-linkage-mapping"><i class="fa fa-check"></i><b>3.4.3</b> High-throughput fitness assay for linkage mapping</a></li>
<li class="chapter" data-level="3.4.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#linkage-mapping-1"><i class="fa fa-check"></i><b>3.4.4</b> Linkage mapping</a></li>
<li class="chapter" data-level="3.4.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#generation-of-chromosome-substitution-strains-csss-and-near-isogenic-lines-nils"><i class="fa fa-check"></i><b>3.4.5</b> Generation of chromosome substitution strains (CSSs) and near-isogenic lines (NILs)</a></li>
<li class="chapter" data-level="3.4.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#statistical-analysis-of-css-and-nil-results"><i class="fa fa-check"></i><b>3.4.6</b> Statistical analysis of CSS and NIL results</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#results-1"><i class="fa fa-check"></i><b>3.5</b> Results</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#growth-dynamics-of-the-n2-and-cb4856-strains"><i class="fa fa-check"></i><b>3.5.1</b> Growth dynamics of the N2 and CB4856 strains</a></li>
<li class="chapter" data-level="3.5.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-qtl-underlying-variation-in-growth"><i class="fa fa-check"></i><b>3.5.2</b> Identification of QTL underlying variation in growth</a></li>
<li class="chapter" data-level="3.5.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#validation-of-loci-associated-with-differences-in-animal-size"><i class="fa fa-check"></i><b>3.5.3</b> Validation of loci associated with differences in animal size</a></li>
<li class="chapter" data-level="3.5.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-candidate-genes-in-the-chromosome-x-qtl"><i class="fa fa-check"></i><b>3.5.4</b> Identification of candidate genes in the chromosome X QTL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#discussion-1"><i class="fa fa-check"></i><b>3.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#a-complex-genetic-architecture-underlies-differences-in-body-size"><i class="fa fa-check"></i><b>3.6.1</b> A complex genetic architecture underlies differences in body size</a></li>
<li class="chapter" data-level="3.6.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#candidate-genes-for-variation-in-body-size"><i class="fa fa-check"></i><b>3.6.2</b> Candidate genes for variation in body size</a></li>
<li class="chapter" data-level="3.6.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#comparison-with-previous-qtl-studies-of-c.-elegans-growth"><i class="fa fa-check"></i><b>3.6.3</b> Comparison with previous QTL studies of <em>C. elegans</em> growth</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#contributions-1"><i class="fa fa-check"></i><b>3.7</b> Contributions</a></li>
<li class="chapter" data-level="3.8" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#supplement-1"><i class="fa fa-check"></i><b>3.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><i class="fa fa-check"></i><b>4</b> easyXpress: An R package to analyze and visualize <em>C. elegans</em> microscopy data generated using CellProfiler</a>
<ul>
<li class="chapter" data-level="4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preface-2"><i class="fa fa-check"></i><b>4.1</b> Preface</a></li>
<li class="chapter" data-level="4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#abstract-3"><i class="fa fa-check"></i><b>4.2</b> Abstract</a></li>
<li class="chapter" data-level="4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#introduction-3"><i class="fa fa-check"></i><b>4.3</b> Introduction</a></li>
<li class="chapter" data-level="4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#materials-and-methods-2"><i class="fa fa-check"></i><b>4.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preparation-of-animals-for-imaging"><i class="fa fa-check"></i><b>4.4.1</b> Preparation of animals for imaging</a></li>
<li class="chapter" data-level="4.4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#imaging"><i class="fa fa-check"></i><b>4.4.2</b> Imaging</a></li>
<li class="chapter" data-level="4.4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#paraquat-dose-response"><i class="fa fa-check"></i><b>4.4.3</b> Paraquat dose response</a></li>
<li class="chapter" data-level="4.4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#naming-conventions"><i class="fa fa-check"></i><b>4.4.4</b> Naming conventions</a></li>
<li class="chapter" data-level="4.4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-availability"><i class="fa fa-check"></i><b>4.4.5</b> Data availability</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#results-2"><i class="fa fa-check"></i><b>4.5</b> Results</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#design-and-implementation"><i class="fa fa-check"></i><b>4.5.1</b> Design and implementation</a></li>
<li class="chapter" data-level="4.5.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-import-and-model-assignment"><i class="fa fa-check"></i><b>4.5.2</b> Data import and model assignment</a></li>
<li class="chapter" data-level="4.5.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-pruning-and-summarization"><i class="fa fa-check"></i><b>4.5.3</b> Data pruning and summarization</a></li>
<li class="chapter" data-level="4.5.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#visualization"><i class="fa fa-check"></i><b>4.5.4</b> Visualization</a></li>
<li class="chapter" data-level="4.5.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#application-to-c.-elegans-growth-data"><i class="fa fa-check"></i><b>4.5.5</b> Application to <em>C. elegans</em> growth data</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#conclusions"><i class="fa fa-check"></i><b>4.6</b> Conclusions</a></li>
<li class="chapter" data-level="4.7" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#contributions-2"><i class="fa fa-check"></i><b>4.7</b> Contributions</a></li>
<li class="chapter" data-level="4.8" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#supplement-2"><i class="fa fa-check"></i><b>4.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion-2.html"><a href="discussion-2.html"><i class="fa fa-check"></i><b>5</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion-2.html"><a href="discussion-2.html#summary"><i class="fa fa-check"></i><b>5.1</b> Summary</a></li>
<li class="chapter" data-level="5.2" data-path="discussion-2.html"><a href="discussion-2.html#the-future-of-developmental-growth-studies"><i class="fa fa-check"></i><b>5.2</b> The future of developmental growth studies</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="discussion-2.html"><a href="discussion-2.html#the-c.-elegans-cuticle-and-its-role-in-developmental-timing"><i class="fa fa-check"></i><b>5.2.1</b> The <em>C. elegans</em> cuticle and its role in developmental timing</a></li>
<li class="chapter" data-level="5.2.2" data-path="discussion-2.html"><a href="discussion-2.html#identifying-genetic-factors-that-contribute-to-variation-in-growth-and-body-size"><i class="fa fa-check"></i><b>5.2.2</b> Identifying genetic factors that contribute to variation in growth and body size</a></li>
<li class="chapter" data-level="5.2.3" data-path="discussion-2.html"><a href="discussion-2.html#phenotypic-plasticity-and-genotype-by-environment-interaction"><i class="fa fa-check"></i><b>5.2.3</b> Phenotypic plasticity and genotype-by-environment interaction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
<li class="chapter" data-level="7" data-path="appendix.html"><a href="appendix.html"><i class="fa fa-check"></i><b>7</b> Appendix</a></li>
<li><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#a.-growth-dynamics-of-c.-elegans-cuticle-mutants">A. Growth dynamics of <em>C. elegans</em> cuticle mutants</a>
<ul>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#description"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#methods"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#contributions-3"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li><a href="b.-food-allocation-during-c.-elegans-growth.html#b.-food-allocation-during-c.-elegans-growth">B. Food allocation during <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#methods-1"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#contributions-4"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Mechanisms of Larval Growth Control in <em>Caenorhabditis elegans</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="discussion-2" class="section level1" number="5">
<h1><span class="header-section-number">5</span> Discussion</h1>
<p>Extensive phenotypic diversity exists in the animal kingdom. A fundamental objective in biological research is to decipher the source of this diversity and study its impact on individuals, populations, and species. Body size, in particular, is a trait that varies strikingly across animal phylogeny, and has critical ramifications on an animal’s ability to survive and reproduce. For this reason, scientists aim to understand how organisms grow to their characteristic sizes. Organismal development comprises a sequence of genetically programmed and intricately linked events that follow precise temporal and spatial ordering. Remarkably, despite variation in environmental conditions (e.g. temperature, nutrient availability, etc), developing systems coordinate the organization and interaction among cells, tissues, and organs with high reproducibility. My graduate work is focused on investigating mechanisms of growth regulation in developing <em>Caenorhabditis elegans</em>. Here, I will elaborate on the implications of my data and provide insights that will hopefully propel future research.</p>
<div id="summary" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Summary</h2>
<p>This dissertation aimed to present mechanisms by which developmental growth may be controlled using the roundworm nematode <em>Caenorhabditis elegans</em>. Studying <em>C. elegans</em> larval development is relatively straightforward, however, doing so at high temporal resolution while maintaining high precision and throughput can be a challenge. In <strong>Chapter 2</strong>, I introduce a high-throughput phenotyping platform I optimized to facilitate the quantitative assessment of size and feeding behavior of thousands of individuals under tightly controlled environment conditions. Using this platform to collect growth data over time, I find that as animals transitioned from one developmental stage to the next, changes in body shape occurred. Given this result, we worked with mathematical scientists to model a physical mechanism by which constraints on cuticle elasticity could cause changes in <em>C. elegans</em> body shape, and found the model-predicted shape changes to be consistent with those observed in the experimental data. This work puts forth a novel “Stretcher” mechanism for how <em>C. elegans</em> could use physical constraints (i.e. restriction of cuticle stretch) as a sensor for growth and as a means to regulate developmental timing. Noting the importance of the cuticle in <em>C. elegans</em> growth, I further examine the larval development of three strains with mutations in distinct cuticle collagen proteins in <strong>Appendix A</strong>. Two of the three mutant strains exhibit delayed growth and maintain a consistently shorter and wider shape when compared to wild type, emphasizing the importance of cuticle structure in development. Importantly, all three strains exhibited dynamics consistent with a “Stretcher” growth model.</p>
<p>The process of development is highly reproducible even in the presence of genetic and environmental perturbations. In <strong>Chapter 3</strong>, I leverage <em>C. elegans</em> natural genetic diversity to explore how genetic variation contributes to differences in animal growth. Using a quantitative genetic mapping approach, I identify distinct genomic regions that separately explain variation in animal length and width. This work highlights the complex genetic architecture of body size, identifies promising candidate genes that potentially underlie variation in animal width, and provides a framework for future studies to investigate genetic mechanisms controlling developmental growth and body size in natural populations of <em>C. elegans</em>.</p>
<p>Like many high-throughput imaging techniques, the experimental platform introduced in Chapter 2 generates large quantities of data that can be difficult to process and visualize efficiently. In <strong>Chapter 4</strong>, I discuss the development of easyXpress: an R package that provides tools for the reading, processing, and visualization of <em>C. elegans</em> microscopy data in a simple and efficient way. This software package enables reproducible analysis, integration with other statistical R packages, and extensibility to many research projects using an open-source analysis pipeline.</p>
</div>
<div id="the-future-of-developmental-growth-studies" class="section level2" number="5.2">
<h2><span class="header-section-number">5.2</span> The future of developmental growth studies</h2>
<div id="the-c.-elegans-cuticle-and-its-role-in-developmental-timing" class="section level3" number="5.2.1">
<h3><span class="header-section-number">5.2.1</span> The <em>C. elegans</em> cuticle and its role in developmental timing</h3>
<p>Molting is a unique characteristic defining the life cycles of many animal species including, arthropods (insects, arachnids, crustaceans), nematodes (roundworms), and other members of the Ecdysozoa group <span class="citation">[<a href="references.html#ref-Aguinaldo1997-ig" role="doc-biblioref">184</a>,<a href="references.html#ref-Telford2008-ya" role="doc-biblioref">185</a>]</span>. Despite its prevalence, the primary function of molting varies across these different groups. Arthropods, covered by rigid chitin-based cuticles, require molting for body size expansion <span class="citation">[<a href="references.html#ref-Alpatov1929-li" role="doc-biblioref">186</a>,<a href="references.html#ref-Rice1968-em" role="doc-biblioref">187</a>]</span>. By contrast, the nematode body is encased in a flexible collagen-based cuticle that allows for growth and expansion between molts <span class="citation">[<a href="references.html#ref-Howells1983-vf" role="doc-biblioref">188</a>,<a href="references.html#ref-Wilson1976-bc" role="doc-biblioref">189</a>]</span>, leaving the question: <em>Does the <em>C. elegans</em> cuticle play a role in developmental growth and molting?</em></p>
<p>Few longitudinal studies of <em>C. elegans</em> growth have been performed. Initial research described <em>C. elegans</em> development as a sigmoidal curve characterized by continuous larval growth <span class="citation">[<a href="references.html#ref-Byerly1976-xt" role="doc-biblioref">107</a>,<a href="references.html#ref-Malakhov1986-nb" role="doc-biblioref">190</a>]</span>. These early studies reasoned that molt events had little effect on continuous growth as the <em>C. elegans</em> flexible cuticle allowed for stretch during development. Later work clarified that growth rate was not continuous but rather piecewise in nature and increased from stage to stage, suggesting that <em>C. elegans</em> may contain a mechanism for regulating growth rate, potentially through each molt event <span class="citation">[<a href="references.html#ref-Knight2002-fb" role="doc-biblioref">108</a>]</span>. Next, researchers found that animals only advance to the next developmental stage once a critical size is reached, implying that each molt decision is controlled by a size threshold <span class="citation">[<a href="references.html#ref-Uppaluri2016-gm" role="doc-biblioref">191</a>]</span>. Recent work shows that <em>C. elegans</em> development is likely controlled not by size-dependent regulation of growth but rather by relative change, or fold change, in body volume, suggesting that animals trigger molting events in ways that maintain a constant fold change in body volume <span class="citation">[<a href="references.html#ref-Towbin2021-vx" role="doc-biblioref">99</a>]</span>. However, two questions remain: <em>How do animals sense their body size, and most importantly, how is this information disseminated to precisely regulate growth?</em></p>
<p>In Chapter 2, I have suggested a novel mechanism by which physical constraints can influence developmental timing and growth rate in <em>C. elegans</em>. Although elasticity of the cuticle permits growth during each larval stage, cuticle “stretchiness” is limited. By sensing the reduction of cuticle elasticity, <em>C. elegans</em> could use this mechanical signal to trigger molting events. To validate our proposed mechanism, we must first determine whether cuticle elasticity changes during a larval stage. Previous atomic force microscopy (AFM) studies of <em>C. elegans</em> suggest that animals experience a loss of stiffness as they age <span class="citation">[<a href="references.html#ref-Essmann2020-cb" role="doc-biblioref">192</a>]</span>, however no studies have been performed on developing animals. Capturing AFM images of the cuticle over the course of a single larval stage would be a potential method to investigate cuticle stiffness during growth. AFM experiments, however, are limited in that they apply small highly localized indentation forces that may not adequately represent the large stretching loads experienced during growth. To confirm elasticity changes in the cuticle, direct measurements of the stiffness of isolated cuticles is essential. Recently published work using microfluidic technology to stretch isolated cuticles found significant increases in longitudinal stiffness at large stretching loads <span class="citation">[<a href="references.html#ref-Rahimi2022-ur" role="doc-biblioref">193</a>]</span>. Performing similar tests to isolated cuticles of animals preceding and following a molt could provide support for our hypothesis that molting occurs as a result of decreased cuticle elasticity. Additionally, if we find that cuticle elasticity changes during development, it could be interesting to collect growth measurements of animals cultured under varying osmotic conditions. Given our stretch-based growth model, I would expect to observe accelerated developmental timing in animals exposed to hypoosmotic conditions because of the relatively high stretching loads on the cuticle throughout growth.</p>
<p>Identifying changes in cuticle elasticity during growth is just one part of understanding the role the cuticle may play in molt timing. We must also consider how information originating from the cuticle is sensed and propagated to the rest of the animal. In <em>C. elegans</em>, attachment complexes called hemidesmosomes connect the cuticle to the epidermis and are shown to respond to mechanical tension exerted by muscle contraction <span class="citation">[<a href="references.html#ref-Zhang2011-ea" role="doc-biblioref">137</a>]</span>. It is possible that <em>C. elegans</em> hemidesmosomes act not only as an attachment structure but also as a mechanosensor that responds to forces at the epidermis and at the cuticle. Additionally, motor neurons may serve as another potential mechanosensor. Mechanosensory neurons are typically used to regulate various behaviors including, locomotion, egg laying, pharyngeal pumping, and defecation <span class="citation">[<a href="references.html#ref-Riddle1997-jv" role="doc-biblioref">194</a>]</span>. Notably, many motor neurons extend along the longitudinal body axis and some also extend circumferential axons <span class="citation">[<a href="references.html#ref-Altun2005-jf" role="doc-biblioref">195</a>]</span>, opening up the possibility that these neurons might provide signals proportional to body length and width. Future studies of how perturbations of hemidesmosomes and motor neurons influence <em>C. elegans</em> body size and developmental timing would be worthwhile.</p>
</div>
<div id="identifying-genetic-factors-that-contribute-to-variation-in-growth-and-body-size" class="section level3" number="5.2.2">
<h3><span class="header-section-number">5.2.2</span> Identifying genetic factors that contribute to variation in growth and body size</h3>
<p>Discerning how complex traits are genetically controlled is essential to understanding the evolution of phenotypic diversity. Genetic regulators of trait variation can be mapped using QTL analysis of RIAILs derived from genetically and phenotypically divergent strains. In my work, I used linkage mapping to identify genetic loci underlying differences in <em>C. elegans</em> body size, and found three small-effect QTL explaining 5-8% of variation in the RIAIL phenotypes. Our ability to detect and validate one of these QTL emphasizes the power of using <em>C. elegans</em> to correlate small-effect genetic loci to quantitative phenotypes. In Chapter 3, I postulate that the inability to recapitulate the other two QTL effects could be attributed to insufficient power or the presence of additional undetected QTL with additive or non-additive effects. The CSSs and NILs I used for validation experiments each contained a single region of introgression in a genetic background of the opposite genotype. Future work should aim to construct double CSSs or multi-region NILs where pairwise combinations of two genomic regions are introgressed in the opposite genotype. Phenotyping these additional strains could reveal additional loci contributing to the chromosome IV and V effects.</p>
<div id="alternatives-to-classical-linkage-mapping" class="section level4" number="5.2.2.1">
<h4><span class="header-section-number">5.2.2.1</span> Alternatives to classical linkage mapping</h4>
<p><strong><em>Genome-wide association (GWA) mapping</em></strong></p>
<p>High levels of genotypic and phenotypic diversity exist in natural populations. In contrast to linkage mapping where genetic diversity is limited to the two parental strains, GWA mapping leverages the existing natural diversity found in a population. By correlating phenotypic variation among wild strains with whole-genome data, GWA studies identify functional variants that contribute to phenotypic diversity. In <em>C. elegans</em>, a growing collection of wild strains and genome-wide variation data are readily available (CeNDR, <span class="citation">[<a href="references.html#ref-Cook2017-zq" role="doc-biblioref">69</a>]</span> ). Using this resource, association mapping has led to the discovery of genes and variants that underlie variation in complex traits (<strong>Figure 1-4</strong>) <span class="citation">[<a href="references.html#ref-Evans2021-zr" role="doc-biblioref">21</a>]</span>.</p>
<p>Recently, the Andersen Lab completed an extensive GWA experiment, collecting body length measurements for 195 wild strains across 54 different drug conditions. Exploring this data to identify size-associated QTL of animals exposed to the control condition (water), and comparing the results to the linkage mapping presented in Chapter 3, would be the next step for subsequent investigations of genetic factors that underlie differences in <em>C. elegans</em> growth and body size. With data from the linkage and association mappings, we could look for QTL that overlap between the two methods. Overlapping QTL would suggest that a common variant present in the CB4856 strain contributes to differences in body size of animals in both mapping populations. Alternatively, non-overlapping QTL would suggest that the QTL identified from linkage mapping is driven by a rare variant in the CB4856 strain, whereas the QTL identified from association mapping is driven by a common variant not found in the CB4856 strain. The combination of both association and linkage mapping provides an approach to both narrow genomic intervals and generate refined lists of potential causal genes.</p>
<p><strong><em>Bulk-segregant analysis</em></strong></p>
<p>QTL mapping has been central to the identification of loci underlying complex traits in <em>C. elegans</em>. However, QTL mapping typically requires the generation of large panels of cross progeny that must be individually genotyped and phenotyped, a lengthy and often costly task. An alternative to traditional QTL mapping is bulk-segregant analysis (BSA). In contrast to constructing, sequencing, and phenotyping individuals for a RIL/RIAIL panel, the BSA approach uses pools of recombinant individuals generated from genetically and phenotypically divergent parents. Recombinant individuals are then selected based on the phenotype of interest and allele frequencies are compared between the unselected and selected pools. Over recent years, researchers have demonstrated the power of using BSA to connect phenotypic differences to specific genes (<strong>Figure 1-4</strong>) <span class="citation">[<a href="references.html#ref-Evans2021-zr" role="doc-biblioref">21</a>]</span>. Leveraging BSA to identify QTL associated with developmental growth would be a worthwhile endeavor. Recombinant individuals from an N2xCB4856 cross could be cultured as described in Chapter 2. Then by selecting the tails of the population distribution, the allele frequency changes of the “large-sized” and “small-sized” individuals during the experiment can be compared. Repeating this over various stages during development could potentially reveal unique QTL for independent larval stages. Alternatively, one could make use of population-wide diversity by performing growth and selection on a pool of wild isolates. The application of a powerful mapping method like BSA to identify QTL will expand our understanding of the molecular mechanisms underlying <em>C. elegans</em> growth and development.</p>
</div>
</div>
<div id="phenotypic-plasticity-and-genotype-by-environment-interaction" class="section level3" number="5.2.3">
<h3><span class="header-section-number">5.2.3</span> Phenotypic plasticity and genotype-by-environment interaction</h3>
<p>In my graduate work I study the growth and development of <em>C. elegans</em> under tightly controlled environmental conditions. However, environmental factors in natural habitats are rarely constant, and this variation can impact developmental progression and growth rate. Research has shown that <em>C. elegans</em> rearing temperature, as well as both food quality and quantity alter the speed of larval development <span class="citation">[<a href="references.html#ref-MacNeil2013-pk" role="doc-biblioref">58</a>,<a href="references.html#ref-Stuhr2020-vr" role="doc-biblioref">60</a>,<a href="references.html#ref-Altun2006-zw" role="doc-biblioref">196</a>]</span>. The term phenotypic plasticity is used to describe this phenomenon where an individual can generate multiple phenotypes based on external conditions <span class="citation">[<a href="references.html#ref-Pigliucci2006-gf" role="doc-biblioref">197</a>]</span>. Notably, distinct genotypes of a single species can show unique patterns of plasticity. Genotype-by-environment (GxE) interactions represent this contribution of genetic variation to environmental sensitivity.</p>
<p>Reaction norms are often used to visually compare plasticity among different genotypes by displaying phenotype differences as a function of environmental change <span class="citation">[<a href="references.html#ref-Lafuente2019-up" role="doc-biblioref">198</a>]</span>. A phenotype may be constant across genotypes and environments, or it may be affected in several ways. First, phenotype could differ between genotypes, indicating genetic variation, but remain unaffected across multiple environments. Displayed as horizontal lines between environmental parameters, this case represents robustness to environmental perturbation (<strong>Figure 5-1A</strong>). Second, regardless of genotype, there could be a general plastic response to environmental change. Here, diagonal lines between environmental conditions reflect plasticity and the slope of the line represents the magnitude of plasticity (<strong>Figure 5-1B</strong>). Third, the magnitude of plasticity for a given trait could vary among genotypes, represented as non-parallel reaction norms (<strong>Figure 5-1C</strong>), indicating the presence of GxE interaction. Although the occurrence of GxE interactions is well studied, the genomic regions involved are not because of the complex interaction between genetics and environmental factors that drives plasticity <span class="citation">[<a href="references.html#ref-Pigliucci2005-te" role="doc-biblioref">199</a>–<a href="references.html#ref-Rocha2009-zc" role="doc-biblioref">201</a>]</span>.</p>
<p><img src="img/fig5-1.png" width="100%" style="display: block; margin: auto;" /></p>
<p><strong>Figure 5-1.</strong> Schematic representation of reaction norms, illustrating various forms of plasticity.<br />
</p>
<p>In the last month, research was published that explored the genetic architecture of temperature affected body-size plasticity in <em>C. elegans</em> <span class="citation">[<a href="references.html#ref-Maulana2022-jy" role="doc-biblioref">171</a>]</span>. Researchers used 40 RILs derived from the N2 and CB4856 strains to investigate the effects of temperature and developmental stage on the reaction norms of various body-size traits (animal length, width, and size of some internal organs). Not only did they find clear evidence of GxE interactions, but they also noticed that the shape of the reaction norm was affected by developmental stage, potentially stemming from differences in interaction between stage-specific gene expression patterns and environmental temperature. Authors also performed QTL mapping and found 18 loci underlying body-size traits and only five plasticity loci. Interestingly, three plasticity QTL colocalized with body-size QTL, suggesting the possibility of co-regulatory loci influencing both plasticity and the traits themselves. Future work should build on these findings by collecting trait measurements for wild <em>C. elegans</em> strains under different environmental conditions. This research is vital to our understanding of the complex interplay between genetic mechanisms and environmental forces that shape phenotypic variation in natural populations.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": false
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-thesis.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
