<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>1 Introduction | Mechanisms of Larval Growth Control in Caenorhabditis elegans</title>
  <meta name="description" content="Nyaanga PhD Dissertation" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="1 Introduction | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/img/NU_seal.png" />
  <meta property="og:description" content="Nyaanga PhD Dissertation" />
  <meta name="github-repo" content="jnyaanga" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="1 Introduction | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  
  <meta name="twitter:description" content="Nyaanga PhD Dissertation" />
  <meta name="twitter:image" content="/img/NU_seal.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="acknowledgements.html"/>
<link rel="next" href="a-role-for-cuticle-stretch-in-developmental-growth-control.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="">Nyaanga Dissertation</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="dedication.html"><a href="dedication.html"><i class="fa fa-check"></i>Dedication</a></li>
<li class="chapter" data-level="" data-path="acknowledgements.html"><a href="acknowledgements.html"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#Section1.1"><i class="fa fa-check"></i><b>1.1</b> Phenotypic variation</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#from-genotype-to-phenotype"><i class="fa fa-check"></i><b>1.1.1</b> From genotype to phenotype</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#Section1.2"><i class="fa fa-check"></i><b>1.2</b> Quantitative trait loci mapping</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#Section1.3"><i class="fa fa-check"></i><b>1.3</b> <em>Caenorhabditis elegans</em></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#as-a-model-organism"><i class="fa fa-check"></i><b>1.3.1</b> As a model organism</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#post-embryonic-development"><i class="fa fa-check"></i><b>1.3.2</b> Post-embryonic development</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#growth-regulation"><i class="fa fa-check"></i><b>1.3.3</b> Growth regulation</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#natural-genetic-variation"><i class="fa fa-check"></i><b>1.3.4</b> Natural genetic variation</a></li>
<li class="chapter" data-level="1.3.5" data-path="introduction.html"><a href="introduction.html#linkage-mapping"><i class="fa fa-check"></i><b>1.3.5</b> Linkage mapping</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><i class="fa fa-check"></i><b>2</b> A role for cuticle stretch in developmental growth control</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#preface"><i class="fa fa-check"></i><b>2.1</b> Preface</a></li>
<li class="chapter" data-level="2.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#abstract-1"><i class="fa fa-check"></i><b>2.2</b> Abstract</a></li>
<li class="chapter" data-level="2.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#introduction-1"><i class="fa fa-check"></i><b>2.3</b> Introduction</a></li>
<li class="chapter" data-level="2.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#materials-and-methods"><i class="fa fa-check"></i><b>2.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#worm-culture"><i class="fa fa-check"></i><b>2.4.1</b> Worm culture</a></li>
<li class="chapter" data-level="2.4.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#bacterial-food"><i class="fa fa-check"></i><b>2.4.2</b> Bacterial food</a></li>
<li class="chapter" data-level="2.4.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#growth-of-the-animals"><i class="fa fa-check"></i><b>2.4.3</b> Growth of the animals</a></li>
<li class="chapter" data-level="2.4.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#high-throughput-measurements-of-body-size-and-fluorescence"><i class="fa fa-check"></i><b>2.4.4</b> High-throughput measurements of body size and fluorescence</a></li>
<li class="chapter" data-level="2.4.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#image-processing"><i class="fa fa-check"></i><b>2.4.5</b> Image processing</a></li>
<li class="chapter" data-level="2.4.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#data-processing"><i class="fa fa-check"></i><b>2.4.6</b> Data processing</a></li>
<li class="chapter" data-level="2.4.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#molt-analysis"><i class="fa fa-check"></i><b>2.4.7</b> Molt analysis</a></li>
<li class="chapter" data-level="2.4.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#comparison-of-model-fits"><i class="fa fa-check"></i><b>2.4.8</b> Comparison of model fits</a></li>
<li class="chapter" data-level="2.4.9" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#stretcher-model-analysis"><i class="fa fa-check"></i><b>2.4.9</b> Stretcher model analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#results"><i class="fa fa-check"></i><b>2.5</b> Results</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#quantitative-measurements-of-c.-elegans-growth"><i class="fa fa-check"></i><b>2.5.1</b> Quantitative measurements of <em>C. elegans</em> growth</a></li>
<li class="chapter" data-level="2.5.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#fluorescence-provides-a-quantitative-measurement-of-animal-feeding-behavior-and-developmental-progression"><i class="fa fa-check"></i><b>2.5.2</b> Fluorescence <strong>provides a quantitative measurement of animal feeding behavior and developmental progression</strong></a></li>
<li class="chapter" data-level="2.5.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#changes-in-c.-elegans-body-shape-occur-at-larval-stage-transitions"><i class="fa fa-check"></i><b>2.5.3</b> Changes in <em>C. elegans</em> body shape occur at larval-stage transitions</a></li>
<li class="chapter" data-level="2.5.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#modeling-c.-elegans-cuticle-stretch-dynamics"><i class="fa fa-check"></i><b>2.5.4</b> Modeling <em>C. elegans</em> cuticle stretch dynamics</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#discussion"><i class="fa fa-check"></i><b>2.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#a-potential-role-for-cuticle-stretch-in-the-timing-of-larval-stage-transitions"><i class="fa fa-check"></i><b>2.6.1</b> A potential role for cuticle stretch in the timing of larval-stage transitions</a></li>
<li class="chapter" data-level="2.6.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#development-comprises-complex-interactions-of-growth-regulation-across-diverse-scales"><i class="fa fa-check"></i><b>2.6.2</b> Development comprises complex interactions of growth regulation across diverse scales</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#contributions"><i class="fa fa-check"></i><b>2.7</b> Contributions</a></li>
<li class="chapter" data-level="2.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#supplement"><i class="fa fa-check"></i><b>2.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><i class="fa fa-check"></i><b>3</b> Multiple genetic loci underlie differences in <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="3.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#preface-1"><i class="fa fa-check"></i><b>3.1</b> Preface</a></li>
<li class="chapter" data-level="3.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#abstract-2"><i class="fa fa-check"></i><b>3.2</b> Abstract</a></li>
<li class="chapter" data-level="3.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#introduction-2"><i class="fa fa-check"></i><b>3.3</b> Introduction</a></li>
<li class="chapter" data-level="3.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#materials-and-methods-1"><i class="fa fa-check"></i><b>3.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#strains"><i class="fa fa-check"></i><b>3.4.1</b> Strains</a></li>
<li class="chapter" data-level="3.4.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-growth-assay"><i class="fa fa-check"></i><b>3.4.2</b> High-throughput growth assay</a></li>
<li class="chapter" data-level="3.4.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-fitness-assay-for-linkage-mapping"><i class="fa fa-check"></i><b>3.4.3</b> High-throughput fitness assay for linkage mapping</a></li>
<li class="chapter" data-level="3.4.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#linkage-mapping-1"><i class="fa fa-check"></i><b>3.4.4</b> Linkage mapping</a></li>
<li class="chapter" data-level="3.4.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#generation-of-chromosome-substitution-strains-csss-and-near-isogenic-lines-nils"><i class="fa fa-check"></i><b>3.4.5</b> Generation of chromosome substitution strains (CSSs) and near-isogenic lines (NILs)</a></li>
<li class="chapter" data-level="3.4.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#statistical-analysis-of-css-and-nil-results"><i class="fa fa-check"></i><b>3.4.6</b> Statistical analysis of CSS and NIL results</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#results-1"><i class="fa fa-check"></i><b>3.5</b> Results</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#growth-dynamics-of-the-n2-and-cb4856-strains"><i class="fa fa-check"></i><b>3.5.1</b> Growth dynamics of the N2 and CB4856 strains</a></li>
<li class="chapter" data-level="3.5.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-qtl-underlying-variation-in-growth"><i class="fa fa-check"></i><b>3.5.2</b> Identification of QTL underlying variation in growth</a></li>
<li class="chapter" data-level="3.5.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#validation-of-loci-associated-with-differences-in-animal-size"><i class="fa fa-check"></i><b>3.5.3</b> Validation of loci associated with differences in animal size</a></li>
<li class="chapter" data-level="3.5.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-candidate-genes-in-the-chromosome-x-qtl"><i class="fa fa-check"></i><b>3.5.4</b> Identification of candidate genes in the chromosome X QTL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#discussion-1"><i class="fa fa-check"></i><b>3.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#a-complex-genetic-architecture-underlies-differences-in-body-size"><i class="fa fa-check"></i><b>3.6.1</b> A complex genetic architecture underlies differences in body size</a></li>
<li class="chapter" data-level="3.6.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#candidate-genes-for-variation-in-body-size"><i class="fa fa-check"></i><b>3.6.2</b> Candidate genes for variation in body size</a></li>
<li class="chapter" data-level="3.6.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#comparison-with-previous-qtl-studies-of-c.-elegans-growth"><i class="fa fa-check"></i><b>3.6.3</b> Comparison with previous QTL studies of <em>C. elegans</em> growth</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#contributions-1"><i class="fa fa-check"></i><b>3.7</b> Contributions</a></li>
<li class="chapter" data-level="3.8" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#supplement-1"><i class="fa fa-check"></i><b>3.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><i class="fa fa-check"></i><b>4</b> easyXpress: An R package to analyze and visualize <em>C. elegans</em> microscopy data generated using CellProfiler</a>
<ul>
<li class="chapter" data-level="4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preface-2"><i class="fa fa-check"></i><b>4.1</b> Preface</a></li>
<li class="chapter" data-level="4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#abstract-3"><i class="fa fa-check"></i><b>4.2</b> Abstract</a></li>
<li class="chapter" data-level="4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#introduction-3"><i class="fa fa-check"></i><b>4.3</b> Introduction</a></li>
<li class="chapter" data-level="4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#materials-and-methods-2"><i class="fa fa-check"></i><b>4.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preparation-of-animals-for-imaging"><i class="fa fa-check"></i><b>4.4.1</b> Preparation of animals for imaging</a></li>
<li class="chapter" data-level="4.4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#imaging"><i class="fa fa-check"></i><b>4.4.2</b> Imaging</a></li>
<li class="chapter" data-level="4.4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#paraquat-dose-response"><i class="fa fa-check"></i><b>4.4.3</b> Paraquat dose response</a></li>
<li class="chapter" data-level="4.4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#naming-conventions"><i class="fa fa-check"></i><b>4.4.4</b> Naming conventions</a></li>
<li class="chapter" data-level="4.4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-availability"><i class="fa fa-check"></i><b>4.4.5</b> Data availability</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#results-2"><i class="fa fa-check"></i><b>4.5</b> Results</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#design-and-implementation"><i class="fa fa-check"></i><b>4.5.1</b> Design and implementation</a></li>
<li class="chapter" data-level="4.5.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-import-and-model-assignment"><i class="fa fa-check"></i><b>4.5.2</b> Data import and model assignment</a></li>
<li class="chapter" data-level="4.5.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-pruning-and-summarization"><i class="fa fa-check"></i><b>4.5.3</b> Data pruning and summarization</a></li>
<li class="chapter" data-level="4.5.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#visualization"><i class="fa fa-check"></i><b>4.5.4</b> Visualization</a></li>
<li class="chapter" data-level="4.5.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#application-to-c.-elegans-growth-data"><i class="fa fa-check"></i><b>4.5.5</b> Application to <em>C. elegans</em> growth data</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#conclusions"><i class="fa fa-check"></i><b>4.6</b> Conclusions</a></li>
<li class="chapter" data-level="4.7" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#contributions-2"><i class="fa fa-check"></i><b>4.7</b> Contributions</a></li>
<li class="chapter" data-level="4.8" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#supplement-2"><i class="fa fa-check"></i><b>4.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion-2.html"><a href="discussion-2.html"><i class="fa fa-check"></i><b>5</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion-2.html"><a href="discussion-2.html#summary"><i class="fa fa-check"></i><b>5.1</b> Summary</a></li>
<li class="chapter" data-level="5.2" data-path="discussion-2.html"><a href="discussion-2.html#the-future-of-developmental-growth-studies"><i class="fa fa-check"></i><b>5.2</b> The future of developmental growth studies</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="discussion-2.html"><a href="discussion-2.html#the-c.-elegans-cuticle-and-its-role-in-developmental-timing"><i class="fa fa-check"></i><b>5.2.1</b> The <em>C. elegans</em> cuticle and its role in developmental timing</a></li>
<li class="chapter" data-level="5.2.2" data-path="discussion-2.html"><a href="discussion-2.html#identifying-genetic-factors-that-contribute-to-variation-in-growth-and-body-size"><i class="fa fa-check"></i><b>5.2.2</b> Identifying genetic factors that contribute to variation in growth and body size</a></li>
<li class="chapter" data-level="5.2.3" data-path="discussion-2.html"><a href="discussion-2.html#phenotypic-plasticity-and-genotype-by-environment-interaction"><i class="fa fa-check"></i><b>5.2.3</b> Phenotypic plasticity and genotype-by-environment interaction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
<li class="chapter" data-level="7" data-path="appendix.html"><a href="appendix.html"><i class="fa fa-check"></i><b>7</b> Appendix</a></li>
<li><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#a.-growth-dynamics-of-c.-elegans-cuticle-mutants">A. Growth dynamics of <em>C. elegans</em> cuticle mutants</a>
<ul>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#description"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#methods"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#contributions-3"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li><a href="b.-food-allocation-during-c.-elegans-growth.html#b.-food-allocation-during-c.-elegans-growth">B. Food allocation during <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#methods-1"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#contributions-4"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Mechanisms of Larval Growth Control in <em>Caenorhabditis elegans</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="introduction" class="section level1" number="1">
<h1><span class="header-section-number">1</span> Introduction</h1>
<p>A fundamental objective in biology is to understand what makes individuals, populations, and species different from one another. The observable characteristics of an organism, such as morphology, physiology, development, and behavior, make up its phenotype. Contrarily, genotype describes the inheritable genetic material that is responsible for determining a given trait <span class="citation">[<a href="references.html#ref-Johannsen1911-yr" role="doc-biblioref">1</a>]</span>. The relationship between genotype and phenotype is often complex and multifaceted. Some traits, like color blindness can be linked to single gene mutations <span class="citation">[<a href="references.html#ref-Nathans1986-mn" role="doc-biblioref">2</a>]</span>. Most traits, however, are influenced by a combination of genetic and environmental factors. Investigating the variation in complex traits, such as animal growth and body size, provides deeper insights to phenotypic diversity and its causes.</p>
<p>This introduction starts by providing an overview of phenotypic heterogeneity, focusing on the genetic variation that can influence its emergence <a href="introduction.html#Section1.1">Section 1.1</a>. I discuss how quantitative genetic mapping is used to study underlying variation in complex traits <a href="introduction.html#Section1.2">Section 1.2</a>. Next, <em>Caenorhabditis elegans</em> is presented as an appropriate model system for the study of phenotypic variation <a href="introduction.html#Section1.3">Section 1.3</a>. I discuss post-embryonic development and examine existing knowledge of how the processes of growth and body size are controlled in the nematode. Finally, I review principles of genetic mapping in <em>C. elegans</em>.</p>
<div id="Section1.1" class="section level2" number="1.1">
<h2><span class="header-section-number">1.1</span> Phenotypic variation</h2>
<p>The field of quantitative genetics classically postulates that variation among phenotypes arises from two major sources: genetics and environment <span class="citation">[<a href="references.html#ref-Lynch1998-rh" role="doc-biblioref">3</a>,<a href="references.html#ref-Falconer1960-ux" role="doc-biblioref">4</a>]</span>. Expressed as an equation, this can be written as,</p>
<p><span class="math display">\[
V_P = V_G + V_E
\]</span></p>
<p>where <span class="math inline">\(V_P\)</span> represents variance in a phenotype, <span class="math inline">\(V_{G}\)</span> is genetic variation, and <span class="math inline">\(V_{E}\)</span> is variation in the environment. The genetic component of variation arises from genetic variants within a population <span class="citation">[<a href="references.html#ref-Griffiths2000-jw" role="doc-biblioref">5</a>]</span>. Conversely, environmental variation arises when individuals of the same genotype experience changes in environmental conditions (i.e. food availability, temperature) and respond with multiple phenotypic differences <span class="citation">[<a href="references.html#ref-Davis2005-zi" role="doc-biblioref">6</a>]</span>. Notably, genetic and environmental components of phenotypic variation are rarely independent; many traits exhibit a genetic component to plasticity. In this case, the observable change in a population’s phenotype due to environmental differences is influenced by genetic variation. This form of variation indicates the presence of genotype-by-environment (GxE) interaction. Given this interaction term, total phenotypic variation is more appropriately expressed as,</p>
<p><span class="math display">\[
V_P = V_G + V_E + V_{GxE}
\]</span></p>
<p>where the contribution of genetic variance to plasticity is captured in the term, <span class="math inline">\(V_{GxE}\)</span> <span class="citation">[<a href="references.html#ref-Gutteling2007-jq" role="doc-biblioref">7</a>]</span>.</p>
<p>Another important source of phenotype variation is stochastic variation. For a given environmental condition, random noise exists in the expression of the true phenotype. This uncontrolled variation manifests as differences among individuals of the same genotype raised in a common environment <span class="citation">[<a href="references.html#ref-Felix2015-yc" role="doc-biblioref">8</a>]</span>. Accordingly, the total phenotypic variation of a population should now be represented as,</p>
<p><span class="math display">\[
V_P = V_G + V_E + V_{GxE} + V_{Noise}
\]</span></p>
<p>where <span class="math inline">\(V_{Noise}\)</span> represents intra-individual variation in response to an intrinsic factor. Often, this noise is driven by stochastic fluctuations in the developmental process and called developmental noise <span class="citation">[<a href="references.html#ref-Kiskowski2019-zd" role="doc-biblioref">9</a>]</span>. The interaction of these various sources of variation (<span class="math inline">\(V_{G}\)</span> , <span class="math inline">\(V_{E}\)</span> , <span class="math inline">\(V_{GxE}\)</span> ,<span class="math inline">\(V_{Noise}\)</span>), combined with the sheer magnitude of the number of phenotypic traits, makes the study of the phenotypic variation in a population strikingly complex.</p>
<div id="from-genotype-to-phenotype" class="section level3" number="1.1.1">
<h3><span class="header-section-number">1.1.1</span> From genotype to phenotype</h3>
<p>In 1865, an Austrian monk by the name of Gregor Mendel published work investigating the connection between genotype and phenotype. For eight years he studied the pea plants that grew in the garden of his monastery, observing patterns in their color, texture, and height from one generation to the next <span class="citation">[<a href="references.html#ref-Mendel1965-ok" role="doc-biblioref">10</a>]</span>. Mendel’s experiments laid the foundation for our understanding of genetic inheritance and the correlation between underlying genetic makeup and physically visible traits. More than 150 years later, scientists continue to study how genetic variation contributes to phenotypic differences.</p>
<p>Recent advances in sequencing technologies have accelerated the generation of vast amounts of genotype data <span class="citation">[<a href="references.html#ref-1000_Genomes_Project_Consortium2010-aw" role="doc-biblioref">11</a>–<a href="references.html#ref-Lek2016-cl" role="doc-biblioref">13</a>]</span> and have also enabled scientists to identify genetic perturbations causally linked to disease traits. However, discerning the relationship between genotype and phenotype remains a challenge. Although there is a strong correlation between the two, phenotype is not solely determined by the genotype of a few trait-associated genes but rather is the cumulative result of interactions among genetic and non-genetic factors over time <span class="citation">[<a href="references.html#ref-Cooper2013-gk" role="doc-biblioref">14</a>]</span>. This is particularly apparent in the context of disease. The clinical symptoms we observe in affected individuals are the result of interactions between their genetic background, mutations, and the environment over time. Disease phenotypes may follow Mendelian inheritance patterns and thus be predominantly driven by a single gene mutation, but oftentimes they involve multiple genes along with additional environmental factors <span class="citation">[<a href="references.html#ref-Antonarakis2006-uh" role="doc-biblioref">15</a>,<a href="references.html#ref-Barton2002-um" role="doc-biblioref">16</a>]</span>. As the complexity of a trait increases, it becomes increasingly more difficult to resolve the genotype to phenotype relationship. However, the identification of genetic variants underlying phenotypes is key to understanding how genotype influences complex traits associated with growth, health, and disease.</p>
</div>
</div>
<div id="Section1.2" class="section level2" number="1.2">
<h2><span class="header-section-number">1.2</span> Quantitative trait loci mapping</h2>
<p>Quantitative trait loci (QTL) mapping is a method for identifying the genetic basis of complex traits. With this technique, regions of the genome that contain loci that affect a measurable trait can be identified. Mapping QTL allows scientists to study the underlying genetic architecture of complex traits. Many studies have mapped QTL that affect human disease and have uncovered new loci providing insights into the biology of disease <span class="citation">[<a href="references.html#ref-Altshuler2008-qh" role="doc-biblioref">17</a>–<a href="references.html#ref-Frazer2009-qw" role="doc-biblioref">19</a>]</span>.</p>
<p>There are two main approaches to identify QTL: linkage mapping and genome-wide association (GWA) mapping <span class="citation">[<a href="references.html#ref-Mackay2009-wr" role="doc-biblioref">20</a>]</span>. Both techniques rely on statistical methods to correlate genetic variation with phenotypic variation. Simply, these methods determine whether the genotype at a given position in the genome explains phenotypic differences among individuals. Linkage mapping uses populations of recombinants generated from a cross between phenotypically and genotypically diverse strains. In contrast, GWA mapping leverages the existing natural genetic diversity among genetically distinct individuals <span class="citation">[<a href="references.html#ref-Mackay2009-wr" role="doc-biblioref">20</a>]</span>.</p>
<p>Oftentimes, QTL span large genomic regions containing many genes. To narrow the search space for potential genes contributing to phenotypic differences, the QTL region can be isolated in a clean genetic background with near-isogenic lines (NILs) <span class="citation">[<a href="references.html#ref-Doroszuk2009-mw" role="doc-biblioref">21</a>,<a href="references.html#ref-Wong2002-aa" role="doc-biblioref">22</a>]</span>. The phenotypes of these NILs can then be used to refine the QTL interval by determining whether the introgressed genomic region recapitulates the QTL effect. Ultimately, in organisms with well-annotated genomes, causality can be tested with genome editing tools <span class="citation">[<a href="references.html#ref-Barrangou2007-zp" role="doc-biblioref">23</a>–<a href="references.html#ref-Doudna2014-dd" role="doc-biblioref">25</a>]</span>.</p>
</div>
<div id="Section1.3" class="section level2" number="1.3">
<h2><span class="header-section-number">1.3</span> <em>Caenorhabditis elegans</em></h2>
<p>French biologist, Emile Maupas, first described isolating <em>Rhabditis elegans</em> (now <em>Caenorhabditis elegans</em>) near the city of Algiers (Algeria) in 1900 <span class="citation">[<a href="references.html#ref-Maupas1900-ka" role="doc-biblioref">26</a>]</span>. He conducted some of the first experiments on the nematode, elaborately detailing its anatomy and development. Several decades later, <em>C. elegans</em> was again isolated for study by Victor Nigon and Ellsworth Dougherty. These scientists worked to refine culture conditions and experimental methods for propagating animals in the laboratory <span class="citation">[<a href="references.html#ref-Nigon1943-ps" role="doc-biblioref">27</a>]</span>. However, it wasn’t until Sydney Brenner published work on the nematode in the 1970s that <em>C. elegans</em> became an established model organism <span class="citation">[<a href="references.html#ref-Corsi2015-hh" role="doc-biblioref">28</a>]</span>. Since then, research on the roundworm has expanded to explore diverse areas of modern biology and is actively studied in over a thousand laboratories worldwide <span class="citation">[<a href="references.html#ref-noauthor_2016-tp" role="doc-biblioref">29</a>]</span>.</p>
<div id="as-a-model-organism" class="section level3" number="1.3.1">
<h3><span class="header-section-number">1.3.1</span> As a model organism</h3>
<p>In 1963, well-known bacterial geneticist, Sydney Brenner, decided to turn his research efforts away from bacteria and towards an animal system. He sought an organism that could serve as a genetic model to probe how mutations disrupt eukaryotic processes. After considering a number of organisms, he ultimately settled on <em>C. elegans</em> for several reasons <span class="citation">[<a href="references.html#ref-Wood1988-zk" role="doc-biblioref">30</a>]</span>. This free-living bacterivore grows rapidly on agar plates or in liquid culture, reaching an adult length of approximately 1.5 mm in three days <span class="citation">[<a href="references.html#ref-Riddle2011-fn" role="doc-biblioref">31</a>]</span>. Its small size and transparent body make the nematode an ideal organism for microscopic observation (<strong>Figure 1-1</strong>). As hermaphrodites, <em>C. elegans</em> can produce up to 300 genetically identical progeny in a lifetime. These individuals can be maintained nearly indefinitely through cryopreservation, providing a method for long-term storage and safeguarding against the accumulation of mutations <span class="citation">[<a href="references.html#ref-Riddle2011-fn" role="doc-biblioref">31</a>,<a href="references.html#ref-Brenner1974-kr" role="doc-biblioref">32</a>]</span>.</p>
<p><img src="img/ch1/figure1-1.png" width="80%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 1-1. Photomicrographs of <em>C. elegans</em>.</strong> Major major anatomical features of the <em>C. elegans</em> adult hermaphrodite (top) and male (bottom) under bright-field illumination. Bar, 20 <span class="math inline">\(\mu m\)</span>. (From <span class="citation">[<a href="references.html#ref-Riddle2011-fn" role="doc-biblioref">31</a>]</span>).</p>
<p>Since Brenner’s first paper establishing <em>C. elegans</em> as a model organism, the animal has become a powerful system for genetic and molecular analysis. Notably, by the 1980s, scientists had documented the entire cell lineage of the nematode, from embryo to adult <span class="citation">[<a href="references.html#ref-Sulston1977-ie" role="doc-biblioref">33</a>–<a href="references.html#ref-Kimble1979-oj" role="doc-biblioref">36</a>]</span>. A decade later, <em>C. elegans</em> became the first multicellular organism to have its genome sequenced <span class="citation">[<a href="references.html#ref-C_elegans_Sequencing_Consortium1998-is" role="doc-biblioref">37</a>]</span>. This 100 Mb genome, though relatively small, comprises six chromosomes and approximately 20,000 protein-coding genes that are homologous to protein-coding genes found in other organisms, including humans <span class="citation">[<a href="references.html#ref-Kaletta2006-wg" role="doc-biblioref">38</a>,<a href="references.html#ref-Kim2018-xy" role="doc-biblioref">39</a>]</span>.</p>
</div>
<div id="post-embryonic-development" class="section level3" number="1.3.2">
<h3><span class="header-section-number">1.3.2</span> Post-embryonic development</h3>
<p><em>C. elegans</em> post-embryonic development consists of four larval stages (L1 - L4) that are punctuated by distinct developmental events called molts where animals shed an exoskeleton called the cuticle (<strong>Figure 1-2</strong>). Each larval stage begins with a period of active growth, and is followed by a period of quiescence (lethargus) where feeding and movement is temporarily halted <span class="citation">[<a href="references.html#ref-Singh1978-xi" role="doc-biblioref">40</a>]</span>. During this time, a new cuticle is synthesized under the old. A molt is completed when animals shed the old cuticle (ecdysis) and transition to the next stage <span class="citation">[<a href="references.html#ref-Monsalve2012-bl" role="doc-biblioref">41</a>]</span>.</p>
<p><img src="img/ch1/figure1-2.png" width="55%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 1-2. Illustration of the <em>C. elegans</em> life cycle.</strong> Embryos hatch to the first larval stage and proceed through four larval stages (L1 - L4) before becoming mature adults. Transitions between larval stages are marked by molt events where the animal’s outer collagen-rich cuticle is shed. An alternate developmental stage, dauer, occurs in response to unfavorable environmental conditions. Created with Biorender.com</p>
<p>The rate of larval development in <em>C. elegans</em> is strongly dependent on environmental conditions. Dietary restriction has been shown to extend developmental progression up to ten-fold <span class="citation">[<a href="references.html#ref-Uppaluri2015-wd" role="doc-biblioref">42</a>]</span>. In extreme conditions, such as absence of food or high population density, <em>C. elegans</em> are able to halt development entirely by entering dauer, a long-lived morphologically distinct stage optimized for survival <span class="citation">[<a href="references.html#ref-Cassada1975-zx" role="doc-biblioref">43</a>,<a href="references.html#ref-Hu2018-qv" role="doc-biblioref">44</a>]</span>. The decision to enter dauer is one of many checkpoints regulating developmental progression. Such checkpoints often occur early in the molting process and have been identified in every larval stage <span class="citation">[<a href="references.html#ref-Schindler2014-hh" role="doc-biblioref">45</a>,<a href="references.html#ref-Baugh2006-nv" role="doc-biblioref">46</a>]</span>. The well-coordinated timing of development is essential to the completion of stage-specific processes including cell-differentiation and tissue morphogenesis <span class="citation">[<a href="references.html#ref-Monsalve2011-vo" role="doc-biblioref">47</a>,<a href="references.html#ref-Zaidel-Bar2010-up" role="doc-biblioref">48</a>]</span>, and underscores the importance of growth regulation during <em>C. elegans</em> larval development.</p>
</div>
<div id="growth-regulation" class="section level3" number="1.3.3">
<h3><span class="header-section-number">1.3.3</span> Growth regulation</h3>
<p>Organismal growth is regulated on a genetic level, as changes in gene expression patterns and signaling dictate much of development. However, environmental conditions (<em>e.g.</em> nutrient availability and temperature) also have strong impacts on growth. Therefore, it is important to account for how the processes of growth and development are controlled genetically, and how they can be influenced environmentally.</p>
<div id="genetic-pathways-regulate-body-size" class="section level4" number="1.3.3.1">
<h4><span class="header-section-number">1.3.3.1</span> Genetic pathways regulate body size</h4>
<p>The control of <em>C. elegans</em> developmental progression has been studied extensively on the genetic level. The very first body size mutants were isolated in genetic screens by Sydney Brenner <span class="citation">[<a href="references.html#ref-Brenner1974-kr" role="doc-biblioref">32</a>]</span>. Subsequent screens for small (Sma) and long (Lon) mutants revealed a major regulator of growth and body size: DBL-1 <span class="citation">[<a href="references.html#ref-Suzuki1999-qm" role="doc-biblioref">49</a>–<a href="references.html#ref-Gumienny2018-oa" role="doc-biblioref">52</a>]</span>. DBL-1 is a ligand in the transforming growth factor-<span class="math inline">\(\beta\)</span> (TGF-<span class="math inline">\(\beta\)</span>) superfamily, and homologous to members of the mammalian bone morphogenetic protein (BMP) family <span class="citation">[<a href="references.html#ref-Wu2009-ap" role="doc-biblioref">53</a>]</span>. Secreted by neurons and body wall muscle, DBL-1 is necessary for body size regulation, and other developmental and homeostatic processes <span class="citation">[<a href="references.html#ref-Suzuki1999-qm" role="doc-biblioref">49</a>]</span>. Mutants deficient in dbl-1 have a marked reduction (40%) in both length and width compared to wild type throughout post-embryonic development <span class="citation">[<a href="references.html#ref-Suzuki1999-qm" role="doc-biblioref">49</a>,<a href="references.html#ref-Morita1999-fu" role="doc-biblioref">50</a>]</span>. In contrast, increased DBL-1 signaling causes animals to be longer (25%) than wild type <span class="citation">[<a href="references.html#ref-Suzuki1999-qm" role="doc-biblioref">49</a>,<a href="references.html#ref-Gumienny2007-qe" role="doc-biblioref">54</a>]</span>.</p>
</div>
<div id="environmental-factors-modulate-body-size" class="section level4" number="1.3.3.2">
<h4><span class="header-section-number">1.3.3.2</span> Environmental factors modulate body size</h4>
<p>As previously mentioned, nutrient restriction is known to decrease <em>C. elegans</em> growth rate or, when extreme, induce complete developmental arrest <span class="citation">[<a href="references.html#ref-Uppaluri2015-wd" role="doc-biblioref">42</a>,<a href="references.html#ref-Hu2018-qv" role="doc-biblioref">44</a>,<a href="references.html#ref-Baugh2013-at" role="doc-biblioref">55</a>]</span>. Body size is also affected by the quality of food. As bacterivores, <em>C. elegans</em> obtain their nutrients primarily from bacteria. Historically, a strain of Escherichia coli, OP50, was chosen as the standard laboratory diet <span class="citation">[<a href="references.html#ref-Brenner1974-wb" role="doc-biblioref">56</a>]</span>. While most experimental studies use OP50 as the primary food source, the number of bacterial diets used to propagate <em>C. elegans</em> has expanded in recent years. Although these diets support development, research has identified several phenotypic effects that are altered. For example, when fed <em>Comamonas</em> DA1877 or <em>E. coli</em> HB101, animals develop faster and grow larger than those fed <em>E. coli</em> OP50 <span class="citation">[<a href="references.html#ref-MacNeil2013-pk" role="doc-biblioref">57</a>–<a href="references.html#ref-Stuhr2020-vr" role="doc-biblioref">59</a>]</span>.</p>
<p>Aside from diet, other environmental stimuli can also regulate <em>C. elegans</em> body size. Standard wild-type animals used in most experiments grow larger when cultured at lower temperatures <span class="citation">[<a href="references.html#ref-Kammenga2007-cp" role="doc-biblioref">60</a>]</span>. Additionally, research has shown that the <em>C. elegans</em> nervous system plays a key role in the regulation of body size by environmental stimuli. When raised in isolation, <em>C. elegans</em> exhibit decreased growth rates and reach smaller adult sizes compared to animals raised in groups <span class="citation">[<a href="references.html#ref-Rose2005-tz" role="doc-biblioref">61</a>]</span>. This result, however, is reversed with mechanosensory stimulation during development, demonstrating a role for mechanosensory neurons in body size regulation. Similarly, a number of mutations affecting ciliated sensory neurons through which <em>C. elegans</em> perceive environmental stimuli are shown to be required for animals to grow to the proper size <span class="citation">[<a href="references.html#ref-Fujiwara1999-zp" role="doc-biblioref">62</a>,<a href="references.html#ref-Fujiwara2002-dv" role="doc-biblioref">63</a>]</span>. This suggests that perception of environmental stimuli is crucial for the regulation of body size in <em>C. elegans</em>.</p>
</div>
<div id="cuticle-structure-influences-body-size" class="section level4" number="1.3.3.3">
<h4><span class="header-section-number">1.3.3.3</span> Cuticle structure influences body size</h4>
<p>The <em>C. elegans</em> cuticle cuticle is a complex, multi-layered structure primarily composed of collagens. The major surface structures of the cuticle include the circumferential ridges (annuli) and troughs (furrows) <span class="citation">[<a href="references.html#ref-Page2007-im" role="doc-biblioref">64</a>]</span> (<strong>Figure 1-3</strong>). As animals progress through their life-cycle, the structure and thickness of the cuticle changes but its role in the maintenance of body morphology and integrity remains. To date, 21 cuticle collagen mutants have been identified that cause a range of body morphology defects <span class="citation">[<a href="references.html#ref-Page2007-im" role="doc-biblioref">64</a>]</span>. Several of these mutants (<em>dpy-2, 3, 7, 8, 10</em>) lack annuli and exhibit a disproportionate reduction in body size <span class="citation">[<a href="references.html#ref-McMahon2003-vz" role="doc-biblioref">65</a>]</span>. These dpy animals are noticeably shorter in length and wider than wild type, clearly demonstrating the importance of the physical structure of the cuticle on growth.</p>
<p><img src="img/ch1/figure1-3.png" width="65%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 1-3. Illustration of the <em>C. elegans</em> cuticle.</strong> The hypodermis (hyp) comprises hypodermal cells which secrete the cuticle (grey). Circumferential rings called annuli run the length of the cuticle. The valleys between annuli are furrows, which form immediately above circumferential bundles of actin (not shown) present in hypodermal cells during embryogenesis and molting. Seam cells (orange) that run along the left and right sides. (From <span class="citation">[<a href="references.html#ref-Lints2003-kr" role="doc-biblioref">66</a>]</span>).</p>
</div>
</div>
<div id="natural-genetic-variation" class="section level3" number="1.3.4">
<h3><span class="header-section-number">1.3.4</span> Natural genetic variation</h3>
<p><em>C. elegans</em> present a scalable and tractable animal model to connect phenotypic differences to genetic variants. <em>C. elegans</em> are easily propagated as clonal cultures, eliminating genetic variance as a contributor to phenotypic variance in experimental studies. Additionally, hundreds of wild strains have been collected worldwide, providing access to extensive genetic diversity [<span class="citation">[<a href="references.html#ref-Andersen2012-cy" role="doc-biblioref">67</a>]</span>; <span class="citation">[<a href="references.html#ref-Cook2017-zq" role="doc-biblioref">68</a>]</span>]. Two heavily studied strains are the laboratory-adapted strain, N2, and a wild isolate, CB4856. N2, derived from an isolate found in Bristol, UK in 1951, is recognized as the canonical laboratory wild-type strain with its genome serving as the reference <span class="citation">[<a href="references.html#ref-Nicholas2006-fh" role="doc-biblioref">69</a>]</span>. Across the globe on the Hawaiian island of Maui, CB4856 was isolated in 1972 <span class="citation">[<a href="references.html#ref-Hodgkin1997-ay" role="doc-biblioref">70</a>]</span>. Compared against the N2 reference, the CB4856 genome is highly divergent <span class="citation">[<a href="references.html#ref-Thompson2015-fp" role="doc-biblioref">71</a>,<a href="references.html#ref-Kim2019-ck" role="doc-biblioref">72</a>]</span>. This extensive genetic diversity likely underlies the many phenotypic differences between the two strains, including social behavior, life-history traits (body size, fecundity, lifespan), and gene expression <span class="citation">[<a href="references.html#ref-Gutteling2007-jq" role="doc-biblioref">7</a>,<a href="references.html#ref-Capra2008-io" role="doc-biblioref">73</a>–<a href="references.html#ref-Andersen2014-zv" role="doc-biblioref">77</a>]</span>.</p>
</div>
<div id="linkage-mapping" class="section level3" number="1.3.5">
<h3><span class="header-section-number">1.3.5</span> Linkage mapping</h3>
<p>The N2 and CB4856 strains can be studied to better understand how genetic variation contributes to phenotypic differences. Crossing the N2 and CB4856 strains for many generations produces individuals with unique variants derived from each parental background. Performed at a large scale, these panels of recombinant inbred lines are a powerful tool for identifying genomic regions that are correlated with phenotypic variation.</p>
<p>Linkage mapping is a popular genetic mapping approach used to identify functional variants that contribute to phenotypic diversity. In <em>C. elegans</em>, linkage mapping leverages recombinant line panels, correlating genotype and phenotype to identify QTL. The Andersen Lab has developed an R package to facilitate such analyses <span class="citation">[<a href="references.html#ref-Shimko2014-eb" role="doc-biblioref">78</a>]</span>. By investigating the natural genetic variation underlying phenotypic differences, researchers are able to uncover genetic pathways involved in biological processes. Several studies have taken this approach to study the genetic underpinnings of complex traits like body size, social behavior, and drug response (<strong>Figure 1-4</strong>) <span class="citation">[<a href="references.html#ref-Andersen2014-zv" role="doc-biblioref">77</a>,<a href="references.html#ref-Bendesky2011-qp" role="doc-biblioref">79</a>–<a href="references.html#ref-Zdraljevic2017-jd" role="doc-biblioref">84</a>]</span>.</p>
<p><img src="img/ch1/figure1-4.png" width="90%"  style="display: block; margin: auto;" /></p>
<p><strong>Figure 1-4. Overview of causal genes identified through QTL mapping approaches.</strong> The colors represent the mapping technique(s) that were used for QTL mapping: bulk-segregant analysis (BSA) (orange); linkage mapping (pink); genome-wide association (GWA) mapping (green); linkage and GWA mapping (purple). (From <span class="citation">[<a href="references.html#ref-Evans2021-zr" role="doc-biblioref">85</a>]</span>)</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="acknowledgements.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": false
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-thesis.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
