<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>6 References | Mechanisms of Larval Growth Control in Caenorhabditis elegans</title>
  <meta name="description" content="Nyaanga PhD Dissertation" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="6 References | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="/img/NU_seal.png" />
  <meta property="og:description" content="Nyaanga PhD Dissertation" />
  <meta name="github-repo" content="jnyaanga" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="6 References | Mechanisms of Larval Growth Control in Caenorhabditis elegans" />
  
  <meta name="twitter:description" content="Nyaanga PhD Dissertation" />
  <meta name="twitter:image" content="/img/NU_seal.png" />




  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="discussion-2.html"/>
<link rel="next" href="appendix.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>



<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="">Nyaanga Dissertation</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="dedication.html"><a href="dedication.html"><i class="fa fa-check"></i>Dedication</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#Section1.1"><i class="fa fa-check"></i><b>1.1</b> Phenotypic variation</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#from-genotype-to-phenotype"><i class="fa fa-check"></i><b>1.1.1</b> From genotype to phenotype</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#Section1.2"><i class="fa fa-check"></i><b>1.2</b> Quantitative trait loci mapping</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#Section1.3"><i class="fa fa-check"></i><b>1.3</b> <em>Caenorhabditis elegans</em></a>
<ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#as-a-model-organism"><i class="fa fa-check"></i><b>1.3.1</b> As a model organism</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#post-embryonic-development"><i class="fa fa-check"></i><b>1.3.2</b> Post-embryonic development</a></li>
<li class="chapter" data-level="1.3.3" data-path="introduction.html"><a href="introduction.html#growth-regulation"><i class="fa fa-check"></i><b>1.3.3</b> Growth regulation</a></li>
<li class="chapter" data-level="1.3.4" data-path="introduction.html"><a href="introduction.html#natural-genetic-variation"><i class="fa fa-check"></i><b>1.3.4</b> Natural genetic variation</a></li>
<li class="chapter" data-level="1.3.5" data-path="introduction.html"><a href="introduction.html#linkage-mapping"><i class="fa fa-check"></i><b>1.3.5</b> Linkage mapping</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><i class="fa fa-check"></i><b>2</b> A role for cuticle stretch in developmental growth control</a>
<ul>
<li class="chapter" data-level="2.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#preface"><i class="fa fa-check"></i><b>2.1</b> Preface</a></li>
<li class="chapter" data-level="2.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#abstract-1"><i class="fa fa-check"></i><b>2.2</b> Abstract</a></li>
<li class="chapter" data-level="2.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#introduction-1"><i class="fa fa-check"></i><b>2.3</b> Introduction</a></li>
<li class="chapter" data-level="2.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#materials-and-methods"><i class="fa fa-check"></i><b>2.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="2.4.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#worm-culture"><i class="fa fa-check"></i><b>2.4.1</b> Worm culture</a></li>
<li class="chapter" data-level="2.4.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#bacterial-food"><i class="fa fa-check"></i><b>2.4.2</b> Bacterial food</a></li>
<li class="chapter" data-level="2.4.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#growth-of-the-animals"><i class="fa fa-check"></i><b>2.4.3</b> Growth of the animals</a></li>
<li class="chapter" data-level="2.4.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#high-throughput-measurements-of-body-size-and-fluorescence"><i class="fa fa-check"></i><b>2.4.4</b> High-throughput measurements of body size and fluorescence</a></li>
<li class="chapter" data-level="2.4.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#image-processing"><i class="fa fa-check"></i><b>2.4.5</b> Image processing</a></li>
<li class="chapter" data-level="2.4.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#data-processing"><i class="fa fa-check"></i><b>2.4.6</b> Data processing</a></li>
<li class="chapter" data-level="2.4.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#molt-analysis"><i class="fa fa-check"></i><b>2.4.7</b> Molt analysis</a></li>
<li class="chapter" data-level="2.4.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#comparison-of-model-fits"><i class="fa fa-check"></i><b>2.4.8</b> Comparison of model fits</a></li>
<li class="chapter" data-level="2.4.9" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#stretcher-model-analysis"><i class="fa fa-check"></i><b>2.4.9</b> Stretcher model analysis</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#results"><i class="fa fa-check"></i><b>2.5</b> Results</a>
<ul>
<li class="chapter" data-level="2.5.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#quantitative-measurements-of-c.-elegans-growth"><i class="fa fa-check"></i><b>2.5.1</b> Quantitative measurements of <em>C. elegans</em> growth</a></li>
<li class="chapter" data-level="2.5.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#fluorescence-provides-a-quantitative-measurement-of-animal-feeding-behavior-and-developmental-progression"><i class="fa fa-check"></i><b>2.5.2</b> Fluorescence <strong>provides a quantitative measurement of animal feeding behavior and developmental progression</strong></a></li>
<li class="chapter" data-level="2.5.3" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#changes-in-c.-elegans-body-shape-occur-at-larval-stage-transitions"><i class="fa fa-check"></i><b>2.5.3</b> Changes in <em>C. elegans</em> body shape occur at larval-stage transitions</a></li>
<li class="chapter" data-level="2.5.4" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#modeling-c.-elegans-cuticle-stretch-dynamics"><i class="fa fa-check"></i><b>2.5.4</b> Modeling <em>C. elegans</em> cuticle stretch dynamics</a></li>
</ul></li>
<li class="chapter" data-level="2.6" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#discussion"><i class="fa fa-check"></i><b>2.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="2.6.1" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#a-potential-role-for-cuticle-stretch-in-the-timing-of-larval-stage-transitions"><i class="fa fa-check"></i><b>2.6.1</b> A potential role for cuticle stretch in the timing of larval-stage transitions</a></li>
<li class="chapter" data-level="2.6.2" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#development-comprises-complex-interactions-of-growth-regulation-across-diverse-scales"><i class="fa fa-check"></i><b>2.6.2</b> Development comprises complex interactions of growth regulation across diverse scales</a></li>
</ul></li>
<li class="chapter" data-level="2.7" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#contributions"><i class="fa fa-check"></i><b>2.7</b> Contributions</a></li>
<li class="chapter" data-level="2.8" data-path="a-role-for-cuticle-stretch-in-developmental-growth-control.html"><a href="a-role-for-cuticle-stretch-in-developmental-growth-control.html#supplement"><i class="fa fa-check"></i><b>2.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><i class="fa fa-check"></i><b>3</b> Multiple genetic loci underlie differences in <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="3.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#preface-1"><i class="fa fa-check"></i><b>3.1</b> Preface</a></li>
<li class="chapter" data-level="3.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#abstract-2"><i class="fa fa-check"></i><b>3.2</b> Abstract</a></li>
<li class="chapter" data-level="3.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#introduction-2"><i class="fa fa-check"></i><b>3.3</b> Introduction</a></li>
<li class="chapter" data-level="3.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#materials-and-methods-1"><i class="fa fa-check"></i><b>3.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#strains"><i class="fa fa-check"></i><b>3.4.1</b> Strains</a></li>
<li class="chapter" data-level="3.4.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-growth-assay"><i class="fa fa-check"></i><b>3.4.2</b> High-throughput growth assay</a></li>
<li class="chapter" data-level="3.4.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#high-throughput-fitness-assay-for-linkage-mapping"><i class="fa fa-check"></i><b>3.4.3</b> High-throughput fitness assay for linkage mapping</a></li>
<li class="chapter" data-level="3.4.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#linkage-mapping-1"><i class="fa fa-check"></i><b>3.4.4</b> Linkage mapping</a></li>
<li class="chapter" data-level="3.4.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#generation-of-chromosome-substitution-strains-csss-and-near-isogenic-lines-nils"><i class="fa fa-check"></i><b>3.4.5</b> Generation of chromosome substitution strains (CSSs) and near-isogenic lines (NILs)</a></li>
<li class="chapter" data-level="3.4.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#statistical-analysis-of-css-and-nil-results"><i class="fa fa-check"></i><b>3.4.6</b> Statistical analysis of CSS and NIL results</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#results-1"><i class="fa fa-check"></i><b>3.5</b> Results</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#growth-dynamics-of-the-n2-and-cb4856-strains"><i class="fa fa-check"></i><b>3.5.1</b> Growth dynamics of the N2 and CB4856 strains</a></li>
<li class="chapter" data-level="3.5.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-qtl-underlying-variation-in-growth"><i class="fa fa-check"></i><b>3.5.2</b> Identification of QTL underlying variation in growth</a></li>
<li class="chapter" data-level="3.5.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#validation-of-loci-associated-with-differences-in-animal-size"><i class="fa fa-check"></i><b>3.5.3</b> Validation of loci associated with differences in animal size</a></li>
<li class="chapter" data-level="3.5.4" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#identification-of-candidate-genes-in-the-chromosome-x-qtl"><i class="fa fa-check"></i><b>3.5.4</b> Identification of candidate genes in the chromosome X QTL</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#discussion-1"><i class="fa fa-check"></i><b>3.6</b> Discussion</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#a-complex-genetic-architecture-underlies-differences-in-body-size"><i class="fa fa-check"></i><b>3.6.1</b> A complex genetic architecture underlies differences in body size</a></li>
<li class="chapter" data-level="3.6.2" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#candidate-genes-for-variation-in-body-size"><i class="fa fa-check"></i><b>3.6.2</b> Candidate genes for variation in body size</a></li>
<li class="chapter" data-level="3.6.3" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#comparison-with-previous-qtl-studies-of-c.-elegans-growth"><i class="fa fa-check"></i><b>3.6.3</b> Comparison with previous QTL studies of <em>C. elegans</em> growth</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#contributions-1"><i class="fa fa-check"></i><b>3.7</b> Contributions</a></li>
<li class="chapter" data-level="3.8" data-path="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html"><a href="multiple-genetic-loci-underlie-differences-in-c.-elegans-growth.html#supplement-1"><i class="fa fa-check"></i><b>3.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><i class="fa fa-check"></i><b>4</b> easyXpress: An R package to analyze and visualize <em>C. elegans</em> microscopy data generated using CellProfiler</a>
<ul>
<li class="chapter" data-level="4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preface-2"><i class="fa fa-check"></i><b>4.1</b> Preface</a></li>
<li class="chapter" data-level="4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#abstract-3"><i class="fa fa-check"></i><b>4.2</b> Abstract</a></li>
<li class="chapter" data-level="4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#introduction-3"><i class="fa fa-check"></i><b>4.3</b> Introduction</a></li>
<li class="chapter" data-level="4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#materials-and-methods-2"><i class="fa fa-check"></i><b>4.4</b> Materials and methods</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#preparation-of-animals-for-imaging"><i class="fa fa-check"></i><b>4.4.1</b> Preparation of animals for imaging</a></li>
<li class="chapter" data-level="4.4.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#imaging"><i class="fa fa-check"></i><b>4.4.2</b> Imaging</a></li>
<li class="chapter" data-level="4.4.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#paraquat-dose-response"><i class="fa fa-check"></i><b>4.4.3</b> Paraquat dose response</a></li>
<li class="chapter" data-level="4.4.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#naming-conventions"><i class="fa fa-check"></i><b>4.4.4</b> Naming conventions</a></li>
<li class="chapter" data-level="4.4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-availability"><i class="fa fa-check"></i><b>4.4.5</b> Data availability</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#results-2"><i class="fa fa-check"></i><b>4.5</b> Results</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#design-and-implementation"><i class="fa fa-check"></i><b>4.5.1</b> Design and implementation</a></li>
<li class="chapter" data-level="4.5.2" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-import-and-model-assignment"><i class="fa fa-check"></i><b>4.5.2</b> Data import and model assignment</a></li>
<li class="chapter" data-level="4.5.3" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#data-pruning-and-summarization"><i class="fa fa-check"></i><b>4.5.3</b> Data pruning and summarization</a></li>
<li class="chapter" data-level="4.5.4" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#visualization"><i class="fa fa-check"></i><b>4.5.4</b> Visualization</a></li>
<li class="chapter" data-level="4.5.5" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#application-to-c.-elegans-growth-data"><i class="fa fa-check"></i><b>4.5.5</b> Application to <em>C. elegans</em> growth data</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#conclusions"><i class="fa fa-check"></i><b>4.6</b> Conclusions</a></li>
<li class="chapter" data-level="4.7" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#contributions-2"><i class="fa fa-check"></i><b>4.7</b> Contributions</a></li>
<li class="chapter" data-level="4.8" data-path="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html"><a href="easyxpress-an-r-package-to-analyze-and-visualize-c.-elegans-microscopy-data-generated-using-cellprofiler.html#supplement-2"><i class="fa fa-check"></i><b>4.8</b> Supplement</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="discussion-2.html"><a href="discussion-2.html"><i class="fa fa-check"></i><b>5</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.1" data-path="discussion-2.html"><a href="discussion-2.html#summary"><i class="fa fa-check"></i><b>5.1</b> Summary</a></li>
<li class="chapter" data-level="5.2" data-path="discussion-2.html"><a href="discussion-2.html#the-future-of-developmental-growth-studies"><i class="fa fa-check"></i><b>5.2</b> The future of developmental growth studies</a>
<ul>
<li class="chapter" data-level="5.2.1" data-path="discussion-2.html"><a href="discussion-2.html#the-c.-elegans-cuticle-and-its-role-in-developmental-timing"><i class="fa fa-check"></i><b>5.2.1</b> The <em>C. elegans</em> cuticle and its role in developmental timing</a></li>
<li class="chapter" data-level="5.2.2" data-path="discussion-2.html"><a href="discussion-2.html#identifying-genetic-factors-that-contribute-to-variation-in-growth-and-body-size"><i class="fa fa-check"></i><b>5.2.2</b> Identifying genetic factors that contribute to variation in growth and body size</a></li>
<li class="chapter" data-level="5.2.3" data-path="discussion-2.html"><a href="discussion-2.html#phenotypic-plasticity-and-genotype-by-environment-interaction"><i class="fa fa-check"></i><b>5.2.3</b> Phenotypic plasticity and genotype-by-environment interaction</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="6" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>6</b> References</a></li>
<li class="chapter" data-level="7" data-path="appendix.html"><a href="appendix.html"><i class="fa fa-check"></i><b>7</b> Appendix</a></li>
<li><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#a.-growth-dynamics-of-c.-elegans-cuticle-mutants">A. Growth dynamics of <em>C. elegans</em> cuticle mutants</a>
<ul>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#description"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#methods"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#contributions-3"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html"><a href="a.-growth-dynamics-of-c.-elegans-cuticle-mutants.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li><a href="b.-food-allocation-during-c.-elegans-growth.html#b.-food-allocation-during-c.-elegans-growth">B. Food allocation during <em>C. elegans</em> growth</a>
<ul>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html"><i class="fa fa-check"></i>Description</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#methods-1"><i class="fa fa-check"></i>Methods</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#contributions-4"><i class="fa fa-check"></i>Contributions</a></li>
<li class="chapter" data-level="" data-path="b.-food-allocation-during-c.-elegans-growth.html"><a href="b.-food-allocation-during-c.-elegans-growth.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Mechanisms of Larval Growth Control in <em>Caenorhabditis elegans</em></a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1" number="6">
<h1><span class="header-section-number">6</span> References</h1>
<div id="refs" class="references csl-bib-body">
<div id="ref-Johannsen1911-yr" class="csl-entry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline">Johannsen W. The genotype conception of heredity. Am Nat. 1911;45: 129–159. doi:<a href="https://doi.org/10.1086/279202">10.1086/279202</a></div>
</div>
<div id="ref-Nathans1986-mn" class="csl-entry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline">Nathans J, Piantanida TP, Eddy RL, Shows TB, Hogness DS. Molecular genetics of inherited variation in human color vision. Science. 1986;232: 203–210. doi:<a href="https://doi.org/10.1126/science.3485310">10.1126/science.3485310</a></div>
</div>
<div id="ref-Lynch1998-rh" class="csl-entry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline">Lynch M, Walsh B, Others. Genetics and analysis of quantitative traits. Sinauer Sunderland, MA; 1998. </div>
</div>
<div id="ref-Falconer1960-ux" class="csl-entry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline">Falconer DS, Others. Introduction to quantitative genetics. Introduction to quantitative genetics. 1960. </div>
</div>
<div id="ref-Griffiths2000-jw" class="csl-entry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline">Griffiths AJF, Miller JH, Suzuki DT, Lewontin RC, Gelbart WM. Genetic variation. W. H. Freeman; 2000. </div>
</div>
<div id="ref-Davis2005-zi" class="csl-entry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline">Davis MB, Shaw RG, Etterson JR. Evolutionary responses to changing climate. Ecology. 2005;86: 1704–1714. doi:<a href="https://doi.org/10.1890/03-0788">10.1890/03-0788</a></div>
</div>
<div id="ref-Gutteling2007-jq" class="csl-entry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline">Gutteling EW, Riksen JAG, Bakker J, Kammenga JE. Mapping phenotypic plasticity and genotype-environment interactions affecting life-history traits in caenorhabditis elegans. Heredity. 2007;98: 28–37. doi:<a href="https://doi.org/10.1038/sj.hdy.6800894">10.1038/sj.hdy.6800894</a></div>
</div>
<div id="ref-Felix2015-yc" class="csl-entry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline">Félix M-A, Barkoulas M. Pervasive robustness in biological systems. Nat Rev Genet. 2015;16: 483–496. doi:<a href="https://doi.org/10.1038/nrg3949">10.1038/nrg3949</a></div>
</div>
<div id="ref-Kiskowski2019-zd" class="csl-entry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline">Kiskowski M, Glimm T, Moreno N, Gamble T, Chiari Y. Isolating and quantifying the role of developmental noise in generating phenotypic variation. PLoS Comput Biol. 2019;15: e1006943. doi:<a href="https://doi.org/10.1371/journal.pcbi.1006943">10.1371/journal.pcbi.1006943</a></div>
</div>
<div id="ref-Mendel1965-ok" class="csl-entry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline">Mendel G. Experiments in plant hybridization. Oliver &amp; Boyd; 1965. </div>
</div>
<div id="ref-1000_Genomes_Project_Consortium2010-aw" class="csl-entry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline">1000 Genomes Project Consortium, Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, et al. A map of human genome variation from population-scale sequencing. Nature. 2010;467: 1061–1073. doi:<a href="https://doi.org/10.1038/nature09534">10.1038/nature09534</a></div>
</div>
<div id="ref-Telenti2016-ov" class="csl-entry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline">Telenti A, Pierce LCT, Biggs WH, Iulio J di, Wong EHM, Fabani MM, et al. Deep sequencing of 10,000 human genomes. Proc Natl Acad Sci U S A. 2016;113: 11901–11906. doi:<a href="https://doi.org/10.1073/pnas.1613365113">10.1073/pnas.1613365113</a></div>
</div>
<div id="ref-Lek2016-cl" class="csl-entry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline">Lek M, Karczewski KJ, Minikel EV, Samocha KE, Banks E, Fennell T, et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature. 2016;536: 285–291. doi:<a href="https://doi.org/10.1038/nature19057">10.1038/nature19057</a></div>
</div>
<div id="ref-Cooper2013-gk" class="csl-entry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline">Cooper DN, Krawczak M, Polychronakos C, Tyler-Smith C, Kehrer-Sawatzki H. Where genotype is not predictive of phenotype: Towards an understanding of the molecular basis of reduced penetrance in human inherited disease. Hum Genet. 2013;132: 1077–1130. doi:<a href="https://doi.org/10.1007/s00439-013-1331-2">10.1007/s00439-013-1331-2</a></div>
</div>
<div id="ref-Antonarakis2006-uh" class="csl-entry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline">Antonarakis SE, Beckmann JS. Mendelian disorders deserve more attention. Nat Rev Genet. 2006;7: 277–282. doi:<a href="https://doi.org/10.1038/nrg1826">10.1038/nrg1826</a></div>
</div>
<div id="ref-Barton2002-um" class="csl-entry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline">Barton NH, Keightley PD. Understanding quantitative genetic variation. Nat Rev Genet. 2002;3: 11–21. doi:<a href="https://doi.org/10.1038/nrg700">10.1038/nrg700</a></div>
</div>
<div id="ref-Altshuler2008-qh" class="csl-entry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline">Altshuler D, Daly MJ, Lander ES. Genetic mapping in human disease. Science. 2008;322: 881–888. doi:<a href="https://doi.org/10.1126/science.1156409">10.1126/science.1156409</a></div>
</div>
<div id="ref-Donnelly2008-ma" class="csl-entry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline">Donnelly P. Progress and challenges in genome-wide association studies in humans. Nature. 2008;456: 728–731. doi:<a href="https://doi.org/10.1038/nature07631">10.1038/nature07631</a></div>
</div>
<div id="ref-Frazer2009-qw" class="csl-entry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline">Frazer KA, Murray SS, Schork NJ, Topol EJ. Human genetic variation and its contribution to complex traits. Nat Rev Genet. 2009;10: 241–251. doi:<a href="https://doi.org/10.1038/nrg2554">10.1038/nrg2554</a></div>
</div>
<div id="ref-Mackay2009-wr" class="csl-entry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline">Mackay TFC, Stone EA, Ayroles JF. The genetics of quantitative traits: Challenges and prospects. Nat Rev Genet. 2009;10: 565–577. doi:<a href="https://doi.org/10.1038/nrg2612">10.1038/nrg2612</a></div>
</div>
<div id="ref-Evans2021-zr" class="csl-entry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline">Evans KS, Wijk MH van, McGrath PT, Andersen EC, Sterken MG. From <span>QTL</span> to gene: C. Elegans facilitates discoveries of the genetic mechanisms underlying natural variation. Trends Genet. 2021;37: 933–947. doi:<a href="https://doi.org/10.1016/j.tig.2021.06.005">10.1016/j.tig.2021.06.005</a></div>
</div>
<div id="ref-Doroszuk2009-mw" class="csl-entry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline">Doroszuk A, Snoek LB, Fradin E, Riksen J, Kammenga J. A genome-wide library of <span>CB4856/N2</span> introgression lines of caenorhabditis elegans. Nucleic Acids Res. 2009;37: e110. doi:<a href="https://doi.org/10.1093/nar/gkp528">10.1093/nar/gkp528</a></div>
</div>
<div id="ref-Wong2002-aa" class="csl-entry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline">Wong GT. Speed congenics: Applications for transgenic and knock-out mouse strains. Neuropeptides. 2002;36: 230–236. doi:<a href="https://doi.org/10.1054/npep.2002.0905">10.1054/npep.2002.0905</a></div>
</div>
<div id="ref-Barrangou2007-zp" class="csl-entry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline">Barrangou R, Fremaux C, Deveau H, Richards M, Boyaval P, Moineau S, et al. <span>CRISPR</span> provides acquired resistance against viruses in prokaryotes. Science. 2007;315: 1709–1712. doi:<a href="https://doi.org/10.1126/science.1138140">10.1126/science.1138140</a></div>
</div>
<div id="ref-Paix2014-yp" class="csl-entry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline">Paix A, Wang Y, Smith HE, Lee C-YS, Calidas D, Lu T, et al. Scalable and versatile genome editing using linear <span>DNAs</span> with microhomology to Cas9 sites in caenorhabditis elegans. Genetics. 2014;198: 1347–1356. doi:<a href="https://doi.org/10.1534/genetics.114.170423">10.1534/genetics.114.170423</a></div>
</div>
<div id="ref-Doudna2014-dd" class="csl-entry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline">Doudna JA, Charpentier E. The new frontier of genome engineering with <span>CRISPR-Cas9</span>. Science. 2014;346: 1258096. doi:<a href="https://doi.org/10.1126/science.1258096">10.1126/science.1258096</a></div>
</div>
<div id="ref-Maupas1900-ka" class="csl-entry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline">Maupas E. Modes et formes de reproduction des nematodes. 1900. </div>
</div>
<div id="ref-Nigon1943-ps" class="csl-entry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline">Nigon V. Le d<span>é</span>terminisme du sexe chez un n<span>é</span>matode libre hermaphrodite, rhabditis elegans maupas. C R Seances Soc Biol Fil. 1943;137: 40–41. </div>
</div>
<div id="ref-Corsi2015-hh" class="csl-entry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline">Corsi AK, Wightman B, Chalfie M. A transparent window into biology: A primer on caenorhabditis elegans. Genetics. 2015;200: 387–407. doi:<a href="https://doi.org/10.1534/genetics.115.176099">10.1534/genetics.115.176099</a></div>
</div>
<div id="ref-noauthor_2016-tp" class="csl-entry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline">All registered worm labs. <a href="http://wbg.wormbook.org/the-worm-lab-project/all-labs/">http://wbg.wormbook.org/the-worm-lab-project/all-labs/</a>; 2016. </div>
</div>
<div id="ref-Wood1988-zk" class="csl-entry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline">Wood WB. The nematode caenorhabditis elegans. Cold Spring Harbor Laboratory; 1988. </div>
</div>
<div id="ref-Riddle2011-fn" class="csl-entry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline">Riddle DL, Blumenthal T, Meyer BJ, Priess JR, editors. C. Elegans <span>II</span>. Cold Spring Harbor (NY): Cold Spring Harbor Laboratory Press; 2011. </div>
</div>
<div id="ref-Brenner1974-kr" class="csl-entry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline">Brenner S. The genetics of caenorhabditis elegans. Genetics. 1974;77: 71–94. </div>
</div>
<div id="ref-Sulston1977-ie" class="csl-entry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline">Sulston JE, Horvitz HR. Post-embryonic cell lineages of the nematode, caenorhabditis elegans. Dev Biol. 1977;56: 110–156. doi:<a href="https://doi.org/10.1016/0012-1606(77)90158-0">10.1016/0012-1606(77)90158-0</a></div>
</div>
<div id="ref-Sulston1983-av" class="csl-entry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline">Sulston JE, Schierenberg E, White JG, Thomson JN. The embryonic cell lineage of the nematode caenorhabditis elegans. Dev Biol. 1983;100: 64–119. doi:<a href="https://doi.org/10.1016/0012-1606(83)90201-4">10.1016/0012-1606(83)90201-4</a></div>
</div>
<div id="ref-Deppe1978-gy" class="csl-entry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline">Deppe U, Schierenberg E, Cole T, Krieg C, Schmitt D, Yoder B, et al. Cell lineages of the embryo of the nematode caenorhabditis elegans. Proceedings of the National Academy of Sciences. 1978;75: 376–380. </div>
</div>
<div id="ref-Kimble1979-oj" class="csl-entry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline">Kimble J, Hirsh D. The postembryonic cell lineages of the hermaphrodite and male gonads in caenorhabditis elegans. Dev Biol. 1979;70: 396–417. doi:<a href="https://doi.org/10.1016/0012-1606(79)90035-6">10.1016/0012-1606(79)90035-6</a></div>
</div>
<div id="ref-C_elegans_Sequencing_Consortium1998-is" class="csl-entry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline">C. elegans Sequencing Consortium. Genome sequence of the nematode c. Elegans: A platform for investigating biology. Science. 1998;282: 2012–2018. doi:<a href="https://doi.org/10.1126/science.282.5396.2012">10.1126/science.282.5396.2012</a></div>
</div>
<div id="ref-Kaletta2006-wg" class="csl-entry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline">Kaletta T, Hengartner MO. Finding function in novel targets: C. Elegans as a model organism. Nat Rev Drug Discov. 2006;5: 387–398. doi:<a href="https://doi.org/10.1038/nrd2031">10.1038/nrd2031</a></div>
</div>
<div id="ref-Kim2018-xy" class="csl-entry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline">Kim W, Underwood RS, Greenwald I, Shaye DD. <span>OrthoList</span> 2: A new comparative genomic analysis of human and caenorhabditis elegans genes. Genetics. 2018;210: 445–461. doi:<a href="https://doi.org/10.1534/genetics.118.301307">10.1534/genetics.118.301307</a></div>
</div>
<div id="ref-Singh1978-xi" class="csl-entry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline">Singh RN, Sulston JE. Some observations on moulting in caenorhabditis elegans. Nematologica. 1978;24: 63–71. doi:<a href="https://doi.org/10.1163/187529278X00074">10.1163/187529278X00074</a></div>
</div>
<div id="ref-Monsalve2012-bl" class="csl-entry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline">Monsalve GC, Frand AR. Toward a unified model of developmental timing: A <span>“molting”</span> approach. Worm. 2012. </div>
</div>
<div id="ref-Uppaluri2015-wd" class="csl-entry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline">Uppaluri S, Brangwynne CP. A size threshold governs caenorhabditis elegans developmental progression. Proc Biol Sci. 2015;282: 20151283. doi:<a href="https://doi.org/10.1098/rspb.2015.1283">10.1098/rspb.2015.1283</a></div>
</div>
<div id="ref-Cassada1975-zx" class="csl-entry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline">Cassada RC, Russell RL. The dauerlarva, a post-embryonic developmental variant of the nematode caenorhabditis elegans. Dev Biol. 1975;46: 326–342. doi:<a href="https://doi.org/10.1016/0012-1606(75)90109-8">10.1016/0012-1606(75)90109-8</a></div>
</div>
<div id="ref-Hu2018-qv" class="csl-entry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline">Hu PJ. Dauer. WormBook; 2018. </div>
</div>
<div id="ref-Schindler2014-hh" class="csl-entry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline">Schindler AJ, Baugh LR, Sherwood DR. Identification of late larval stage developmental checkpoints in caenorhabditis elegans regulated by <span class="nocase">insulin/IGF</span> and steroid hormone signaling pathways. PLoS Genet. 2014;10: e1004426. doi:<a href="https://doi.org/10.1371/journal.pgen.1004426">10.1371/journal.pgen.1004426</a></div>
</div>
<div id="ref-Baugh2006-nv" class="csl-entry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline">Baugh LR, Sternberg PW. <span>DAF-16/FOXO</span> regulates transcription of <span class="nocase">cki-1/Cip/Kip</span> and repression of lin-4 during c. Elegans <span>L1</span> arrest. Curr Biol. 2006;16: 780–785. doi:<a href="https://doi.org/10.1016/j.cub.2006.03.021">10.1016/j.cub.2006.03.021</a></div>
</div>
<div id="ref-Monsalve2011-vo" class="csl-entry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline">Monsalve GC, Van Buskirk C, Frand AR. <span>LIN-42/PERIOD</span> controls cyclical and developmental progression of c. Elegans molts. Curr Biol. 2011;21: 2033–2045. doi:<a href="https://doi.org/10.1016/j.cub.2011.10.054">10.1016/j.cub.2011.10.054</a></div>
</div>
<div id="ref-Zaidel-Bar2010-up" class="csl-entry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline">Zaidel-Bar R, Miller S, Kaminsky R, Broday L. Molting-specific downregulation of c. Elegans body-wall muscle attachment sites: The role of <span>RNF-5</span> <span>E3</span> ligase. Biochem Biophys Res Commun. 2010;395: 509–514. doi:<a href="https://doi.org/10.1016/j.bbrc.2010.04.049">10.1016/j.bbrc.2010.04.049</a></div>
</div>
<div id="ref-Suzuki1999-qm" class="csl-entry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline">Suzuki Y, Yandell MD, Roy PJ, Krishna S, Savage-Dunn C, Ross RM, et al. A <span>BMP</span> homolog acts as a dose-dependent regulator of body size and male tail patterning in caenorhabditis elegans. Development. 1999;126: 241–250. </div>
</div>
<div id="ref-Morita1999-fu" class="csl-entry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline">Morita K, Chow KL, Ueno N. Regulation of body length and male tail ray pattern formation of caenorhabditis elegans by a member of <span class="nocase">TGF-beta</span> family. Development. 1999;126: 1337–1347. </div>
</div>
<div id="ref-Savage-Dunn2003-sj" class="csl-entry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline">Savage-Dunn C, Maduzia LL, Zimmerman CM, Roberts AF, Cohen S, Tokarz R, et al. Genetic screen for small body size mutants in c. Elegans reveals many <span>TGFbeta</span> pathway components. Genesis. 2003;35: 239–247. doi:<a href="https://doi.org/10.1002/gene.10184">10.1002/gene.10184</a></div>
</div>
<div id="ref-Gumienny2018-oa" class="csl-entry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline">Gumienny TL, Savage-Dunn C. <span>TGF-<span class="math inline">\(\beta\)</span></span> signaling in c. elegans. WormBook; 2018. </div>
</div>
<div id="ref-Wu2009-ap" class="csl-entry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline">Wu MY, Hill CS. Tgf-beta superfamily signaling in embryonic development and homeostasis. Dev Cell. 2009;16: 329–343. doi:<a href="https://doi.org/10.1016/j.devcel.2009.02.012">10.1016/j.devcel.2009.02.012</a></div>
</div>
<div id="ref-Gumienny2007-qe" class="csl-entry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline">Gumienny TL, MacNeil LT, Wang H, Bono M de, Wrana JL, Padgett RW. Glypican <span>LON-2</span> is a conserved negative regulator of <span class="nocase">BMP-like</span> signaling in caenorhabditis elegans. Curr Biol. 2007;17: 159–164. doi:<a href="https://doi.org/10.1016/j.cub.2006.11.065">10.1016/j.cub.2006.11.065</a></div>
</div>
<div id="ref-Baugh2013-at" class="csl-entry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline">Baugh LR. To grow or not to grow: Nutritional control of development during caenorhabditis elegans <span>L1</span> arrest. Genetics. 2013;194: 539–555. doi:<a href="https://doi.org/10.1534/genetics.113.150847">10.1534/genetics.113.150847</a></div>
</div>
<div id="ref-Brenner1974-wb" class="csl-entry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline">Brenner S. The genetics of caenorhabditis elegans. Genetics. 1974;77: 71–94. </div>
</div>
<div id="ref-MacNeil2013-pk" class="csl-entry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline">MacNeil LT, Watson E, Arda HE, Zhu LJ, Walhout AJM. Diet-induced developmental acceleration independent of <span>TOR</span> and insulin in c. elegans. Cell. 2013;153: 240–252. doi:<a href="https://doi.org/10.1016/j.cell.2013.02.049">10.1016/j.cell.2013.02.049</a></div>
</div>
<div id="ref-So2011-ls" class="csl-entry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline">So S, Miyahara K, Ohshima Y. Control of body size in c. Elegans dependent on food and <span class="nocase">insulin/IGF-1</span> signal. Genes Cells. 2011;16: 639–651. doi:<a href="https://doi.org/10.1111/j.1365-2443.2011.01514.x">10.1111/j.1365-2443.2011.01514.x</a></div>
</div>
<div id="ref-Stuhr2020-vr" class="csl-entry">
<div class="csl-left-margin">60. </div><div class="csl-right-inline">Stuhr NL, Curran SP. Bacterial diets differentially alter lifespan and healthspan trajectories in c. elegans. Commun Biol. 2020;3: 653. doi:<a href="https://doi.org/10.1038/s42003-020-01379-1">10.1038/s42003-020-01379-1</a></div>
</div>
<div id="ref-Kammenga2007-cp" class="csl-entry">
<div class="csl-left-margin">61. </div><div class="csl-right-inline">Kammenga JE, Doroszuk A, Riksen JAG, Hazendonk E, Spiridon L, Petrescu A-J, et al. A caenorhabditis elegans wild type defies the <span>Temperature–Size</span> rule owing to a single nucleotide polymorphism in tra-3. PLoS Genet. 2007;3: e34. doi:<a href="https://doi.org/10.1371/journal.pgen.0030034">10.1371/journal.pgen.0030034</a></div>
</div>
<div id="ref-Rose2005-tz" class="csl-entry">
<div class="csl-left-margin">62. </div><div class="csl-right-inline">Rose JK, Sangha S, Rai S, Norman KR, Rankin CH. Decreased sensory stimulation reduces behavioral responding, retards development, and alters neuronal connectivity in caenorhabditis elegans. J Neurosci. 2005;25: 7159–7168. doi:<a href="https://doi.org/10.1523/JNEUROSCI.1833-05.2005">10.1523/JNEUROSCI.1833-05.2005</a></div>
</div>
<div id="ref-Fujiwara1999-zp" class="csl-entry">
<div class="csl-left-margin">63. </div><div class="csl-right-inline">Fujiwara M, Ishihara T, Katsura I. A novel <span>WD40</span> protein, <span>CHE-2</span>, acts cell-autonomously in the formation of c. Elegans sensory cilia. Development. 1999;126: 4839–4848. </div>
</div>
<div id="ref-Fujiwara2002-dv" class="csl-entry">
<div class="csl-left-margin">64. </div><div class="csl-right-inline">Fujiwara M, Sengupta P, McIntire SL. Regulation of body size and behavioral state of c. Elegans by sensory perception and the <span>EGL-4</span> <span class="nocase">cGMP-dependent</span> protein kinase. Neuron. 2002;36: 1091–1102. doi:<a href="https://doi.org/10.1016/s0896-6273(02)01093-0">10.1016/s0896-6273(02)01093-0</a></div>
</div>
<div id="ref-Page2007-im" class="csl-entry">
<div class="csl-left-margin">65. </div><div class="csl-right-inline">Page AP, Johnstone IL. The cuticle. WormBook; 2007. </div>
</div>
<div id="ref-McMahon2003-vz" class="csl-entry">
<div class="csl-left-margin">66. </div><div class="csl-right-inline">McMahon L, Muriel JM, Roberts B, Quinn M, Johnstone IL. Two sets of interacting collagens form functionally distinct substructures within a caenorhabditis elegansExtracellular matrix. MBoC. 2003;14: 1366–1378. doi:<a href="https://doi.org/10.1091/mbc.e02-08-0479">10.1091/mbc.e02-08-0479</a></div>
</div>
<div id="ref-Lints2003-kr" class="csl-entry">
<div class="csl-left-margin">67. </div><div class="csl-right-inline">Lints R, Hall DH. <span>WormAtlas</span> hermaphrodite handbook - the cuticle. WormAtlas. 2003. doi:<a href="https://doi.org/10.3908/wormatlas.1.12">10.3908/wormatlas.1.12</a></div>
</div>
<div id="ref-Andersen2012-cy" class="csl-entry">
<div class="csl-left-margin">68. </div><div class="csl-right-inline">Andersen EC, Gerke JP, Shapiro JA, Crissman JR, Ghosh R, Bloom JS, et al. Chromosome-scale selective sweeps shape caenorhabditis elegans genomic diversity. Nat Genet. 2012;44: 285–290. doi:<a href="https://doi.org/10.1038/ng.1050">10.1038/ng.1050</a></div>
</div>
<div id="ref-Cook2017-zq" class="csl-entry">
<div class="csl-left-margin">69. </div><div class="csl-right-inline">Cook DE, Zdraljevic S, Roberts JP, Andersen EC. <span>CeNDR</span>, the caenorhabditis elegans natural diversity resource. Nucleic Acids Res. 2017;45: D650–D657. doi:<a href="https://doi.org/10.1093/nar/gkw893">10.1093/nar/gkw893</a></div>
</div>
<div id="ref-Nicholas2006-fh" class="csl-entry">
<div class="csl-left-margin">70. </div><div class="csl-right-inline">Nicholas WL, Dougherty EC, Hansen EL. <span>AXENIC</span> <span>CULTIVATION</span> <span>OF</span> <span>CAENORHARDITIS</span> <span>BRIGGSAE</span> (<span>NEMATODA</span>: <span>RHABDITIDAE</span>) <span>WITH</span> <span>CHEMICALLY</span> <span>UNDEFINED</span> <span>SUPPLEMENTS</span>; <span>COMPARATIVE</span> <span>STUDIES</span> <span>WITH</span> <span>RELATED</span> <span>NEMATODES*</span>. Ann N Y Acad Sci. 2006;77: 218–236. doi:<a href="https://doi.org/10.1111/j.1749-6632.1959.tb36902.x">10.1111/j.1749-6632.1959.tb36902.x</a></div>
</div>
<div id="ref-Hodgkin1997-ay" class="csl-entry">
<div class="csl-left-margin">71. </div><div class="csl-right-inline">Hodgkin J, Doniach T. Natural variation and copulatory plug formation in caenorhabditis elegans. Genetics. 1997;146: 149–164. </div>
</div>
<div id="ref-Thompson2015-fp" class="csl-entry">
<div class="csl-left-margin">72. </div><div class="csl-right-inline">Thompson OA, Snoek LB, Nijveen H, Sterken MG, Volkers RJM, Brenchley R, et al. Remarkably divergent regions punctuate the genome assembly of the caenorhabditis elegans hawaiian strain <span>CB4856</span>. Genetics. 2015;200: 975–989. doi:<a href="https://doi.org/10.1534/genetics.115.175950">10.1534/genetics.115.175950</a></div>
</div>
<div id="ref-Kim2019-ck" class="csl-entry">
<div class="csl-left-margin">73. </div><div class="csl-right-inline">Kim C, Kim J, Kim S, Cook DE, Evans KS, Andersen EC, et al. Long-read sequencing reveals intra-species tolerance of substantial structural variations and new subtelomere formation in c. elegans. Genome Res. 2019;29: 1023–1035. doi:<a href="https://doi.org/10.1101/gr.246082.118">10.1101/gr.246082.118</a></div>
</div>
<div id="ref-Capra2008-io" class="csl-entry">
<div class="csl-left-margin">74. </div><div class="csl-right-inline">Capra EJ, Skrovanek SM, Kruglyak L. Comparative developmental expression profiling of two c. Elegans isolates. PLoS One. 2008;3: e4055. doi:<a href="https://doi.org/10.1371/journal.pone.0004055">10.1371/journal.pone.0004055</a></div>
</div>
<div id="ref-Rockman2010-ne" class="csl-entry">
<div class="csl-left-margin">75. </div><div class="csl-right-inline">Rockman MV, Skrovanek SS, Kruglyak L. Selection at linked sites shapes heritable phenotypic variation in c. elegans. Science. 2010;330: 372–376. doi:<a href="https://doi.org/10.1126/science.1194208">10.1126/science.1194208</a></div>
</div>
<div id="ref-Vinuela2012-lr" class="csl-entry">
<div class="csl-left-margin">76. </div><div class="csl-right-inline">Viñuela A, Snoek LB, Riksen JAG, Kammenga JE. Aging uncouples heritability and <span>Expression-QTL</span> in caenorhabditis elegans. G3. 2012;2: 597–605. doi:<a href="https://doi.org/10.1534/g3.112.002212">10.1534/g3.112.002212</a></div>
</div>
<div id="ref-De_Bono1998-ea" class="csl-entry">
<div class="csl-left-margin">77. </div><div class="csl-right-inline">Bono M de, Bargmann CI. Natural variation in a neuropeptide <span>Y</span> receptor homolog modifies social behavior and food response in c. elegans. Cell. 1998;94: 679–689. doi:<a href="https://doi.org/10.1016/s0092-8674(00)81609-8">10.1016/s0092-8674(00)81609-8</a></div>
</div>
<div id="ref-Andersen2014-zv" class="csl-entry">
<div class="csl-left-margin">78. </div><div class="csl-right-inline">Andersen EC, Bloom JS, Gerke JP, Kruglyak L. A variant in the neuropeptide receptor npr-1 is a major determinant of caenorhabditis elegans growth and physiology. PLoS Genet. 2014;10: e1004156. doi:<a href="https://doi.org/10.1371/journal.pgen.1004156">10.1371/journal.pgen.1004156</a></div>
</div>
<div id="ref-Shimko2014-eb" class="csl-entry">
<div class="csl-left-margin">79. </div><div class="csl-right-inline">Shimko TC, Andersen EC. <span>COPASutils</span>: An <span>R</span> package for reading, processing, and visualizing data from <span>COPAS</span> large-particle flow cytometers. PLoS One. 2014;9: e111090. doi:<a href="https://doi.org/10.1371/journal.pone.0111090">10.1371/journal.pone.0111090</a></div>
</div>
<div id="ref-Bendesky2011-qp" class="csl-entry">
<div class="csl-left-margin">80. </div><div class="csl-right-inline">Bendesky A, Bargmann CI. Genetic contributions to behavioural diversity at the gene–environment interface. Nat Rev Genet. 2011;12: 809–820. doi:<a href="https://doi.org/10.1038/nrg3065">10.1038/nrg3065</a></div>
</div>
<div id="ref-Brady2019-qg" class="csl-entry">
<div class="csl-left-margin">81. </div><div class="csl-right-inline">Brady SC, Zdraljevic S, Bisaga KW, Tanny RE, Cook DE, Lee D, et al. A novel gene underlies <span>Bleomycin-Response</span> variation in caenorhabditis elegans. Genetics. 2019;212: 1453–1468. doi:<a href="https://doi.org/10.1534/genetics.119.302286">10.1534/genetics.119.302286</a></div>
</div>
<div id="ref-Evans2020-db" class="csl-entry">
<div class="csl-left-margin">82. </div><div class="csl-right-inline">Evans KS, Andersen EC. The gene scb-1 underlies variation in caenorhabditis elegans chemotherapeutic responses. G3 Genes|Genomes|Genetics. 2020;10: 2353–2364. doi:<a href="https://doi.org/10.1534/g3.120.401310">10.1534/g3.120.401310</a></div>
</div>
<div id="ref-Hahnel2018-da" class="csl-entry">
<div class="csl-left-margin">83. </div><div class="csl-right-inline">Hahnel SR, Zdraljevic S, Rodriguez BC, Zhao Y, McGrath PT, Andersen EC. Extreme allelic heterogeneity at a caenorhabditis elegans beta-tubulin locus explains natural resistance to benzimidazoles. PLoS Pathog. 2018;14: e1007226. doi:<a href="https://doi.org/10.1371/journal.ppat.1007226">10.1371/journal.ppat.1007226</a></div>
</div>
<div id="ref-Zdraljevic2019-fr" class="csl-entry">
<div class="csl-left-margin">84. </div><div class="csl-right-inline">Zdraljevic S, Fox BW, Strand C, Panda O, Tenjo FJ, Brady SC, et al. Natural variation in c. Elegans arsenic toxicity is explained by differences in branched chain amino acid metabolism. 2019. doi:<a href="https://doi.org/10.7554/eLife.40260">10.7554/eLife.40260</a></div>
</div>
<div id="ref-Zdraljevic2017-jd" class="csl-entry">
<div class="csl-left-margin">85. </div><div class="csl-right-inline">Zdraljevic S, Strand C, Seidel HS, Cook DE, Doench JG, Andersen EC. Natural variation in a single amino acid substitution underlies physiological responses to topoisomerase <span>II</span> poisons. PLoS Genet. 2017;13: e1006891. doi:<a href="https://doi.org/10.1371/journal.pgen.1006891">10.1371/journal.pgen.1006891</a></div>
</div>
<div id="ref-Nyaanga2022-zd" class="csl-entry">
<div class="csl-left-margin">86. </div><div class="csl-right-inline">Nyaanga J, Goss C, Zhang G, Ahmed HN, Andersen EJ, Miller IR, et al. Changes in body shape implicate cuticle stretch in c. Elegans growth control. bioRxiv. 2022. p. 2021.04.01.438121. doi:<a href="https://doi.org/10.1101/2021.04.01.438121">10.1101/2021.04.01.438121</a></div>
</div>
<div id="ref-Hone2005-ex" class="csl-entry">
<div class="csl-left-margin">87. </div><div class="csl-right-inline">Hone DWE, Benton MJ. The evolution of large size: How does cope’s rule work? Trends Ecol Evol. 2005;20: 4–6. doi:<a href="https://doi.org/10.1016/j.tree.2004.10.012">10.1016/j.tree.2004.10.012</a></div>
</div>
<div id="ref-Bjorklund2019-ox" class="csl-entry">
<div class="csl-left-margin">88. </div><div class="csl-right-inline">Björklund M. Cell size homeostasis: Metabolic control of growth and cell division. Biochim Biophys Acta Mol Cell Res. 2019;1866: 409–417. doi:<a href="https://doi.org/10.1016/j.bbamcr.2018.10.002">10.1016/j.bbamcr.2018.10.002</a></div>
</div>
<div id="ref-Willis2017-uh" class="csl-entry">
<div class="csl-left-margin">89. </div><div class="csl-right-inline">Willis L, Huang KC. Sizing up the bacterial cell cycle. Nat Rev Microbiol. 2017;15: 606–620. doi:<a href="https://doi.org/10.1038/nrmicro.2017.79">10.1038/nrmicro.2017.79</a></div>
</div>
<div id="ref-Turner2012-ky" class="csl-entry">
<div class="csl-left-margin">90. </div><div class="csl-right-inline">Turner JJ, Ewald JC, Skotheim JM. Cell size control in yeast. Curr Biol. 2012;22: R350–9. doi:<a href="https://doi.org/10.1016/j.cub.2012.02.041">10.1016/j.cub.2012.02.041</a></div>
</div>
<div id="ref-Donnan1983-uq" class="csl-entry">
<div class="csl-left-margin">91. </div><div class="csl-right-inline">Donnan L, John PC. Cell cycle control by timer and sizer in chlamydomonas. Nature. 1983;304: 630–633. doi:<a href="https://doi.org/10.1038/304630a0">10.1038/304630a0</a></div>
</div>
<div id="ref-Wang2000-zi" class="csl-entry">
<div class="csl-left-margin">92. </div><div class="csl-right-inline">Wang P, Hayden S, Masui Y. Transition of the blastomere cell cycle from cell size-independent to size-dependent control at the midblastula stage in xenopus laevis. J Exp Zool. 2000;287: 128–144. doi:<a href="https://doi.org/10.1002/1097-010x(20000701)287:2&lt;128::aid-jez3&gt;3.0.co;2-g">10.1002/1097-010x(20000701)287:2&lt;128::aid-jez3&gt;3.0.co;2-g</a></div>
</div>
<div id="ref-Iyer-Biswas2014-ik" class="csl-entry">
<div class="csl-left-margin">93. </div><div class="csl-right-inline">Iyer-Biswas S, Wright CS, Henry JT, Lo K, Burov S, Lin Y, et al. Scaling laws governing stochastic growth and division of single bacterial cells. Proc Natl Acad Sci U S A. 2014;111: 15912–15917. doi:<a href="https://doi.org/10.1073/pnas.1403232111">10.1073/pnas.1403232111</a></div>
</div>
<div id="ref-Tzur2009-hl" class="csl-entry">
<div class="csl-left-margin">94. </div><div class="csl-right-inline">Tzur A, Kafri R, LeBleu VS, Lahav G, Kirschner MW. Cell growth and size homeostasis in proliferating animal cells. Science. 2009;325: 167–171. doi:<a href="https://doi.org/10.1126/science.1174294">10.1126/science.1174294</a></div>
</div>
<div id="ref-Pavelescu2018-ve" class="csl-entry">
<div class="csl-left-margin">95. </div><div class="csl-right-inline">Pavelescu I, Vilarrasa-Blasi J, Planas-Riverola A, González-Garcı́a M-P, Caño-Delgado AI, Ibañes M. A sizer model for cell differentiation in arabidopsis thaliana root growth. Mol Syst Biol. 2018;14: e7687. doi:<a href="https://doi.org/10.15252/msb.20177687">10.15252/msb.20177687</a></div>
</div>
<div id="ref-Sveiczer1996-yu" class="csl-entry">
<div class="csl-left-margin">96. </div><div class="csl-right-inline">Sveiczer A, Novak B, Mitchison JM. The size control of fission yeast revisited. J Cell Sci. 1996;109 ( Pt 12): 2947–2957. </div>
</div>
<div id="ref-Campos2014-sl" class="csl-entry">
<div class="csl-left-margin">97. </div><div class="csl-right-inline">Campos M, Surovtsev IV, Kato S, Paintdakhi A, Beltran B, Ebmeier SE, et al. A constant size extension drives bacterial cell size homeostasis. Cell. 2014;159: 1433–1446. doi:<a href="https://doi.org/10.1016/j.cell.2014.11.022">10.1016/j.cell.2014.11.022</a></div>
</div>
<div id="ref-Taheri-Araghi2015-sw" class="csl-entry">
<div class="csl-left-margin">98. </div><div class="csl-right-inline">Taheri-Araghi S, Bradde S, Sauls JT, Hill NS, Levin PA, Paulsson J, et al. Cell-size control and homeostasis in bacteria. Curr Biol. 2015;25: 385–391. doi:<a href="https://doi.org/10.1016/j.cub.2014.12.009">10.1016/j.cub.2014.12.009</a></div>
</div>
<div id="ref-Towbin2021-vx" class="csl-entry">
<div class="csl-left-margin">99. </div><div class="csl-right-inline">Towbin BD, Grosshans H. A folder mechanism ensures size uniformity among c. Elegans individuals by coupling growth and development. bioRxiv. 2021. </div>
</div>
<div id="ref-Soifer2016-tn" class="csl-entry">
<div class="csl-left-margin">100. </div><div class="csl-right-inline">Soifer I, Robert L, Amir A. <span>Single-Cell</span> analysis of growth in budding yeast and bacteria reveals a common size regulation strategy. Curr Biol. 2016;26: 356–361. doi:<a href="https://doi.org/10.1016/j.cub.2015.11.067">10.1016/j.cub.2015.11.067</a></div>
</div>
<div id="ref-Osella2014-sm" class="csl-entry">
<div class="csl-left-margin">101. </div><div class="csl-right-inline">Osella M, Nugent E, Cosentino Lagomarsino M. Concerted control of escherichia coli cell division. Proc Natl Acad Sci U S A. 2014;111: 3431–3435. doi:<a href="https://doi.org/10.1073/pnas.1313715111">10.1073/pnas.1313715111</a></div>
</div>
<div id="ref-Jorgensen2004-ir" class="csl-entry">
<div class="csl-left-margin">102. </div><div class="csl-right-inline">Jorgensen P, Tyers M. How cells coordinate growth and division. Curr Biol. 2004;14: R1014–27. doi:<a href="https://doi.org/10.1016/j.cub.2004.11.027">10.1016/j.cub.2004.11.027</a></div>
</div>
<div id="ref-Wang2010-qw" class="csl-entry">
<div class="csl-left-margin">103. </div><div class="csl-right-inline">Wang P, Robert L, Pelletier J, Dang WL, Taddei F, Wright A, et al. Robust growth of escherichia coli. Curr Biol. 2010;20: 1099–1103. doi:<a href="https://doi.org/10.1016/j.cub.2010.04.045">10.1016/j.cub.2010.04.045</a></div>
</div>
<div id="ref-Cadart2018-dv" class="csl-entry">
<div class="csl-left-margin">104. </div><div class="csl-right-inline">Cadart C, Monnier S, Grilli J, Sáez PJ, Srivastava N, Attia R, et al. Size control in mammalian cells involves modulation of both growth rate and cell cycle duration. Nat Commun. 2018;9: 3275. doi:<a href="https://doi.org/10.1038/s41467-018-05393-0">10.1038/s41467-018-05393-0</a></div>
</div>
<div id="ref-Moss-Taylor2019-bf" class="csl-entry">
<div class="csl-left-margin">105. </div><div class="csl-right-inline">Moss-Taylor L, Upadhyay A, Pan X, Kim M-J, O’Connor MB. Body size and <span>Tissue-Scaling</span> is regulated by <span>Motoneuron-Derived</span> activin<span>ß</span> in drosophila melanogaster. Genetics. 2019;213: 1447–1464. doi:<a href="https://doi.org/10.1534/genetics.119.302394">10.1534/genetics.119.302394</a></div>
</div>
<div id="ref-Spence2009-dk" class="csl-entry">
<div class="csl-left-margin">106. </div><div class="csl-right-inline">Spence AJ. Scaling in biology. Curr Biol. 2009;19: R57–61. doi:<a href="https://doi.org/10.1016/j.cub.2008.10.042">10.1016/j.cub.2008.10.042</a></div>
</div>
<div id="ref-Byerly1976-xt" class="csl-entry">
<div class="csl-left-margin">107. </div><div class="csl-right-inline">Byerly L, Cassada RC, Russell RL. The life cycle of the nematode caenorhabditis elegans. I. Wild-type growth and reproduction. Dev Biol. 1976;51: 23–33. doi:<a href="https://doi.org/10.1016/0012-1606(76)90119-6">10.1016/0012-1606(76)90119-6</a></div>
</div>
<div id="ref-Knight2002-fb" class="csl-entry">
<div class="csl-left-margin">108. </div><div class="csl-right-inline">Knight CG, Patel MN, Azevedo RBR, Leroi AM. A novel mode of ecdysozoan growth in caenorhabditis elegans. Evol Dev. 2002;4: 16–27. doi:<a href="https://doi.org/10.1046/j.1525-142x.2002.01058.x">10.1046/j.1525-142x.2002.01058.x</a></div>
</div>
<div id="ref-Faerberg2021-ui" class="csl-entry">
<div class="csl-left-margin">109. </div><div class="csl-right-inline">Faerberg DF, Gurarie V, Ruvinsky I. Inferring temporal organization of postembryonic development from high-content behavioral tracking. Dev Biol. 2021;475: 54–64. doi:<a href="https://doi.org/10.1016/j.ydbio.2021.02.007">10.1016/j.ydbio.2021.02.007</a></div>
</div>
<div id="ref-Filina2020-ia" class="csl-entry">
<div class="csl-left-margin">110. </div><div class="csl-right-inline">Filina O, Haagmans R, Zon JS van. Temporal scaling in c. Elegans larval development. bioRxiv. 2020. p. 2020.09.21.306423. doi:<a href="https://doi.org/10.1101/2020.09.21.306423">10.1101/2020.09.21.306423</a></div>
</div>
<div id="ref-Cook2016-lh" class="csl-entry">
<div class="csl-left-margin">111. </div><div class="csl-right-inline">Cook DE, Zdraljevic S, Tanny RE, Seo B, Riccardi DD, Noble LM, et al. The genetic basis of natural variation in caenorhabditis elegans telomere length. Genetics. 2016;204: 371–383. doi:<a href="https://doi.org/10.1534/genetics.116.191148">10.1534/genetics.116.191148</a></div>
</div>
<div id="ref-Stiernagle2006-ch" class="csl-entry">
<div class="csl-left-margin">112. </div><div class="csl-right-inline">Stiernagle T. Maintenance of c. elegans. WormBook; 2006. </div>
</div>
<div id="ref-Abramoff2004-my" class="csl-entry">
<div class="csl-left-margin">113. </div><div class="csl-right-inline">Abràmoff MD, Magalhães PJ, Ram SJ. Image processing with <span>ImageJ</span>. Biophotonics international. 2004;11: 36–42. </div>
</div>
<div id="ref-Andersen2015-uj" class="csl-entry">
<div class="csl-left-margin">114. </div><div class="csl-right-inline">Andersen EC, Shimko TC, Crissman JR, Ghosh R, Bloom JS, Seidel HS, et al. A powerful new quantitative genetics platform, combining caenorhabditis elegans <span>High-Throughput</span> fitness assays with a large collection of recombinant strains. G3. 2015;5: 911–920. doi:<a href="https://doi.org/10.1534/g3.115.017178">10.1534/g3.115.017178</a></div>
</div>
<div id="ref-Scrucca2016-by" class="csl-entry">
<div class="csl-left-margin">115. </div><div class="csl-right-inline">Scrucca L, Fop M, Murphy TB, Raftery AE. Mclust 5: Clustering, classification and density estimation using gaussian finite mixture models. R J. 2016;8: 289–317. </div>
</div>
<div id="ref-Smith2009-lx" class="csl-entry">
<div class="csl-left-margin">116. </div><div class="csl-right-inline">Smith MV, Boyd WA, Kissling GE, Rice JR, Snyder DW, Portier CJ, et al. A discrete time model for the analysis of medium-throughput c. Elegans growth data. PLoS One. 2009;4: e7018. doi:<a href="https://doi.org/10.1371/journal.pone.0007018">10.1371/journal.pone.0007018</a></div>
</div>
<div id="ref-Hermann2016-qs" class="csl-entry">
<div class="csl-left-margin">117. </div><div class="csl-right-inline">Hermann E. Lokern: Kernel regression smoothing with local or global plug-in bandwidth. <span>R</span> package version 1.1-8. <a href="https://CRAN.R-project.org/package=lokern">https://CRAN.R-project.org/package=lokern</a>; 2016. </div>
</div>
<div id="ref-Sakamoto1986-er" class="csl-entry">
<div class="csl-left-margin">118. </div><div class="csl-right-inline">Sakamoto Y, Ishiguro M, Kitagawa G. Akaike information criterion statistics. Dordrecht, The Netherlands: D Reidel. 1986;81: 26853. </div>
</div>
<div id="ref-Schwarz1978-mc" class="csl-entry">
<div class="csl-left-margin">119. </div><div class="csl-right-inline">Schwarz G. Estimating the dimension of a model. aos. 1978;6: 461–464. doi:<a href="https://doi.org/10.1214/aos/1176344136">10.1214/aos/1176344136</a></div>
</div>
<div id="ref-Kass1995-si" class="csl-entry">
<div class="csl-left-margin">120. </div><div class="csl-right-inline">Kass RE, Raftery AE. Bayes factors. J Am Stat Assoc. 1995;90: 773. doi:<a href="https://doi.org/10.2307/2291091">10.2307/2291091</a></div>
</div>
<div id="ref-Burnham2007-ye" class="csl-entry">
<div class="csl-left-margin">121. </div><div class="csl-right-inline">Burnham KP, Anderson DR. Model selection and multimodel inference: A practical <span>Information-Theoretic</span> approach. Springer Science &amp; Business Media; 2007. </div>
</div>
<div id="ref-Pulak2006-lc" class="csl-entry">
<div class="csl-left-margin">122. </div><div class="csl-right-inline">Pulak R. Techniques for analysis, sorting, and dispensing of c. Elegans on the <span>COPAS™</span> <span>Flow-Sorting</span> system. In: Strange K, editor. C Elegans: Methods and applications. Totowa, NJ: Humana Press; 2006. pp. 275–286. doi:<a href="https://doi.org/10.1385/1-59745-151-7:275">10.1385/1-59745-151-7:275</a></div>
</div>
<div id="ref-Nika2016-kc" class="csl-entry">
<div class="csl-left-margin">123. </div><div class="csl-right-inline">Nika L, Gibson T, Konkus R, Karp X. Fluorescent beads are a versatile tool for staging caenorhabditis elegans in different life histories. G3. 2016;6: 1923–1933. doi:<a href="https://doi.org/10.1534/g3.116.030163">10.1534/g3.116.030163</a></div>
</div>
<div id="ref-Vuaridel-Thurre2020-ug" class="csl-entry">
<div class="csl-left-margin">124. </div><div class="csl-right-inline">Vuaridel-Thurre G, Vuaridel AR, Dhar N, McKinney JD. Computational analysis of the mutual constraints between <span>Single-Cell</span> growth and division control models. Adv Biosyst. 2020;4: e1900103. doi:<a href="https://doi.org/10.1002/adbi.201900103">10.1002/adbi.201900103</a></div>
</div>
<div id="ref-Holzapfel2017-ds" class="csl-entry">
<div class="csl-left-margin">125. </div><div class="csl-right-inline">Holzapfel GA. Similarities between soft biological tissues and rubberlike materials. Constitutive Models for Rubber IV. 2017; 607–617. doi:<a href="https://doi.org/10.1201/9781315140216-105">10.1201/9781315140216-105</a></div>
</div>
<div id="ref-Essmann2017-zv" class="csl-entry">
<div class="csl-left-margin">126. </div><div class="csl-right-inline">Essmann CL, Elmi M, Shaw M, Anand GM, Pawar VM, Srinivasan MA. In-vivo high resolution <span>AFM</span> topographic imaging of caenorhabditis elegans reveals previously unreported surface structures of cuticle mutants. Nanomedicine. 2017;13: 183–189. doi:<a href="https://doi.org/10.1016/j.nano.2016.09.006">10.1016/j.nano.2016.09.006</a></div>
</div>
<div id="ref-Dodd2018-mz" class="csl-entry">
<div class="csl-left-margin">127. </div><div class="csl-right-inline">Dodd W, Tang L, Lone J-C, Wimberly K, Wu C-W, Consalvo C, et al. A damage sensor associated with the cuticle coordinates three core environmental stress responses in caenorhabditis elegans. Genetics. 2018;208: 1467–1482. doi:<a href="https://doi.org/10.1534/genetics.118.300827">10.1534/genetics.118.300827</a></div>
</div>
<div id="ref-Yochem2015-bs" class="csl-entry">
<div class="csl-left-margin">128. </div><div class="csl-right-inline">Yochem J, Lažetić V, Bell L, Chen L, Fay D. C. Elegans <span class="nocase">NIMA-related</span> kinases <span>NEKL-2</span> and <span>NEKL-3</span> are required for the completion of molting. Dev Biol. 2015;398: 255–266. doi:<a href="https://doi.org/10.1016/j.ydbio.2014.12.008">10.1016/j.ydbio.2014.12.008</a></div>
</div>
<div id="ref-Petzold2011-vi" class="csl-entry">
<div class="csl-left-margin">129. </div><div class="csl-right-inline">Petzold BC, Park S-J, Ponce P, Roozeboom C, Powell C, Goodman MB, et al. Caenorhabditis elegans body mechanics are regulated by body wall muscle tone. Biophys J. 2011;100: 1977–1985. doi:<a href="https://doi.org/10.1016/j.bpj.2011.02.035">10.1016/j.bpj.2011.02.035</a></div>
</div>
<div id="ref-Cox1981-ua" class="csl-entry">
<div class="csl-left-margin">130. </div><div class="csl-right-inline">Cox GN, Staprans S, Edgar RS. The cuticle of caenorhabditis elegans. <span>II</span>. Stage-specific changes in ultrastructure and protein composition during postembryonic development. Dev Biol. 1981;86: 456–470. doi:<a href="https://doi.org/10.1016/0012-1606(81)90204-9">10.1016/0012-1606(81)90204-9</a></div>
</div>
<div id="ref-Park2007-rm" class="csl-entry">
<div class="csl-left-margin">131. </div><div class="csl-right-inline">Park S-J, Goodman MB, Pruitt BL. Analysis of nematode mechanics by piezoresistive displacement clamp. Proc Natl Acad Sci U S A. 2007;104: 17376–17381. doi:<a href="https://doi.org/10.1073/pnas.0702138104">10.1073/pnas.0702138104</a></div>
</div>
<div id="ref-Gilpin2015-hq" class="csl-entry">
<div class="csl-left-margin">132. </div><div class="csl-right-inline">Gilpin W, Uppaluri S, Brangwynne CP. Worms under pressure: Bulk mechanical properties of c. Elegans are independent of the cuticle. Biophysical Journal. 2015. pp. 1887–1898. doi:<a href="https://doi.org/10.1016/j.bpj.2015.03.020">10.1016/j.bpj.2015.03.020</a></div>
</div>
<div id="ref-Golden1984-hp" class="csl-entry">
<div class="csl-left-margin">133. </div><div class="csl-right-inline">Golden JW, Riddle DL. The caenorhabditis elegans dauer larva: Developmental effects of pheromone, food, and temperature. Dev Biol. 1984;102: 368–378. doi:<a href="https://doi.org/10.1016/0012-1606(84)90201-x">10.1016/0012-1606(84)90201-x</a></div>
</div>
<div id="ref-Schiller2013-ub" class="csl-entry">
<div class="csl-left-margin">134. </div><div class="csl-right-inline">Schiller HB, Fässler R. Mechanosensitivity and compositional dynamics of cell-matrix adhesions. EMBO Rep. 2013;14: 509–519. doi:<a href="https://doi.org/10.1038/embor.2013.49">10.1038/embor.2013.49</a></div>
</div>
<div id="ref-Wolfenson2011-ad" class="csl-entry">
<div class="csl-left-margin">135. </div><div class="csl-right-inline">Wolfenson H, Bershadsky A, Henis YI, Geiger B. Actomyosin-generated tension controls the molecular kinetics of focal adhesions. J Cell Sci. 2011;124: 1425–1432. doi:<a href="https://doi.org/10.1242/jcs.077388">10.1242/jcs.077388</a></div>
</div>
<div id="ref-Suman2019-wz" class="csl-entry">
<div class="csl-left-margin">136. </div><div class="csl-right-inline">Suman SK, Daday C, Ferraro T, Vuong-Brender T, Tak S, Quintin S, et al. The plakin domain of <span class="nocase">VAB-10/plectin</span> acts as a hub in a mechanotransduction pathway to promote morphogenesis. Development. 2019;146. doi:<a href="https://doi.org/10.1242/dev.183780">10.1242/dev.183780</a></div>
</div>
<div id="ref-Zhang2011-ea" class="csl-entry">
<div class="csl-left-margin">137. </div><div class="csl-right-inline">Zhang H, Landmann F, Zahreddine H, Rodriguez D, Koch M, Labouesse M. A tension-induced mechanotransduction pathway promotes epithelial morphogenesis. Nature. 2011;471: 99–103. doi:<a href="https://doi.org/10.1038/nature09765">10.1038/nature09765</a></div>
</div>
<div id="ref-Moerman2006-yw" class="csl-entry">
<div class="csl-left-margin">138. </div><div class="csl-right-inline">Moerman DG, Williams BD. Sarcomere assembly in c. Elegans muscle. WormBook. 2006; 1–16. doi:<a href="https://doi.org/10.1895/wormbook.1.81.1">10.1895/wormbook.1.81.1</a></div>
</div>
<div id="ref-Costa1997-qb" class="csl-entry">
<div class="csl-left-margin">139. </div><div class="csl-right-inline">Costa M, Draper BW, Priess JR. The role of actin filaments in patterning the caenorhabditis elegans cuticle. Dev Biol. 1997;184: 373–384. doi:<a href="https://doi.org/10.1006/dbio.1997.8530">10.1006/dbio.1997.8530</a></div>
</div>
<div id="ref-Broday2007-fx" class="csl-entry">
<div class="csl-left-margin">140. </div><div class="csl-right-inline">Broday L, Hauser CA, Kolotuev I, Ronai Z. Muscle-epidermis interactions affect exoskeleton patterning in caenorhabditis elegans. Dev Dyn. 2007;236: 3129–3136. doi:<a href="https://doi.org/10.1002/dvdy.21341">10.1002/dvdy.21341</a></div>
</div>
<div id="ref-Madaan2020-ni" class="csl-entry">
<div class="csl-left-margin">141. </div><div class="csl-right-inline">Madaan U, Faure L, Chowdhury A, Ahmed S, Ciccarelli EJ, Gumienny TL, et al. Feedback regulation of <span>BMP</span> signaling by cuticle collagens. Mol Biol Cell. 2020;31: 825–832. doi:<a href="https://doi.org/10.1091/mbc.E19-07-0390">10.1091/mbc.E19-07-0390</a></div>
</div>
<div id="ref-Katz2018-we" class="csl-entry">
<div class="csl-left-margin">142. </div><div class="csl-right-inline">Katz SS, Maybrun C, Maul-Newby HM, Frand AR. Non-canonical apical constriction shapes emergent matrices in c. elegans. bioRxiv. 2018. p. 189951. doi:<a href="https://doi.org/10.1101/189951">10.1101/189951</a></div>
</div>
<div id="ref-Fechner2018-zw" class="csl-entry">
<div class="csl-left-margin">143. </div><div class="csl-right-inline">Fechner S, Loizeau F, Nekimken AL, Pruitt BL, Goodman MB. The bodies of dpy-10(e128) are twice as stiff as wild type. MicroPubl Biol. 2018;2018. doi:<a href="https://doi.org/10.17912/ecsm-mp67">10.17912/ecsm-mp67</a></div>
</div>
<div id="ref-Horvitz1980-ha" class="csl-entry">
<div class="csl-left-margin">144. </div><div class="csl-right-inline">Horvitz HR, Sulston JE. Isolation and genetic characterization of cell-lineage mutants of the nematode caenorhabditis elegans. Genetics. 1980;96: 435–454. </div>
</div>
<div id="ref-Waddington1942-ih" class="csl-entry">
<div class="csl-left-margin">145. </div><div class="csl-right-inline">Waddington CH. <span>CANALIZATION</span> <span>OF</span> <span>DEVELOPMENT</span> <span>AND</span> <span>THE</span> <span>INHERITANCE</span> <span>OF</span> <span>ACQUIRED</span> <span>CHARACTERS</span>. Nature. 1942;150: 563–565. doi:<a href="https://doi.org/10.1038/150563a0">10.1038/150563a0</a></div>
</div>
<div id="ref-Tanner1963-wa" class="csl-entry">
<div class="csl-left-margin">146. </div><div class="csl-right-inline">Tanner JM. <span>THE</span> <span>REGULATION</span> <span>OF</span> <span>HUMAN</span> <span>GROWTH</span>. Child Dev. 1963;34: 817–847. doi:<a href="https://doi.org/10.1111/j.1467-8624.1963.tb05970.x">10.1111/j.1467-8624.1963.tb05970.x</a></div>
</div>
<div id="ref-Cameron2012-fz" class="csl-entry">
<div class="csl-left-margin">147. </div><div class="csl-right-inline">Cameron N, Bogin B. Human growth and development. Academic Press; 2012. </div>
</div>
<div id="ref-Desmond2017-ek" class="csl-entry">
<div class="csl-left-margin">148. </div><div class="csl-right-inline">Desmond C, Casale D. Catch-up growth in stunted children: Definitions and predictors. PLoS One. 2017;12: e0189135. doi:<a href="https://doi.org/10.1371/journal.pone.0189135">10.1371/journal.pone.0189135</a></div>
</div>
<div id="ref-Hector2012-xi" class="csl-entry">
<div class="csl-left-margin">149. </div><div class="csl-right-inline">Hector KL, Nakagawa S. Quantitative analysis of compensatory and catch-up growth in diverse taxa. J Anim Ecol. 2012;81: 583–593. doi:<a href="https://doi.org/10.1111/j.1365-2656.2011.01942.x">10.1111/j.1365-2656.2011.01942.x</a></div>
</div>
<div id="ref-Andersen2022-su" class="csl-entry">
<div class="csl-left-margin">150. </div><div class="csl-right-inline">Andersen EC, Rockman MV. Natural genetic variation as a tool for discovery in caenorhabditis nematodes. Genetics. 2022;220. doi:<a href="https://doi.org/10.1093/genetics/iyab156">10.1093/genetics/iyab156</a></div>
</div>
<div id="ref-Garcia-Gonzalez2017-wl" class="csl-entry">
<div class="csl-left-margin">151. </div><div class="csl-right-inline">Garcı́a-González AP, Ritter AD, Shrestha S, Andersen EC, Yilmaz LS, Walhout AJM. Bacterial metabolism affects the c. Elegans response to cancer chemotherapeutics. Cell. 2017;169: 431–441.e8. doi:<a href="https://doi.org/10.1016/j.cell.2017.03.046">10.1016/j.cell.2017.03.046</a></div>
</div>
<div id="ref-Evans2018-sj" class="csl-entry">
<div class="csl-left-margin">152. </div><div class="csl-right-inline">Evans KS, Brady SC, Bloom JS, Tanny RE, Cook DE, Giuliani SE, et al. Shared genomic regions underlie natural variation in diverse toxin responses. Genetics. 2018;210: 1509–1525. doi:<a href="https://doi.org/10.1534/genetics.118.301311">10.1534/genetics.118.301311</a></div>
</div>
<div id="ref-Team2014-fu" class="csl-entry">
<div class="csl-left-margin">153. </div><div class="csl-right-inline">Team RC. R: A language and environment for statistical computing http://www. R-project org. 2014. </div>
</div>
<div id="ref-Rockman2009-qy" class="csl-entry">
<div class="csl-left-margin">154. </div><div class="csl-right-inline">Rockman MV, Kruglyak L. Recombinational landscape and population genomics of caenorhabditis elegans. PLoS Genet. 2009;5: e1000419. doi:<a href="https://doi.org/10.1371/journal.pgen.1000419">10.1371/journal.pgen.1000419</a></div>
</div>
<div id="ref-Lee2021-wx" class="csl-entry">
<div class="csl-left-margin">155. </div><div class="csl-right-inline">Lee D, Zdraljevic S, Stevens L, Wang Y, Tanny RE, Crombie TA, et al. Balancing selection maintains hyper-divergent haplotypes in caenorhabditis elegans. Nat Ecol Evol. 2021;5: 794–807. doi:<a href="https://doi.org/10.1038/s41559-021-01435-x">10.1038/s41559-021-01435-x</a></div>
</div>
<div id="ref-Gillooly2001-tu" class="csl-entry">
<div class="csl-left-margin">156. </div><div class="csl-right-inline">Gillooly DJ, Simonsen A, Stenmark H. Cellular functions of phosphatidylinositol 3-phosphate and <span>FYVE</span> domain proteins. Biochem J. 2001;355: 249–258. doi:<a href="https://doi.org/10.1042/bj3550249">10.1042/bj3550249</a></div>
</div>
<div id="ref-Nicot2006-rn" class="csl-entry">
<div class="csl-left-margin">157. </div><div class="csl-right-inline">Nicot A-S, Fares H, Payrastre B, Chisholm AD, Labouesse M, Laporte J. The phosphoinositide kinase <span>PIKfyve/Fab1p</span> regulates terminal lysosome maturation in caenorhabditis elegans. Mol Biol Cell. 2006;17: 3062–3074. doi:<a href="https://doi.org/10.1091/mbc.e05-12-1120">10.1091/mbc.e05-12-1120</a></div>
</div>
<div id="ref-Asahina2000-zz" class="csl-entry">
<div class="csl-left-margin">158. </div><div class="csl-right-inline">Asahina M, Ishihara T, Jindra M, Kohara Y, Katsura I, Hirose S. The conserved nuclear receptor <span>Ftz-F1</span> is required for embryogenesis, moulting and reproduction in caenorhabditis elegans. Genes Cells. 2000;5: 711–723. doi:<a href="https://doi.org/10.1046/j.1365-2443.2000.00361.x">10.1046/j.1365-2443.2000.00361.x</a></div>
</div>
<div id="ref-Gissendanner2000-hs" class="csl-entry">
<div class="csl-left-margin">159. </div><div class="csl-right-inline">Gissendanner CR, Sluder AE. Nhr-25, the caenorhabditis elegans ortholog of ftz-f1, is required for epidermal and somatic gonad development. Dev Biol. 2000;221: 259–272. doi:<a href="https://doi.org/10.1006/dbio.2000.9679">10.1006/dbio.2000.9679</a></div>
</div>
<div id="ref-Maeda2001-eo" class="csl-entry">
<div class="csl-left-margin">160. </div><div class="csl-right-inline">Maeda I, Kohara Y, Yamamoto M, Sugimoto A. Large-scale analysis of gene function in caenorhabditis elegans by high-throughput <span>RNAi</span>. Curr Biol. 2001;11: 171–176. doi:<a href="https://doi.org/10.1016/s0960-9822(01)00052-5">10.1016/s0960-9822(01)00052-5</a></div>
</div>
<div id="ref-Houle1992-vs" class="csl-entry">
<div class="csl-left-margin">161. </div><div class="csl-right-inline">Houle D. Comparing evolvability and variability of quantitative traits. Genetics. 1992;130: 195–204. doi:<a href="https://doi.org/10.1093/genetics/130.1.195">10.1093/genetics/130.1.195</a></div>
</div>
<div id="ref-Ghosh2015-bw" class="csl-entry">
<div class="csl-left-margin">162. </div><div class="csl-right-inline">Ghosh R, Bloom JS, Mohammadi A, Schumer ME, Andolfatto P, Ryu W, et al. Genetics of intraspecies variation in avoidance behavior induced by a thermal stimulus in caenorhabditis elegans. Genetics. 2015;200: 1327–1339. doi:<a href="https://doi.org/10.1534/genetics.115.178491">10.1534/genetics.115.178491</a></div>
</div>
<div id="ref-Noble2017-py" class="csl-entry">
<div class="csl-left-margin">163. </div><div class="csl-right-inline">Noble LM, Chelo I, Guzella T, Afonso B, Riccardi DD, Ammerman P, et al. Polygenicity and epistasis underlie <span>Fitness-Proximal</span> traits in the caenorhabditis elegans multiparental experimental evolution (<span>CeMEE</span>) panel. Genetics. 2017;207: 1663–1685. doi:<a href="https://doi.org/10.1534/genetics.117.300406">10.1534/genetics.117.300406</a></div>
</div>
<div id="ref-Bernstein2019-pq" class="csl-entry">
<div class="csl-left-margin">164. </div><div class="csl-right-inline">Bernstein MR, Zdraljevic S, Andersen EC, Rockman MV. Tightly linked antagonistic-effect loci underlie polygenic phenotypic variation in c. elegans. Evol Lett. 2019;3: 462–473. doi:<a href="https://doi.org/10.1002/evl3.139">10.1002/evl3.139</a></div>
</div>
<div id="ref-Rockman2012-pu" class="csl-entry">
<div class="csl-left-margin">165. </div><div class="csl-right-inline">Rockman MV. The <span>QTN</span> program and the alleles that matter for evolution: All that’s gold does not glitter. Evolution. 2012;66: 1–17. doi:<a href="https://doi.org/10.1111/j.1558-5646.2011.01486.x">10.1111/j.1558-5646.2011.01486.x</a></div>
</div>
<div id="ref-Yengo2022-yw" class="csl-entry">
<div class="csl-left-margin">166. </div><div class="csl-right-inline">Yengo L, Vedantam S, Marouli E, Sidorenko J, Bartell E, Sakaue S, et al. A saturated map of common genetic variants associated with human height from 5.4 million individuals of diverse ancestries. bioRxiv. 2022. p. 2022.01.07.475305. doi:<a href="https://doi.org/10.1101/2022.01.07.475305">10.1101/2022.01.07.475305</a></div>
</div>
<div id="ref-Morck2006-pl" class="csl-entry">
<div class="csl-left-margin">167. </div><div class="csl-right-inline">Mörck C, Pilon M. C. Elegans feeding defective mutants have shorter body lengths and increased autophagy. BMC Dev Biol. 2006;6: 39. doi:<a href="https://doi.org/10.1186/1471-213X-6-39">10.1186/1471-213X-6-39</a></div>
</div>
<div id="ref-Cho2021-xa" class="csl-entry">
<div class="csl-left-margin">168. </div><div class="csl-right-inline">Cho JY, Choi T-W, Kim SH, Ahnn J, Lee S-K. Morphological characterization of small, dumpy, and long phenotypes in caenorhabditis elegans. Mol Cells. 2021;44: 160–167. doi:<a href="https://doi.org/10.14348/molcells.2021.2236">10.14348/molcells.2021.2236</a></div>
</div>
<div id="ref-Boyle2017-bb" class="csl-entry">
<div class="csl-left-margin">169. </div><div class="csl-right-inline">Boyle EA, Li YI, Pritchard JK. An expanded view of complex traits: From polygenic to omnigenic. Cell. 2017;169: 1177–1186. doi:<a href="https://doi.org/10.1016/j.cell.2017.05.038">10.1016/j.cell.2017.05.038</a></div>
</div>
<div id="ref-Snoek2014-tv" class="csl-entry">
<div class="csl-left-margin">170. </div><div class="csl-right-inline">Snoek LB, Orbidans HE, Stastna JJ, Aartse A, Rodriguez M, Riksen JAG, et al. Widespread genomic incompatibilities in caenorhabditis elegans. G3. 2014;4: 1813–1823. doi:<a href="https://doi.org/10.1534/g3.114.013151">10.1534/g3.114.013151</a></div>
</div>
<div id="ref-Maulana2022-jy" class="csl-entry">
<div class="csl-left-margin">171. </div><div class="csl-right-inline">Maulana MI, Riksen JAG, Snoek BL, Kammenga JE, Sterken MG. The genetic architecture underlying body-size traits plasticity over different temperatures and developmental stages in caenorhabditis elegans. Heredity. 2022. doi:<a href="https://doi.org/10.1038/s41437-022-00528-y">10.1038/s41437-022-00528-y</a></div>
</div>
<div id="ref-Xie2016-zd" class="csl-entry">
<div class="csl-left-margin">172. </div><div class="csl-right-inline">Xie Y. Bookdown: Authoring books and technical documents with <span>R</span> markdown. CRC Press; 2016. </div>
</div>
<div id="ref-Nyaanga2021-ch" class="csl-entry">
<div class="csl-left-margin">173. </div><div class="csl-right-inline">Nyaanga J, Crombie TA, Widmayer SJ, Andersen EC. easyXpress: An <span>R</span> package to analyze and visualize high-throughput c. Elegans microscopy data generated using <span>CellProfiler</span>. PLoS One. 2021;16: e0252000. doi:<a href="https://doi.org/10.1371/journal.pone.0252000">10.1371/journal.pone.0252000</a></div>
</div>
<div id="ref-Swedlow2012-vb" class="csl-entry">
<div class="csl-left-margin">174. </div><div class="csl-right-inline">Swedlow JR. Innovation in biological microscopy: Current status and future directions. Bioessays. 2012;34: 333–340. doi:<a href="https://doi.org/10.1002/bies.201100168">10.1002/bies.201100168</a></div>
</div>
<div id="ref-Cassidy2005-sf" class="csl-entry">
<div class="csl-left-margin">175. </div><div class="csl-right-inline">Cassidy PJ, Radda GK. Molecular imaging perspectives. J R Soc Interface. 2005;2: 133–144. doi:<a href="https://doi.org/10.1098/rsif.2005.0040">10.1098/rsif.2005.0040</a></div>
</div>
<div id="ref-White1986-mb" class="csl-entry">
<div class="csl-left-margin">176. </div><div class="csl-right-inline">White JG, Southgate E, Thomson JN, Brenner S. The structure of the nervous system of the nematode caenorhabditis elegans. Philos Trans R Soc Lond B Biol Sci. 1986;314: 1–340. doi:<a href="https://doi.org/10.1098/rstb.1986.0056">10.1098/rstb.1986.0056</a></div>
</div>
<div id="ref-Teotonio2017-fr" class="csl-entry">
<div class="csl-left-margin">177. </div><div class="csl-right-inline">Teotónio H, Estes S, Phillips PC, Baer CF. Experimental evolution with nematodes. Genetics. 2017;206: 691–716. doi:<a href="https://doi.org/10.1534/genetics.115.186288">10.1534/genetics.115.186288</a></div>
</div>
<div id="ref-Wahlby2012-lu" class="csl-entry">
<div class="csl-left-margin">178. </div><div class="csl-right-inline">Wählby C, Kamentsky L, Liu ZH, Riklin-Raviv T, Conery AL, O’Rourke EJ, et al. An image analysis toolbox for high-throughput c. Elegans assays. Nat Methods. 2012;9: 714–716. doi:<a href="https://doi.org/10.1038/nmeth.1984">10.1038/nmeth.1984</a></div>
</div>
<div id="ref-Boyd2012-ii" class="csl-entry">
<div class="csl-left-margin">179. </div><div class="csl-right-inline">Boyd WA, Smith MV, Freedman JH. Caenorhabditis elegans as a model in developmental toxicology. In: Harris C, Hansen JM, editors. Developmental toxicology: Methods and protocols. Totowa, NJ: Humana Press; 2012. pp. 15–24. doi:<a href="https://doi.org/10.1007/978-1-61779-867-2\_3">10.1007/978-1-61779-867-2\_3</a></div>
</div>
<div id="ref-Tukey1977-vu" class="csl-entry">
<div class="csl-left-margin">180. </div><div class="csl-right-inline">Tukey JW, Others. Exploratory data analysis. Reading, Mass.; 1977. </div>
</div>
<div id="ref-Dilks2020-jf" class="csl-entry">
<div class="csl-left-margin">181. </div><div class="csl-right-inline">Dilks CM, Hahnel SR, Sheng Q, Long L, McGrath PT, Andersen EC. Quantitative benzimidazole resistance and fitness effects of parasitic nematode beta-tubulin alleles. Int J Parasitol Drugs Drug Resist. 2020;14: 28–36. doi:<a href="https://doi.org/10.1016/j.ijpddr.2020.08.003">10.1016/j.ijpddr.2020.08.003</a></div>
</div>
<div id="ref-Wit2021-xa" class="csl-entry">
<div class="csl-left-margin">182. </div><div class="csl-right-inline">Wit J, Rodriguez BC, Andersen EC. Natural variation in caenorhabditis elegans responses to the anthelmintic emodepside. Int J Parasitol Drugs Drug Resist. 2021;16: 1–8. doi:<a href="https://doi.org/10.1016/j.ijpddr.2021.04.001">10.1016/j.ijpddr.2021.04.001</a></div>
</div>
<div id="ref-Nyaanga2021-xh" class="csl-entry">
<div class="csl-left-margin">183. </div><div class="csl-right-inline">Nyaanga J, Goss C, Zhang G, Ahmed HN, Andersen EJ, Miller IR, et al. Highly scaled measurements of c. Elegans development suggest that physical constraints guide growth trajectories and animal shape. bioRxiv. 2021. p. 2021.04.01.438121. doi:<a href="https://doi.org/10.1101/2021.04.01.438121">10.1101/2021.04.01.438121</a></div>
</div>
<div id="ref-Aguinaldo1997-ig" class="csl-entry">
<div class="csl-left-margin">184. </div><div class="csl-right-inline">Aguinaldo AM, Turbeville JM, Linford LS, Rivera MC, Garey JR, Raff RA, et al. Evidence for a clade of nematodes, arthropods and other moulting animals. Nature. 1997;387: 489–493. doi:<a href="https://doi.org/10.1038/387489a0">10.1038/387489a0</a></div>
</div>
<div id="ref-Telford2008-ya" class="csl-entry">
<div class="csl-left-margin">185. </div><div class="csl-right-inline">Telford MJ, Bourlat SJ, Economou A, Papillon D, Rota-Stabelli O. The evolution of the ecdysozoa. Philos Trans R Soc Lond B Biol Sci. 2008;363: 1529–1537. doi:<a href="https://doi.org/10.1098/rstb.2007.2243">10.1098/rstb.2007.2243</a></div>
</div>
<div id="ref-Alpatov1929-li" class="csl-entry">
<div class="csl-left-margin">186. </div><div class="csl-right-inline">Alpatov WW. Growth and variation of the larvae of drosophila melanogaster. J Exp Zool. 1929;52: 407–437. doi:<a href="https://doi.org/10.1002/jez.1400520303">10.1002/jez.1400520303</a></div>
</div>
<div id="ref-Rice1968-em" class="csl-entry">
<div class="csl-left-margin">187. </div><div class="csl-right-inline">Rice AL. Growth <span>‘rules’</span> and the larvae of decapod crustaceans. J Nat Hist. 1968;2: 525–530. doi:<a href="https://doi.org/10.1080/00222936800771011">10.1080/00222936800771011</a></div>
</div>
<div id="ref-Howells1983-vf" class="csl-entry">
<div class="csl-left-margin">188. </div><div class="csl-right-inline">Howells RE, Blainey LJ. The moulting process and the phenomenon of intermoult growth in the filarial nematode brugia pahangi. Parasitology. 1983;87 (Pt 3): 493–505. doi:<a href="https://doi.org/10.1017/s0031182000083013">10.1017/s0031182000083013</a></div>
</div>
<div id="ref-Wilson1976-bc" class="csl-entry">
<div class="csl-left-margin">189. </div><div class="csl-right-inline">Wilson PAG. Nematode growth patterns and the moulting cycle: The population growth profile. J Zool. 1976;179: 135–151. doi:<a href="https://doi.org/10.1111/j.1469-7998.1976.tb03231.x">10.1111/j.1469-7998.1976.tb03231.x</a></div>
</div>
<div id="ref-Malakhov1986-nb" class="csl-entry">
<div class="csl-left-margin">190. </div><div class="csl-right-inline">Malakhov UV, Others. Nematodes: Structure, development, classification and phylogeny. Nematodes: structure, development, classification and phylogeny. 1986. </div>
</div>
<div id="ref-Uppaluri2016-gm" class="csl-entry">
<div class="csl-left-margin">191. </div><div class="csl-right-inline">Uppaluri S, Weber SC, Brangwynne CP. Hierarchical size scaling during multicellular growth and development. Cell Rep. 2016;17: 345–352. doi:<a href="https://doi.org/10.1016/j.celrep.2016.09.007">10.1016/j.celrep.2016.09.007</a></div>
</div>
<div id="ref-Essmann2020-cb" class="csl-entry">
<div class="csl-left-margin">192. </div><div class="csl-right-inline">Essmann CL, Martinez-Martinez D, Pryor R, Leung K-Y, Krishnan KB, Lui PP, et al. Mechanical properties measured by atomic force microscopy define health biomarkers in ageing c. elegans. Nature Communications. 2020. doi:<a href="https://doi.org/10.1038/s41467-020-14785-0">10.1038/s41467-020-14785-0</a></div>
</div>
<div id="ref-Rahimi2022-ur" class="csl-entry">
<div class="csl-left-margin">193. </div><div class="csl-right-inline">Rahimi M, Sohrabi S, Murphy CT. Novel elasticity measurements reveal c. Elegans cuticle stiffens with age and in a long-lived mutant. Biophys J. 2022;121: 515–524. doi:<a href="https://doi.org/10.1016/j.bpj.2022.01.013">10.1016/j.bpj.2022.01.013</a></div>
</div>
<div id="ref-Riddle1997-jv" class="csl-entry">
<div class="csl-left-margin">194. </div><div class="csl-right-inline">Riddle DL, Blumenthal T, Meyer BJ, Priess JR. Introduction: The neural circuit for locomotion. Cold Spring Harbor Laboratory Press; 1997. </div>
</div>
<div id="ref-Altun2005-jf" class="csl-entry">
<div class="csl-left-margin">195. </div><div class="csl-right-inline">Altun ZF, Hall DH. <span>WormAtlas</span> hermaphrodite handbook - nervous system - general description. WormAtlas. 2005. doi:<a href="https://doi.org/10.3908/wormatlas.1.18">10.3908/wormatlas.1.18</a></div>
</div>
<div id="ref-Altun2006-zw" class="csl-entry">
<div class="csl-left-margin">196. </div><div class="csl-right-inline">Altun ZF, Hall DH. <span>WormAtas</span> hermaphrodite handbook - introduction. WormAtlas. 2006. doi:<a href="https://doi.org/10.3908/wormatlas.1.1">10.3908/wormatlas.1.1</a></div>
</div>
<div id="ref-Pigliucci2006-gf" class="csl-entry">
<div class="csl-left-margin">197. </div><div class="csl-right-inline">Pigliucci M, Murren CJ, Schlichting CD. Phenotypic plasticity and evolution by genetic assimilation. J Exp Biol. 2006;209: 2362–2367. doi:<a href="https://doi.org/10.1242/jeb.02070">10.1242/jeb.02070</a></div>
</div>
<div id="ref-Lafuente2019-up" class="csl-entry">
<div class="csl-left-margin">198. </div><div class="csl-right-inline">Lafuente E, Beldade P. Genomics of developmental plasticity in animals. Front Genet. 2019;10: 720. doi:<a href="https://doi.org/10.3389/fgene.2019.00720">10.3389/fgene.2019.00720</a></div>
</div>
<div id="ref-Pigliucci2005-te" class="csl-entry">
<div class="csl-left-margin">199. </div><div class="csl-right-inline">Pigliucci M. Evolution of phenotypic plasticity: Where are we going now? Trends in Ecology &amp; Evolution. 2005. pp. 481–486. doi:<a href="https://doi.org/10.1016/j.tree.2005.06.001">10.1016/j.tree.2005.06.001</a></div>
</div>
<div id="ref-Hodgins-Davis2012-ok" class="csl-entry">
<div class="csl-left-margin">200. </div><div class="csl-right-inline">Hodgins-Davis A, Adomas AB, Warringer J, Townsend JP. Abundant gene-by-environment interactions in gene expression reaction norms to copper within saccharomyces cerevisiae. Genome Biol Evol. 2012;4: 1061–1079. doi:<a href="https://doi.org/10.1093/gbe/evs084">10.1093/gbe/evs084</a></div>
</div>
<div id="ref-Rocha2009-zc" class="csl-entry">
<div class="csl-left-margin">201. </div><div class="csl-right-inline">Rocha F, Medeiros HF, Klaczko LB. The reaction norm for abdominal pigmentation and its curve in drosophila mediopunctata depend on the mean phenotypic value. Evolution. 2009;63: 280–287. doi:<a href="https://doi.org/10.1111/j.1558-5646.2008.00503.x">10.1111/j.1558-5646.2008.00503.x</a></div>
</div>
</div>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="discussion-2.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="appendix.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": false,
"twitter": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": false
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-thesis.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
